<html><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><title>Draft genome of the European medicinal leech Hirudo medicinalis (Annelida, Clitellata, Hirudiniformes) with emphasis on anticoagulants | Scientific Reports</title><style>
/* cspell:disable-file */
/* webkit printing magic: print all background colors */
html {
	-webkit-print-color-adjust: exact;
}
* {
	box-sizing: border-box;
	-webkit-print-color-adjust: exact;
}

html,
body {
	margin: 0;
	padding: 0;
}
@media only screen {
	body {
		margin: 2em auto;
		max-width: 900px;
		color: rgb(55, 53, 47);
	}
}

body {
	line-height: 1.5;
	white-space: pre-wrap;
}

a,
a.visited {
	color: inherit;
	text-decoration: underline;
}

.pdf-relative-link-path {
	font-size: 80%;
	color: #444;
}

h1,
h2,
h3 {
	letter-spacing: -0.01em;
	line-height: 1.2;
	font-weight: 600;
	margin-bottom: 0;
}

.page-title {
	font-size: 2.5rem;
	font-weight: 700;
	margin-top: 0;
	margin-bottom: 0.75em;
}

h1 {
	font-size: 1.875rem;
	margin-top: 1.875rem;
}

h2 {
	font-size: 1.5rem;
	margin-top: 1.5rem;
}

h3 {
	font-size: 1.25rem;
	margin-top: 1.25rem;
}

.source {
	border: 1px solid #ddd;
	border-radius: 3px;
	padding: 1.5em;
	word-break: break-all;
}

.callout {
	border-radius: 3px;
	padding: 1rem;
}

figure {
	margin: 1.25em 0;
	page-break-inside: avoid;
}

figcaption {
	opacity: 0.5;
	font-size: 85%;
	margin-top: 0.5em;
}

mark {
	background-color: transparent;
}

.indented {
	padding-left: 1.5em;
}

hr {
	background: transparent;
	display: block;
	width: 100%;
	height: 1px;
	visibility: visible;
	border: none;
	border-bottom: 1px solid rgba(55, 53, 47, 0.09);
}

img {
	max-width: 100%;
}

@media only print {
	img {
		max-height: 100vh;
		object-fit: contain;
	}
}

@page {
	margin: 1in;
}

.collection-content {
	font-size: 0.875rem;
}

.column-list {
	display: flex;
	justify-content: space-between;
}

.column {
	padding: 0 1em;
}

.column:first-child {
	padding-left: 0;
}

.column:last-child {
	padding-right: 0;
}

.table_of_contents-item {
	display: block;
	font-size: 0.875rem;
	line-height: 1.3;
	padding: 0.125rem;
}

.table_of_contents-indent-1 {
	margin-left: 1.5rem;
}

.table_of_contents-indent-2 {
	margin-left: 3rem;
}

.table_of_contents-indent-3 {
	margin-left: 4.5rem;
}

.table_of_contents-link {
	text-decoration: none;
	opacity: 0.7;
	border-bottom: 1px solid rgba(55, 53, 47, 0.18);
}

table,
th,
td {
	border: 1px solid rgba(55, 53, 47, 0.09);
	border-collapse: collapse;
}

table {
	border-left: none;
	border-right: none;
}

th,
td {
	font-weight: normal;
	padding: 0.25em 0.5em;
	line-height: 1.5;
	min-height: 1.5em;
	text-align: left;
}

th {
	color: rgba(55, 53, 47, 0.6);
}

ol,
ul {
	margin: 0;
	margin-block-start: 0.6em;
	margin-block-end: 0.6em;
}

li > ol:first-child,
li > ul:first-child {
	margin-block-start: 0.6em;
}

ul > li {
	list-style: disc;
}

ul.to-do-list {
	padding-inline-start: 0;
}

ul.to-do-list > li {
	list-style: none;
}

.to-do-children-checked {
	text-decoration: line-through;
	opacity: 0.375;
}

ul.toggle > li {
	list-style: none;
}

ul {
	padding-inline-start: 1.7em;
}

ul > li {
	padding-left: 0.1em;
}

ol {
	padding-inline-start: 1.6em;
}

ol > li {
	padding-left: 0.2em;
}

.mono ol {
	padding-inline-start: 2em;
}

.mono ol > li {
	text-indent: -0.4em;
}

.toggle {
	padding-inline-start: 0em;
	list-style-type: none;
}

/* Indent toggle children */
.toggle > li > details {
	padding-left: 1.7em;
}

.toggle > li > details > summary {
	margin-left: -1.1em;
}

.selected-value {
	display: inline-block;
	padding: 0 0.5em;
	background: rgba(206, 205, 202, 0.5);
	border-radius: 3px;
	margin-right: 0.5em;
	margin-top: 0.3em;
	margin-bottom: 0.3em;
	white-space: nowrap;
}

.collection-title {
	display: inline-block;
	margin-right: 1em;
}

.page-description {
    margin-bottom: 2em;
}

.simple-table {
	margin-top: 1em;
	font-size: 0.875rem;
	empty-cells: show;
}
.simple-table td {
	height: 29px;
	min-width: 120px;
}

.simple-table th {
	height: 29px;
	min-width: 120px;
}

.simple-table-header-color {
	background: rgb(247, 246, 243);
	color: black;
}
.simple-table-header {
	font-weight: 500;
}

time {
	opacity: 0.5;
}

.icon {
	display: inline-block;
	max-width: 1.2em;
	max-height: 1.2em;
	text-decoration: none;
	vertical-align: text-bottom;
	margin-right: 0.5em;
}

img.icon {
	border-radius: 3px;
}

.user-icon {
	width: 1.5em;
	height: 1.5em;
	border-radius: 100%;
	margin-right: 0.5rem;
}

.user-icon-inner {
	font-size: 0.8em;
}

.text-icon {
	border: 1px solid #000;
	text-align: center;
}

.page-cover-image {
	display: block;
	object-fit: cover;
	width: 100%;
	max-height: 30vh;
}

.page-header-icon {
	font-size: 3rem;
	margin-bottom: 1rem;
}

.page-header-icon-with-cover {
	margin-top: -0.72em;
	margin-left: 0.07em;
}

.page-header-icon img {
	border-radius: 3px;
}

.link-to-page {
	margin: 1em 0;
	padding: 0;
	border: none;
	font-weight: 500;
}

p > .user {
	opacity: 0.5;
}

td > .user,
td > time {
	white-space: nowrap;
}

input[type="checkbox"] {
	transform: scale(1.5);
	margin-right: 0.6em;
	vertical-align: middle;
}

p {
	margin-top: 0.5em;
	margin-bottom: 0.5em;
}

.image {
	border: none;
	margin: 1.5em 0;
	padding: 0;
	border-radius: 0;
	text-align: center;
}

.code,
code {
	background: rgba(135, 131, 120, 0.15);
	border-radius: 3px;
	padding: 0.2em 0.4em;
	border-radius: 3px;
	font-size: 85%;
	tab-size: 2;
}

code {
	color: #eb5757;
}

.code {
	padding: 1.5em 1em;
}

.code-wrap {
	white-space: pre-wrap;
	word-break: break-all;
}

.code > code {
	background: none;
	padding: 0;
	font-size: 100%;
	color: inherit;
}

blockquote {
	font-size: 1.25em;
	margin: 1em 0;
	padding-left: 1em;
	border-left: 3px solid rgb(55, 53, 47);
}

.bookmark {
	text-decoration: none;
	max-height: 8em;
	padding: 0;
	display: flex;
	width: 100%;
	align-items: stretch;
}

.bookmark-title {
	font-size: 0.85em;
	overflow: hidden;
	text-overflow: ellipsis;
	height: 1.75em;
	white-space: nowrap;
}

.bookmark-text {
	display: flex;
	flex-direction: column;
}

.bookmark-info {
	flex: 4 1 180px;
	padding: 12px 14px 14px;
	display: flex;
	flex-direction: column;
	justify-content: space-between;
}

.bookmark-image {
	width: 33%;
	flex: 1 1 180px;
	display: block;
	position: relative;
	object-fit: cover;
	border-radius: 1px;
}

.bookmark-description {
	color: rgba(55, 53, 47, 0.6);
	font-size: 0.75em;
	overflow: hidden;
	max-height: 4.5em;
	word-break: break-word;
}

.bookmark-href {
	font-size: 0.75em;
	margin-top: 0.25em;
}

.sans { font-family: ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol"; }
.code { font-family: "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace; }
.serif { font-family: Lyon-Text, Georgia, ui-serif, serif; }
.mono { font-family: iawriter-mono, Nitti, Menlo, Courier, monospace; }
.pdf .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK JP'; }
.pdf:lang(zh-CN) .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK SC'; }
.pdf:lang(zh-TW) .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK TC'; }
.pdf:lang(ko-KR) .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK KR'; }
.pdf .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK JP'; }
.pdf:lang(zh-CN) .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK SC'; }
.pdf:lang(zh-TW) .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK TC'; }
.pdf:lang(ko-KR) .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK KR'; }
.pdf .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK JP'; }
.pdf:lang(zh-CN) .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK SC'; }
.pdf:lang(zh-TW) .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK TC'; }
.pdf:lang(ko-KR) .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK KR'; }
.pdf .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK JP'; }
.pdf:lang(zh-CN) .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK SC'; }
.pdf:lang(zh-TW) .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK TC'; }
.pdf:lang(ko-KR) .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK KR'; }
.highlight-default {
	color: rgba(55, 53, 47, 1);
}
.highlight-gray {
	color: rgba(120, 119, 116, 1);
	fill: rgba(120, 119, 116, 1);
}
.highlight-brown {
	color: rgba(159, 107, 83, 1);
	fill: rgba(159, 107, 83, 1);
}
.highlight-orange {
	color: rgba(217, 115, 13, 1);
	fill: rgba(217, 115, 13, 1);
}
.highlight-yellow {
	color: rgba(203, 145, 47, 1);
	fill: rgba(203, 145, 47, 1);
}
.highlight-teal {
	color: rgba(68, 131, 97, 1);
	fill: rgba(68, 131, 97, 1);
}
.highlight-blue {
	color: rgba(51, 126, 169, 1);
	fill: rgba(51, 126, 169, 1);
}
.highlight-purple {
	color: rgba(144, 101, 176, 1);
	fill: rgba(144, 101, 176, 1);
}
.highlight-pink {
	color: rgba(193, 76, 138, 1);
	fill: rgba(193, 76, 138, 1);
}
.highlight-red {
	color: rgba(212, 76, 71, 1);
	fill: rgba(212, 76, 71, 1);
}
.highlight-gray_background {
	background: rgba(241, 241, 239, 1);
}
.highlight-brown_background {
	background: rgba(244, 238, 238, 1);
}
.highlight-orange_background {
	background: rgba(251, 236, 221, 1);
}
.highlight-yellow_background {
	background: rgba(251, 243, 219, 1);
}
.highlight-teal_background {
	background: rgba(237, 243, 236, 1);
}
.highlight-blue_background {
	background: rgba(231, 243, 248, 1);
}
.highlight-purple_background {
	background: rgba(244, 240, 247, 0.8);
}
.highlight-pink_background {
	background: rgba(249, 238, 243, 0.8);
}
.highlight-red_background {
	background: rgba(253, 235, 236, 1);
}
.block-color-default {
	color: inherit;
	fill: inherit;
}
.block-color-gray {
	color: rgba(120, 119, 116, 1);
	fill: rgba(120, 119, 116, 1);
}
.block-color-brown {
	color: rgba(159, 107, 83, 1);
	fill: rgba(159, 107, 83, 1);
}
.block-color-orange {
	color: rgba(217, 115, 13, 1);
	fill: rgba(217, 115, 13, 1);
}
.block-color-yellow {
	color: rgba(203, 145, 47, 1);
	fill: rgba(203, 145, 47, 1);
}
.block-color-teal {
	color: rgba(68, 131, 97, 1);
	fill: rgba(68, 131, 97, 1);
}
.block-color-blue {
	color: rgba(51, 126, 169, 1);
	fill: rgba(51, 126, 169, 1);
}
.block-color-purple {
	color: rgba(144, 101, 176, 1);
	fill: rgba(144, 101, 176, 1);
}
.block-color-pink {
	color: rgba(193, 76, 138, 1);
	fill: rgba(193, 76, 138, 1);
}
.block-color-red {
	color: rgba(212, 76, 71, 1);
	fill: rgba(212, 76, 71, 1);
}
.block-color-gray_background {
	background: rgba(241, 241, 239, 1);
}
.block-color-brown_background {
	background: rgba(244, 238, 238, 1);
}
.block-color-orange_background {
	background: rgba(251, 236, 221, 1);
}
.block-color-yellow_background {
	background: rgba(251, 243, 219, 1);
}
.block-color-teal_background {
	background: rgba(237, 243, 236, 1);
}
.block-color-blue_background {
	background: rgba(231, 243, 248, 1);
}
.block-color-purple_background {
	background: rgba(244, 240, 247, 0.8);
}
.block-color-pink_background {
	background: rgba(249, 238, 243, 0.8);
}
.block-color-red_background {
	background: rgba(253, 235, 236, 1);
}
.select-value-color-pink { background-color: rgba(245, 224, 233, 1); }
.select-value-color-purple { background-color: rgba(232, 222, 238, 1); }
.select-value-color-green { background-color: rgba(219, 237, 219, 1); }
.select-value-color-gray { background-color: rgba(227, 226, 224, 1); }
.select-value-color-translucentGray { background-color: rgba(255, 255, 255, 0.0375); }
.select-value-color-orange { background-color: rgba(250, 222, 201, 1); }
.select-value-color-brown { background-color: rgba(238, 224, 218, 1); }
.select-value-color-red { background-color: rgba(255, 226, 221, 1); }
.select-value-color-yellow { background-color: rgba(253, 236, 200, 1); }
.select-value-color-blue { background-color: rgba(211, 229, 239, 1); }

.checkbox {
	display: inline-flex;
	vertical-align: text-bottom;
	width: 16;
	height: 16;
	background-size: 16px;
	margin-left: 2px;
	margin-right: 5px;
}

.checkbox-on {
	background-image: url("data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2216%22%20height%3D%2216%22%20viewBox%3D%220%200%2016%2016%22%20fill%3D%22none%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%0A%3Crect%20width%3D%2216%22%20height%3D%2216%22%20fill%3D%22%2358A9D7%22%2F%3E%0A%3Cpath%20d%3D%22M6.71429%2012.2852L14%204.9995L12.7143%203.71436L6.71429%209.71378L3.28571%206.2831L2%207.57092L6.71429%2012.2852Z%22%20fill%3D%22white%22%2F%3E%0A%3C%2Fsvg%3E");
}

.checkbox-off {
	background-image: url("data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2216%22%20height%3D%2216%22%20viewBox%3D%220%200%2016%2016%22%20fill%3D%22none%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%0A%3Crect%20x%3D%220.75%22%20y%3D%220.75%22%20width%3D%2214.5%22%20height%3D%2214.5%22%20fill%3D%22white%22%20stroke%3D%22%2336352F%22%20stroke-width%3D%221.5%22%2F%3E%0A%3C%2Fsvg%3E");
}
	
</style></head><body><article id="9d4ab427-05ad-410f-bf07-4e586ba466ae" class="page sans"><header><h1 class="page-title">Draft genome of the European medicinal leech Hirudo medicinalis (Annelida, Clitellata, Hirudiniformes) with emphasis on anticoagulants | Scientific Reports</h1><p class="page-description"></p><table class="properties"><tbody><tr class="property-row property-row-multi_select"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesMultipleSelect"><path d="M1.91602 4.83789C2.44238 4.83789 2.87305 4.40723 2.87305 3.87402C2.87305 3.34766 2.44238 2.91699 1.91602 2.91699C1.38281 2.91699 0.952148 3.34766 0.952148 3.87402C0.952148 4.40723 1.38281 4.83789 1.91602 4.83789ZM5.1084 4.52344H14.3984C14.7607 4.52344 15.0479 4.23633 15.0479 3.87402C15.0479 3.51172 14.7607 3.22461 14.3984 3.22461H5.1084C4.74609 3.22461 4.45898 3.51172 4.45898 3.87402C4.45898 4.23633 4.74609 4.52344 5.1084 4.52344ZM1.91602 9.03516C2.44238 9.03516 2.87305 8.60449 2.87305 8.07129C2.87305 7.54492 2.44238 7.11426 1.91602 7.11426C1.38281 7.11426 0.952148 7.54492 0.952148 8.07129C0.952148 8.60449 1.38281 9.03516 1.91602 9.03516ZM5.1084 8.7207H14.3984C14.7607 8.7207 15.0479 8.43359 15.0479 8.07129C15.0479 7.70898 14.7607 7.42188 14.3984 7.42188H5.1084C4.74609 7.42188 4.45898 7.70898 4.45898 8.07129C4.45898 8.43359 4.74609 8.7207 5.1084 8.7207ZM1.91602 13.2324C2.44238 13.2324 2.87305 12.8018 2.87305 12.2686C2.87305 11.7422 2.44238 11.3115 1.91602 11.3115C1.38281 11.3115 0.952148 11.7422 0.952148 12.2686C0.952148 12.8018 1.38281 13.2324 1.91602 13.2324ZM5.1084 12.918H14.3984C14.7607 12.918 15.0479 12.6309 15.0479 12.2686C15.0479 11.9062 14.7607 11.6191 14.3984 11.6191H5.1084C4.74609 11.6191 4.45898 11.9062 4.45898 12.2686C4.45898 12.6309 4.74609 12.918 5.1084 12.918Z"></path></svg></span>Journal/Publisher</th><td></td></tr><tr class="property-row property-row-number"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesNumber"><path d="M2.4834 10.9902H4.33594L3.75488 13.8887C3.74121 13.9639 3.72754 14.0664 3.72754 14.1416C3.72754 14.5381 3.99414 14.7637 4.39746 14.7637C4.81445 14.7637 5.09473 14.5449 5.18359 14.1143L5.80566 10.9902H8.79297L8.21191 13.8887C8.19824 13.9639 8.18457 14.0664 8.18457 14.1416C8.18457 14.5381 8.45117 14.7637 8.85449 14.7637C9.27148 14.7637 9.55176 14.5449 9.63379 14.1143L10.2627 10.9902H12.4502C12.9287 10.9902 13.2432 10.6758 13.2432 10.2109C13.2432 9.8418 12.9902 9.56836 12.6006 9.56836H10.5498L11.2129 6.28711H13.3662C13.8379 6.28711 14.1523 5.96582 14.1523 5.50098C14.1523 5.13184 13.9062 4.8584 13.5098 4.8584H11.5L12.0195 2.27441C12.0264 2.21973 12.0469 2.11035 12.0469 2.01465C12.0469 1.625 11.7666 1.39258 11.3633 1.39258C10.9053 1.39258 10.6797 1.63867 10.5977 2.05566L10.0303 4.8584H7.04297L7.5625 2.27441C7.57617 2.21973 7.58984 2.11035 7.58984 2.01465C7.58984 1.625 7.30957 1.39258 6.91309 1.39258C6.44824 1.39258 6.21582 1.63867 6.13379 2.05566L5.57324 4.8584H3.54297C3.06445 4.8584 2.75 5.18652 2.75 5.65137C2.75 6.03418 3.00293 6.28711 3.39258 6.28711H5.28613L4.62305 9.56836H2.63379C2.15527 9.56836 1.84082 9.89648 1.84082 10.3613C1.84082 10.7373 2.09375 10.9902 2.4834 10.9902ZM6.09277 9.56836L6.75586 6.28711H9.74316L9.08008 9.56836H6.09277Z"></path></svg></span>Year</th><td></td></tr><tr class="property-row property-row-text"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesText"><path d="M1.56738 3.25879H14.4258C14.7676 3.25879 15.0479 2.97852 15.0479 2.63672C15.0479 2.29492 14.7744 2.02148 14.4258 2.02148H1.56738C1.21875 2.02148 0.952148 2.29492 0.952148 2.63672C0.952148 2.97852 1.22559 3.25879 1.56738 3.25879ZM1.56738 6.84082H14.4258C14.7676 6.84082 15.0479 6.56055 15.0479 6.21875C15.0479 5.87695 14.7744 5.60352 14.4258 5.60352H1.56738C1.21875 5.60352 0.952148 5.87695 0.952148 6.21875C0.952148 6.56055 1.22559 6.84082 1.56738 6.84082ZM1.56738 10.4229H14.4258C14.7676 10.4229 15.0479 10.1426 15.0479 9.80078C15.0479 9.45898 14.7744 9.18555 14.4258 9.18555H1.56738C1.21875 9.18555 0.952148 9.45898 0.952148 9.80078C0.952148 10.1426 1.22559 10.4229 1.56738 10.4229ZM1.56738 14.0049H8.75879C9.10059 14.0049 9.38086 13.7246 9.38086 13.3828C9.38086 13.041 9.10742 12.7676 8.75879 12.7676H1.56738C1.21875 12.7676 0.952148 13.041 0.952148 13.3828C0.952148 13.7246 1.22559 14.0049 1.56738 14.0049Z"></path></svg></span>Authors</th><td></td></tr><tr class="property-row property-row-url"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesUrl"><path d="M7.69922 10.8945L8.73828 9.84863C7.91797 9.77344 7.34375 9.51367 6.91992 9.08984C5.76465 7.93457 5.76465 6.29395 6.91309 5.14551L9.18262 2.87598C10.3379 1.7207 11.9717 1.7207 13.127 2.87598C14.2891 4.04492 14.2822 5.67188 13.1338 6.82031L11.958 7.99609C12.1768 8.49512 12.2451 9.10352 12.1289 9.62988L14.0908 7.6748C15.7725 6 15.7793 3.62109 14.084 1.92578C12.3887 0.223633 10.0098 0.237305 8.33496 1.91211L5.95605 4.29785C4.28125 5.97266 4.26758 8.35156 5.96289 10.0469C6.36621 10.4434 6.90625 10.7441 7.69922 10.8945ZM8.30078 5.13184L7.26855 6.17773C8.08203 6.25293 8.66309 6.51953 9.08008 6.93652C10.2422 8.09863 10.2422 9.73242 9.08691 10.8809L6.81738 13.1504C5.66211 14.3057 4.03516 14.3057 2.87305 13.1504C1.71094 11.9883 1.71777 10.3545 2.87305 9.20605L4.04199 8.03027C3.83008 7.53125 3.75488 6.92969 3.87109 6.39648L1.91602 8.35156C0.234375 10.0264 0.227539 12.4121 1.92285 14.1074C3.61816 15.8027 5.99707 15.7891 7.67188 14.1143L10.0439 11.7354C11.7256 10.0537 11.7324 7.6748 10.0371 5.98633C9.64062 5.58301 9.10059 5.28223 8.30078 5.13184Z"></path></svg></span>Link</th><td><a href="https://www.nature.com/articles/s41598-020-66749-5" class="url-value">https://www.nature.com/articles/s41598-020-66749-5</a></td></tr><tr class="property-row property-row-url"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesUrl"><path d="M7.69922 10.8945L8.73828 9.84863C7.91797 9.77344 7.34375 9.51367 6.91992 9.08984C5.76465 7.93457 5.76465 6.29395 6.91309 5.14551L9.18262 2.87598C10.3379 1.7207 11.9717 1.7207 13.127 2.87598C14.2891 4.04492 14.2822 5.67188 13.1338 6.82031L11.958 7.99609C12.1768 8.49512 12.2451 9.10352 12.1289 9.62988L14.0908 7.6748C15.7725 6 15.7793 3.62109 14.084 1.92578C12.3887 0.223633 10.0098 0.237305 8.33496 1.91211L5.95605 4.29785C4.28125 5.97266 4.26758 8.35156 5.96289 10.0469C6.36621 10.4434 6.90625 10.7441 7.69922 10.8945ZM8.30078 5.13184L7.26855 6.17773C8.08203 6.25293 8.66309 6.51953 9.08008 6.93652C10.2422 8.09863 10.2422 9.73242 9.08691 10.8809L6.81738 13.1504C5.66211 14.3057 4.03516 14.3057 2.87305 13.1504C1.71094 11.9883 1.71777 10.3545 2.87305 9.20605L4.04199 8.03027C3.83008 7.53125 3.75488 6.92969 3.87109 6.39648L1.91602 8.35156C0.234375 10.0264 0.227539 12.4121 1.92285 14.1074C3.61816 15.8027 5.99707 15.7891 7.67188 14.1143L10.0439 11.7354C11.7256 10.0537 11.7324 7.6748 10.0371 5.98633C9.64062 5.58301 9.10059 5.28223 8.30078 5.13184Z"></path></svg></span>Dataset</th><td></td></tr><tr class="property-row property-row-created_time"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesCreatedAt"><path d="M8 15.126C11.8623 15.126 15.0615 11.9336 15.0615 8.06445C15.0615 4.20215 11.8623 1.00293 7.99316 1.00293C4.13086 1.00293 0.938477 4.20215 0.938477 8.06445C0.938477 11.9336 4.1377 15.126 8 15.126ZM8 13.7383C4.85547 13.7383 2.33301 11.209 2.33301 8.06445C2.33301 4.91992 4.84863 2.39746 7.99316 2.39746C11.1377 2.39746 13.6738 4.91992 13.6738 8.06445C13.6738 11.209 11.1445 13.7383 8 13.7383ZM4.54102 8.91211H7.99316C8.30078 8.91211 8.54004 8.67285 8.54004 8.37207V3.8877C8.54004 3.58691 8.30078 3.34766 7.99316 3.34766C7.69238 3.34766 7.45312 3.58691 7.45312 3.8877V7.83203H4.54102C4.2334 7.83203 4.00098 8.06445 4.00098 8.37207C4.00098 8.67285 4.2334 8.91211 4.54102 8.91211Z"></path></svg></span>Log Date</th><td><time>@January 24, 2021 6:19 PM</time></td></tr><tr class="property-row property-row-multi_select"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesMultipleSelect"><path d="M1.91602 4.83789C2.44238 4.83789 2.87305 4.40723 2.87305 3.87402C2.87305 3.34766 2.44238 2.91699 1.91602 2.91699C1.38281 2.91699 0.952148 3.34766 0.952148 3.87402C0.952148 4.40723 1.38281 4.83789 1.91602 4.83789ZM5.1084 4.52344H14.3984C14.7607 4.52344 15.0479 4.23633 15.0479 3.87402C15.0479 3.51172 14.7607 3.22461 14.3984 3.22461H5.1084C4.74609 3.22461 4.45898 3.51172 4.45898 3.87402C4.45898 4.23633 4.74609 4.52344 5.1084 4.52344ZM1.91602 9.03516C2.44238 9.03516 2.87305 8.60449 2.87305 8.07129C2.87305 7.54492 2.44238 7.11426 1.91602 7.11426C1.38281 7.11426 0.952148 7.54492 0.952148 8.07129C0.952148 8.60449 1.38281 9.03516 1.91602 9.03516ZM5.1084 8.7207H14.3984C14.7607 8.7207 15.0479 8.43359 15.0479 8.07129C15.0479 7.70898 14.7607 7.42188 14.3984 7.42188H5.1084C4.74609 7.42188 4.45898 7.70898 4.45898 8.07129C4.45898 8.43359 4.74609 8.7207 5.1084 8.7207ZM1.91602 13.2324C2.44238 13.2324 2.87305 12.8018 2.87305 12.2686C2.87305 11.7422 2.44238 11.3115 1.91602 11.3115C1.38281 11.3115 0.952148 11.7422 0.952148 12.2686C0.952148 12.8018 1.38281 13.2324 1.91602 13.2324ZM5.1084 12.918H14.3984C14.7607 12.918 15.0479 12.6309 15.0479 12.2686C15.0479 11.9062 14.7607 11.6191 14.3984 11.6191H5.1084C4.74609 11.6191 4.45898 11.9062 4.45898 12.2686C4.45898 12.6309 4.74609 12.918 5.1084 12.918Z"></path></svg></span>Type</th><td></td></tr><tr class="property-row property-row-multi_select"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesMultipleSelect"><path d="M1.91602 4.83789C2.44238 4.83789 2.87305 4.40723 2.87305 3.87402C2.87305 3.34766 2.44238 2.91699 1.91602 2.91699C1.38281 2.91699 0.952148 3.34766 0.952148 3.87402C0.952148 4.40723 1.38281 4.83789 1.91602 4.83789ZM5.1084 4.52344H14.3984C14.7607 4.52344 15.0479 4.23633 15.0479 3.87402C15.0479 3.51172 14.7607 3.22461 14.3984 3.22461H5.1084C4.74609 3.22461 4.45898 3.51172 4.45898 3.87402C4.45898 4.23633 4.74609 4.52344 5.1084 4.52344ZM1.91602 9.03516C2.44238 9.03516 2.87305 8.60449 2.87305 8.07129C2.87305 7.54492 2.44238 7.11426 1.91602 7.11426C1.38281 7.11426 0.952148 7.54492 0.952148 8.07129C0.952148 8.60449 1.38281 9.03516 1.91602 9.03516ZM5.1084 8.7207H14.3984C14.7607 8.7207 15.0479 8.43359 15.0479 8.07129C15.0479 7.70898 14.7607 7.42188 14.3984 7.42188H5.1084C4.74609 7.42188 4.45898 7.70898 4.45898 8.07129C4.45898 8.43359 4.74609 8.7207 5.1084 8.7207ZM1.91602 13.2324C2.44238 13.2324 2.87305 12.8018 2.87305 12.2686C2.87305 11.7422 2.44238 11.3115 1.91602 11.3115C1.38281 11.3115 0.952148 11.7422 0.952148 12.2686C0.952148 12.8018 1.38281 13.2324 1.91602 13.2324ZM5.1084 12.918H14.3984C14.7607 12.918 15.0479 12.6309 15.0479 12.2686C15.0479 11.9062 14.7607 11.6191 14.3984 11.6191H5.1084C4.74609 11.6191 4.45898 11.9062 4.45898 12.2686C4.45898 12.6309 4.74609 12.918 5.1084 12.918Z"></path></svg></span>Technique</th><td></td></tr><tr class="property-row property-row-checkbox"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesCheckbox"><path d="M3.85742 14.4561H12.1357C13.6123 14.4561 14.3779 13.6904 14.3779 12.2344V3.91504C14.3779 2.45215 13.6123 1.69336 12.1357 1.69336H3.85742C2.38086 1.69336 1.61523 2.45215 1.61523 3.91504V12.2344C1.61523 13.6973 2.38086 14.4561 3.85742 14.4561ZM3.93945 13.1162C3.30371 13.1162 2.95508 12.7812 2.95508 12.1182V4.02441C2.95508 3.36133 3.30371 3.0332 3.93945 3.0332H12.0537C12.6826 3.0332 13.0381 3.36133 13.0381 4.02441V12.1182C13.0381 12.7812 12.6826 13.1162 12.0537 13.1162H3.93945ZM7.26855 11.3115C7.51465 11.3115 7.72656 11.1885 7.87012 10.9697L10.9258 6.19141C11.0146 6.04785 11.0967 5.88379 11.0967 5.72656C11.0967 5.3916 10.7959 5.16602 10.4746 5.16602C10.2695 5.16602 10.085 5.27539 9.94141 5.50781L7.24121 9.8418L5.96973 8.22168C5.80566 8.00977 5.6416 7.93457 5.43652 7.93457C5.10156 7.93457 4.8418 8.20117 4.8418 8.54297C4.8418 8.70703 4.90332 8.85742 5.01953 9.00098L6.63281 10.9697C6.81738 11.209 7.01562 11.3115 7.26855 11.3115Z"></path></svg></span>Dataset Raw</th><td><div class="checkbox checkbox-off"></div></td></tr></tbody></table></header><div class="page-body"><figure id="c07b03e3-493a-48ef-9eb6-ea74507df01d" class="image"><a href="41598_2020_66749_Fig1_HTML.png"><img style="width:299px" src="41598_2020_66749_Fig1_HTML.png"/></a></figure><h2 id="c81aaeed-24f2-469f-a5a3-a75760c954f8" class="">Abstract</h2><p id="66cb8c6d-a560-4113-8aa6-8af0817d9724" class="">The European medicinal leech has been used for medicinal purposes for millennia, and continues to be used today in modern hospital settings. Its utility is granted by the extremely potent anticoagulation factors that the leech secretes into the incision wound during feeding and, although a handful of studies have targeted certain anticoagulants, the full range of anticoagulation factors expressed by this species remains unknown. Here, we present the first draft genome of the European medicinal leech, Hirudo medicinalis, and estimate that we have sequenced between 79–94% of the full genome. Leveraging these data, we searched for anticoagulation factors across the genome of H. medicinalis. Following orthology determination through a series of BLAST searches, as well as phylogenetic analyses, we estimate that fully 15 different known anticoagulation factors are utilized by the species, and that 17 other proteins that have been linked to antihemostasis are also present in the genome. We underscore the utility of the draft genome for comparative studies of leeches and discuss our results in an evolutionary context.</p><h2 id="09b88530-9e92-40c4-83c1-2cc64699b8cc" class="">Introduction</h2><p id="992e3e9d-631b-4a70-82b8-bd76c43964e1" class="">Notwithstanding the significance of the European medicinal leech Hirudo medicinalis Linnaeus, 1758 as a medical tool from antiquity through modern science-based medicine, it is remarkable that this species’ repertoire of anticoagulants and other bioactive salivary proteins has escaped published in-depth investigation (but see<a href="https://www.nature.com/articles/s41598-020-66749-5">1</a>). In order to maintain blood flow in the tissues surrounding the incision wounds during their extended periods of feeding and, importantly, keeping the blood from clotting inside of the crop during the periods of digestion, bloodfeeding leeches secrete a variety of bioactive compounds from their salivary glands. Over 20 different compounds relating to antihemsotasis have been isolated from leech saliva including, for example, direct thrombin inhibitors, factor Xa inhibitors, trypsin inhibitors, endoglucuronidases and antiplatelet proteins<a href="https://www.nature.com/articles/s41598-020-66749-5">2</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">3</a>. Despite its historical significance in medicine, the detailed composition of the anticoagulant profile of Hirudo medicinalis Linnaeus, 1758 has escaped scrutiny insofar as modern studies have been concerned primarily with the commercially available species Hirudo verbana (Carena, 1820)<a href="https://www.nature.com/articles/s41598-020-66749-5">4</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">5</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">6</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">7</a>; but see also<a href="https://www.nature.com/articles/s41598-020-66749-5">1</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">8</a>. Whereas H. medicinalis was long thought to be the centerpiece for the clinical application of leeches in hospital settings, recent investigations have shown that Hirudo verbana is far more commonly employed for this purpose<a href="https://www.nature.com/articles/s41598-020-66749-5">4</a> yet sold under the name H. medicinalis; this confusion also prompted the designation of a neotype specimen for H. medicinalis in order to ameliorate future taxonomic confusion<a href="https://www.nature.com/articles/s41598-020-66749-5">9</a>. The utility of leeches in modern medicine focuses on post-surgical procedures, chiefly following digit replantation or skin grafting surgery<a href="https://www.nature.com/articles/s41598-020-66749-5">10</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">11</a> wherein the build-up of blood in the extremities and resulting congestion of the veins is relieved by their application. By contrast, medieval medicinal practices, the pinnacle of leech-centric phlebotomy, focused on restoring the balance between bodily humors to treat or prevent a variety of conditions<a href="https://www.nature.com/articles/s41598-020-66749-5">12</a>. The exceedingly frequent use of leeches caused overharvesting of several species, including Hirudo medicinalis<a href="https://www.nature.com/articles/s41598-020-66749-5">13</a>. Indeed, such was the level of harvesting in the 18th and 19th centuries that the collection of both H. medicinalis and H. verbana are still protected and governed by the Convention on International Trade in Endangered Species of Wild Flora and Fauna (CITES) (see<a href="https://www.nature.com/articles/s41598-020-66749-5">14</a> for an in-depth treatise of the species).</p><p id="ebc484aa-49f7-408f-be6c-f2c4f84e5e13" class="">Exploring the diversity of anticoagulant-related proteins and framing these in an evolutionary context in terms of presence or absence of orthologues across the phylogeny of leeches would help evince the general evolution of these proteins. Members of several leech families have been investigated at the transcriptome level for compositions of the anticoagulant repertoires, including members of Piscicolidae, Glossiphoniidae, Macrobdellidae, Haemadipsidae, Praobdellidae and Hirudinidae<a href="https://www.nature.com/articles/s41598-020-66749-5">3</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">8</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">15</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">16</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">17</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">18</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">19</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">20</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">21</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">22</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">23</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">24</a>. Genome-level evaluations within a species like H. medicinalis would allow insights into the diversity of bioactive proteins available. Exploring the copy number of anticoagulant proteins across a draft genome would also provide comparative data for future studies on putative duplication events, co-option of genes, and alternative splicing events. Beyond this, our draft genome should, when coupled with that of the non-bloodfeeding glossiphoniid Helobdella robusta (Shankland, Bissen &amp; Weisblat, 1992) (see<a href="https://www.nature.com/articles/s41598-020-66749-5">25</a>), aid a larger diversity of studies into leech evolution and allow for comparative analyses between leeches at the genome level.</p><h2 id="83531355-54eb-469a-8b94-72cd83dbcaaf" class="">Results</h2><h3 id="9744fa84-1af2-46a8-a929-88f7d9351395" class="">Assembly statistics and BUSCO analysis</h3><p id="e00f96a9-a5a9-4178-b1aa-7c604f98f11c" class="">The draft genome of Hirudo medicinalis ROMIZI 11733 was assembled to 19,929 scaffolds spanning 176.96 Mbps with a median coverage of 146.78×, an N50 score of 50,382 bps, and an L50 score of 772 (full statistics for the draft genome assembly can be found in Supplementary File 1). BUSCO ver. 4.0.5<a href="https://www.nature.com/articles/s41598-020-66749-5">26</a> was run to assess completeness of the genome. The analysis of the predicted proteins vs. the metazoa_odb10 resulted in a completeness score of 94.2% (90.0% complete + 4.2% fragmented).</p><p id="c7f491ba-122d-4aa4-b9d1-e7a8c8e9d3e5" class="">The assembly size represents 78.67% of the estimated genome size (230 Mbps; <a href="http://genomesize.com/">http://genomesize.com</a>). A two-pass annotation with MAKER<a href="https://www.nature.com/articles/s41598-020-66749-5">27</a> resulted in 35,166 predicted proteins with 780 splice variants. Using tRNAscan-SE<a href="https://www.nature.com/articles/s41598-020-66749-5">28</a>, a total of 429 tRNA genes were predicted, as well as an additional 116 pseudogenes. An additional 535 non-coding RNA genes/motifs were predicted using Infernal<a href="https://www.nature.com/articles/s41598-020-66749-5">29</a>, which included 64 rRNA genes and 316 microRNAs. The repetitive content of the genome was estimated at 24.71% (14.43% interspersed and 10.28% simple repeats) by RepeatModeler<a href="https://www.nature.com/articles/s41598-020-66749-5">30</a>, with the most abundant unit being unclassified (6.14%). All annotation files have been deposited at <a href="https://doi.org/10.5281/zenodo.3555585">https://doi.org/10.5281/zenodo.3555585</a> (last accessed January 20th 2020). The raw reads, as well as assembled sequences have been deposited in the European Nucleotide Archive (ENA) under the study accession PRJEB35865.</p><h3 id="ff5d3c7c-e96f-4cac-b598-66687628c2db" class="">Leech anticoagulants, copy number and tandem repeats</h3><p id="ccde2629-b78c-43ce-96a8-4da3794035d7" class="">In total, gene products were found in the Hirudo medicinalis genome that showed adequate BLASTp hits (superior to 1E−5) against 18 well-characterized leech-derived proteins with functions related to antihemostasis (Table <a href="https://www.nature.com/articles/s41598-020-66749-5">1</a>). These include eglin C, destabilase I, ghilanten, leech-derived tryptase inhibitor (LDTI), guamerin, cystatin, hirudin, hirudin-like factor 3, ficolin, Kazal-type serine protease inhibitors (serpins), C-type lectin, manillase, bdellin, piguamerin, antistasin, bdellastasin, lefaxin and an unidentified thrombininhibitor. Table <a href="https://www.nature.com/articles/s41598-020-66749-5">1</a> shows the top hits from the H. medicinalis genome, together with the hits against the three global databases, the copy number of the gene throughout the genome, and the presence or absence of a signal peptide.</p><p id="19ad4ef1-1842-4d9a-9273-a462941f98de" class=""><strong>Table 1 Known, leech-derived antihemostatis-related proteins with high scoring matches in the </strong><strong>Hirudo medicinalis</strong><strong> genome.</strong></p><p id="426e3eee-2610-4350-829e-e0e46811936f" class="">Seven out of the 18 putative anticoagulants occur in a single copy across our data (Table <a href="https://www.nature.com/articles/s41598-020-66749-5">1</a>); note that there is still a chance that more copies are present in the unsequenced parts of the genome. These are piguamerin, hirudin, ghilanten, hirudin-like factor 3, LDTI, the unidentified thrombin inhibitor and cystatin. The highest copy number (n = 7) was found for eglin C and destabilase; whereas copies for the former seemed scattered across the scaffolds (the seven copies occurred on six different scaffolds), the latter included three copies on the same scaffold and another scaffold with two copies.</p><p id="ea37f213-c4d8-48e5-9be1-59e2ada34d8a" class="">Whereas most of the anticoagulants targeted here are not positioned adjacent to each other in our draft genome, the following proteins seem to occur in tandem arrays as two or more copies: C-type lectin (with strong conservation of exon and intron sizes between the copies), guamerin (with only low conservation of exon and intron sizes between the copies), Kazal-type serine protease inhibitor (with only low conservation of exon and intron sizes between the copies) and bdellin (with exon sizes being relatively conserved, but intron sizes differing between the copies). Further, LDTI and fully three tandem copies of bdellin are adjacent to each other on scaffold 209471.</p><h3 id="d6832e5b-ec53-4ed9-9f61-45a31292a329" class="">Other bioactive peptides</h3><p id="60510fd7-7f82-4e2c-b27f-f84e500149e6" class="">In addition to the hits against known, leech-derived anticoagulation factors, fully 1,176 hits against 227 different bioactive compounds isolated from bloodfeeding organisms were recovered in the H. medicinalis genome; the function and pathways for most of these are remain unknown and, as such, we will only focus on the non-leech bioactive proteins that have been shown to be involved in anticoagulation. Robust hits (superior to 1E−5) were retrieved against 23 different proteins that negatively affect the coagulation cascade. These include a disintegrin and metalloproteinase with a thrombospondin motif (ADAMTS), apyrase, Kunitz-type serine protease inhibitor, fibrinogenase, chrysoptin, bothrojaracin, nitric oxide (vasodilator), agglucetin, snaclec, hemorrhagic metalloproteinase kaouthiagin, batroxstatin, thrombin inhibitor (from the Lone Star tick Amblyomma americanum), annexin, tabserin, thrombin inhibitor protein (from Rhodnius prolixus), snake venom serine protease, chymotrypsin, brasiliensin, cathepsin B, dipetalogastin, achelase, halyxin and antithrombin-III (from the King cobra Ophiophagus hannah). Supplementary File 2 shows the hits for these peptides, together with their reciprocal BLAST hits and signal peptide prediction. Several of the reciprocal BLAST hits were against unannotated (i.e., “hypothetical protein” or “uncharacterized protein”) genes in the Helobdella robusta genome, such that little information can be deduced regarding the identity of the matches. However, we also evaluated inferior hits (but still superior to 1E−5) against well-annotated genes in the three global databases. After evaluation of all available information, only the following protein products could not be robustly inferred to be present in the H. medicinalis genome (i.e., the remaining protein products are all present): chrysoptin, nitric oxide, thrombin inhibitor (from Amblyomma americanum), chymotrypsin, dipelogastin and achelase.</p><h3 id="fb4f8bbc-6a59-4aac-be6a-ab6a31aba203" class="">Sequence similarity and pairwise alignments</h3><p id="340f4b26-481d-49a0-aff8-c5fe3be4890c" class="">Each of the anticoagulation-related proteins derived from H. medicinalis were aligned with their archetypal counterpart, and the alignments are presented in Fig. <a href="https://www.nature.com/articles/s41598-020-66749-5">1</a> (for destabilase I, LDTI, hirudin and hirudin-like factor 3, and bdellin) and Supplementary File 3 (for eglin C, ghilanten, guamerin, cystatin, ficolin, the Kazal-type serpin, C-type lectin, manillase, piguamerin, antistasin, bdellastasin and the unidentified thrombininhibitor). Note that the H. medicinalis sequence with a hit against lefaxin found a far superior hit against hemerythrin when reciprocally BLASTed and was not further considered an orthologue of lefaxin.</p><p id="d087a48c-6cd0-4966-84e8-7357aa4a28e7" class=""><strong>Figure 1</strong></p><p id="ce950b4f-6e30-433b-b35b-6ff17f10db0d" class="">MAFFT-based amino acid alignments of putative anticoagulant orthologues derived from the genome of Hirudo medicinalis and the respective top BLASTp hits. (<strong>A</strong>) Putative destabilase I from H. medicinalis aligned with the known sequence of the salivary bioactive protein (GenBank accession number AAA96144); (<strong>B</strong>) putative Leech Derived Tryptase Inhibitor (LDTI) from H. medicinalis aligned with the known sequence of the salivary bioactive protein (GenBank accession number AAB33769); (<strong>C</strong>) putative hirudin (HV1) from H. medicinalis aligned with the known sequence of the salivary bioactive protein (GenBank accession number APA20833); (<strong>D</strong>) putative bdellin from H. medicinalis aligned with the known sequence of the salivary bioactive protein (GenBank accession number P09865). Red boxes denote conserved cysteine residues and blue shadings represent conservation of residues between the sequences.</p><p id="9e52e0aa-779a-49f1-989a-2c598bdf0340" class="">For destabilase I, the newly derived sequence shows 78% similarity in positions with shared amino acids (i.e., when gaps are not counted) with the known anticoagulant (GenBank accession number AAA96144<a href="https://www.nature.com/articles/s41598-020-66749-5">31</a>), which was also originally derived from Hirudo medicinalis. In addition, the positions of all 14 cysteine residues show full conservation, suggesting a similar folding structure between the proteins. No instances of indels were encountered in the alignment (Fig. <a href="https://www.nature.com/articles/s41598-020-66749-5">1A</a>).</p><p id="09c26094-44c2-4273-8142-796b5f0343e4" class="">In the amino acid alignment for LDTI, including the new sequence and the archetypal sequence derived from Hirudo medicinalis (GenBank accession number AAB33769<a href="https://www.nature.com/articles/s41598-020-66749-5">32</a>), the two sequences show 99% sequence similarity and full conservation of the positions of the six cysteine residues. No indels were encountered (Fig. <a href="https://www.nature.com/articles/s41598-020-66749-5">1B</a>).</p><p id="3d9e5b1f-f3f5-4799-a9b4-bb96d7908518" class="">The newly sequenced hirudin orthologue shows almost full conservation (sequence similarity 99.9%) when compared to the archetypal sequence (GenBank accession number APA20833<a href="https://www.nature.com/articles/s41598-020-66749-5">33</a>), originally derived from Hirudo verbana. In addition, positions of the six cysteine residues present in the mature peptide are fully conserved (two cysteines are also conserved in the signal peptide region). No indel events were present in the alignment (Fig. <a href="https://www.nature.com/articles/s41598-020-66749-5">1C</a>).</p><p id="1910847b-d4a1-4df9-bbd8-3c70b73720f2" class="">The bdellin sequence recovered from the H. medicinalis genome shows 99.2% sequence similarity with the archetypal sequence (GenBank accession number P09865<a href="https://www.nature.com/articles/s41598-020-66749-5">34</a>) originally derived from H. medicinalis, and the positions of all cysteine residues (n = 6) are fully conserved. No indels were present in the alignment (Fig. <a href="https://www.nature.com/articles/s41598-020-66749-5">1D</a>).</p><p id="3dee1820-c34a-4feb-8d77-ddc09baf9485" class="">For eglin C, the new sequence shows 99.9% sequence similarity when compared to the archetypal sequence (GenBank accession number 0905140 A<a href="https://www.nature.com/articles/s41598-020-66749-5">35</a>). No cysteine residues are present in either sequence and no indels were present (Supplementary File 3A).</p><p id="6dead314-2b74-4fe7-876e-cc74126faf0b" class="">The newly acquired ghilanten sequence shows only 32% sequence similarity when compared to the archetypal sequence (GenBank accession number AAB21233<a href="https://www.nature.com/articles/s41598-020-66749-5">36</a>) derived from the glossiphoniid leech Haementeria ghilianii (de Filippi, 1849). The low affinity between the sequences suggests that these may not be orthologous sequences. Regardless, the new sequence includes 25 cysteine residues in the mature protein and the positions of 17 of these are conserved in the alignment. Indel events were present in both sequences, the largest of which covers 25 residues (insertion in the new sequence or deletion in the archetypal sequence), and only short strings (maximum of n = 3) of homopolymers were identified (Supplementary File 3B).</p><p id="c6d65c07-b3e9-4a95-bbcb-98e3a94883e7" class="">For guamerin, the new sequence shows 67% sequence similarity with the archetypal protein (GenBank accession number AAD09442<a href="https://www.nature.com/articles/s41598-020-66749-5">37</a>), which was originally derived from Hirudo nipponia Whitman, 1886. The positions of all nine cysteine residues present in the alignment are fully conserved. No indel events were encountered (Supplementary File 3C).</p><p id="f01bc87d-f54b-4248-bdf0-47f38a84c4e4" class="">The new sequence for cystatin shows 57% similarity when compared to its archetypal counterpart (GenBank accession number AAN28679<a href="https://www.nature.com/articles/s41598-020-66749-5">38</a>), from the glossiphoniid Theromyzon tessulatum (Müller, 1774); the position of the single cysteine residue is conserved between the sequences. No indel events were present in the alignment (Supplementary File 3D).</p><p id="363aa8e5-f314-4df2-b8f6-672355cc1c7c" class="">For ficolin, sequence similarity is 59% between shared amino acid positions between our new sequence and the archetypal sequence (leveraged from the dataset used by Min et al.<a href="https://www.nature.com/articles/s41598-020-66749-5">15</a>), derived from the North American medicinal leech Macrobdella decora (Say, 1824). Two out of the three cysteine residues present in the M. decora-derived sequence are also present in the same position in the new sequence. A rather extensive insertion is present in the new sequence (or, alternatively, a deletion event in the archetypal sequence) and covers 27 amino acids residues (Supplementary File 3E).</p><p id="5a43e7d7-e36d-4623-b6c7-05e6c1531a21" class="">The newly derived Kazal-type serpin shows only 26% sequence similarity for shared amino acid sites when compared to the Macrobdella decora-derived sequence (from the dataset used by Min et al.<a href="https://www.nature.com/articles/s41598-020-66749-5">15</a>). Out of the 13 cysteine residues present in the “archetypal” sequence, 12 show conserved positions in the new sequence. Short indels are present in both sequences (Supplementary File 3F).</p><p id="9d577b2f-2dd4-41f3-9d64-7ef4023a2411" class="">The C-type lectin alignment indicates that 43% sequence similarity exists between the new sequence and the archetypal comparison derived from M. decora (see<a href="https://www.nature.com/articles/s41598-020-66749-5">15</a>). Thirteen cysteine residues exist in the archetypal sequence and the positions for nine of these are conserved in the newly acquired sequence. Three isolated, short deletions are present in the archetypal sequence (or, alternatively, these represent insertions in the new sequence) (Supplementary File 3G).</p><p id="77a48f79-6640-4484-ab94-e77d34f3f547" class="">For manillase, 83% of the shared amino acid residues are identical between the new sequence and that derived from a US patent application (no. 2006 US 7.049.124 B1P09856) and extracted from the Asian medicinal leech Hirudinaria manillensis Lesson, 1842. Notoriously, manillase is completely devoid of cysteine residues, and so is the sequence derived from Hirudo medicinalis. An insertion/deletion is present in the middle of the alignment and spans 16 residues (Supplementary File 3H).</p><p id="df19d3ed-81fa-49d4-b693-1e2beec41716" class="">For the trypsin inhibitor piguamerin, our newly sequenced gene product shows 46% similarity with the archetypal sequence (GenBank accession number P81499<a href="https://www.nature.com/articles/s41598-020-66749-5">39</a>), originally derived from Hirudo nipponia. The archetypal sequence includes ten cysteine residues and six of these are in conserved positions in the new sequence. No indels are present in the alignment (Supplementary File 3I).</p><p id="805849db-af2f-428d-98c1-d7ee9651f4a9" class="">For antistasin, the sequence derived from our specimen of H. medicinalis shows 36% similarity with the archetypal sequence (GenBank accession number P15358<a href="https://www.nature.com/articles/s41598-020-66749-5">40</a>) from Haementeria officinalis. In addition, the position of 18 out of the 21 cysteines present in the archetypal sequence are conserved between the sequences (Supplementary File 3J).</p><p id="e0e587b8-d814-4e5a-ae14-f27591e37434" class="">Our newly derived sequence in the bdellastasin alignment shows almost full conservation (99.9% similarity at shared amino acid sites) when compared to the archetypal variant (GenBank accession number 1C9P<a href="https://www.nature.com/articles/s41598-020-66749-5">41</a>), also from Hirudo medicinalis. The positions of the 10 cysteines are fully conserved between the sequences (Supplementary File 3K).</p><p id="0522a371-81be-46d3-9402-77a8934ba653" class="">The unknown thrombin inhibitor that here serves as the archetypal anticoagulant was originally derived from the piscicolid Pontobdella macrothela (Schmarda, 1861) (see<a href="https://www.nature.com/articles/s41598-020-66749-5">16</a>) following BLAST-based hits against a putative thrombin inhibitor from the haemadipsid leech Haemadipsa sylvestris Blanchard, 1894. Our Hirudo medicinalis-derived sequence shows only 28% sequence similarity for shared amino acid sites, yet the positions for seven out of the eight cysteine residues present in the target sequence are fully conserved in the newly acquired sequence. Interestingly, a large insertion of 27 amino acids is present in the middle of the new sequence; alternatively, this is a deletion in the archetypal sequence (Supplementary File 3L).</p><h3 id="541d0de8-d658-47a3-a8a6-8756868ce59e" class="">Gene trees</h3><p id="171d529e-464a-4583-8500-4b30f655fb30" class="">For each of the 16 leech-derived putative anticoagulants, we describe the unrooted tree topologies using the terminology proposed by Wilkinson et al.<a href="https://www.nature.com/articles/s41598-020-66749-5">42</a>, in which a “clan” in an unrooted tree is potentially equivalent to a monophyletic group in a rooted tree and “adjacent group” is equivalent to sister group.</p><p id="91e42a4e-a39e-43f0-bea2-2660ed9d00ed" class="">In the gene tree for destabilase I (Fig. <a href="https://www.nature.com/articles/s41598-020-66749-5">2A</a>), the newly acquired sequence forms a clan, albeit with rather low support (likelihood bootstrap support [LBS] = 72%), with the archetypal sequence and several variants retrieved from previous sequencing efforts for Hirudo medicinalis<a href="https://www.nature.com/articles/s41598-020-66749-5">31</a>.</p><p id="b36dcd68-64b2-4730-9726-784f7747ddb1" class=""><strong>Figure 2</strong></p><p id="a0750d0a-7919-4c86-a424-4829cf6229a9" class="">Phylogenetic hypotheses resulting from maximum likelihood analyses of a set of putative orthologues for each anticoagulant or anticoagulant family. (<strong>A</strong>) Destabilase I (ln L = −3340.015305); (<strong>B</strong>) LDTI (ln L = −640.341632). Green shades indicate the smallest clan that includes both the newly derived sequence and the archetypal variant of the anticoagulant.</p><p id="c3848b8e-d541-42a8-b830-1b1570728a82" class="">For LDTI (Fig. <a href="https://www.nature.com/articles/s41598-020-66749-5">2B</a>), the new sequence forms a clan (LBS = 100%) with all of the archetypal variants of the anticoagulant – note that branch lengths are very short or zero within this cluster of sequences, supporting the notion of orthology between them.</p><p id="02b25e76-bc40-4fec-b7d2-deb3667d6857" class="">Our newly derived hirudin sequence also forms a clan with two archetypal variants of the thrombin inhibitor (LBS = 97%) (Fig. <a href="https://www.nature.com/articles/s41598-020-66749-5">3A</a>) derived from the hirudinid leech Poecilobdella viridis (Blanchard, 1864) and Hirudo verbana. Again, the branch lengths are negligible. Moreover, the tree corroborates the BLAST results for hirudin-like factor 3, inasmuch as our newly derived sequence from H. medicinalis forms a clan (LBS = 87%) with the archetypal sequence derived from Hirudo orientalis.</p><p id="c84e422f-328b-410a-8ab2-27538d48a237" class=""><strong>Figure 3</strong></p><p id="dd660062-f206-42bc-a4a2-3ec8b867d0d2" class="">Phylogenetic hypotheses resulting from maximum likelihood analyses of a set of putative orthologues for each anticoagulant or anticoagulant family. (<strong>A</strong>) Hirudin (ln L = −4750.252905); (<strong>B</strong>) bdellin (ln L = −1771.698797). Green shades indicate the smallest clan that includes both the newly derived sequence and the archetypal variant of the anticoagulant.</p><p id="c297c534-58d9-4941-bc5e-78cbc2be7244" class="">In the bdellin tree (Fig. <a href="https://www.nature.com/articles/s41598-020-66749-5">3B</a>), the new sequence forms a clan, albeit with low support, (LBS = 26%) with the archetypal sequence, as well as previously sequenced variants from Hirudo nipponia, Hirudo medicinalis and Macrobdella decora. The branch length between the new sequence and the archetypal sequence is very short, corroborating the similarity-based homology determination (see above).</p><p id="bdcc6fcc-73e0-4262-b765-b51c2bf83d48" class="">For eglin C, the new sequence forms a clan (LBS &lt; 75%) with the archetypal variant; note that the species-level identity of the leech from which the archetypal sequence was derived is unknown. In addition, the clan includes a sequence from a separate sequencing effort for Hirudo medicinalis (unpublished), and branch lengths within this clan are very short or zero (Supplementary File 4A).</p><p id="7088dcf2-2d39-415f-87f9-e6ab51bd7a79" class="">In the tree constructed from members of the antistasin-family of anticoagulants (Supplementary File 4B), our “ghilanten” sequence from H. medicinalis forms a clan (LBS &lt; 75%) with three other sequences from Heptacyclus cf. viridus, Placobdella kwetlumye and Pontobdella macrothela. This clan is the adjacent group to a clan that includes the archetypal sequence for therostasin, as well as several variants of this gene from various leech species. By contrast, the archetypal sequence for ghilanten forms a clan (with very short branch length) with the archetypal sequence of antistasin, in a far removed section of the unrooted tree. As such, the identity of the newly derived ghilanten sequence is still debatable, but it seems reasonable to suggest that it belongs to therostasin, rather than ghilanten. When compared to therostasin directly (data not shown), the new sequence shows 43%, which is an 11% improvement compared to the alignment with ghilanten (see above). Each of the remaining antistasin-family proteins in the H. medicinalis dataset form clans with their respective archetypal anticoagulant. For bdellastasin, piguamerin and guamerin, each of the newly derived sequences nest as the adjacent sequence to the archetypal variants. For antistasin, our sequence places in a larger clan, including sequences from several leech species, as well as the archetypal variants of both ghilanten and antistasin.</p><p id="a317faaf-2a27-40ca-a40c-c05f3e02da2f" class="">For cystatin, only three additional, comparative sequences were available as the basis for the matrix. Despite this paucity of data, the new sequence forms a clan (LBS &lt; 75%) with the archetypal sequence and the branch length is comparable to those for the other terminals (Supplementary File 4C).</p><p id="78e5a5a8-50ef-45ab-ad9c-e73f950e3d7e" class="">Whereas no archetypal, leech-derived sequence is available for ficolin, our new sequence forms a clan together with a variant previously derived (unpublished) from H. medicinalis (LBS = 100%), with zero branch length separating the sequences (Supplementary File 4D).</p><p id="72c1a938-900e-4dca-8433-f8a7499d289a" class="">The tree for Kazal-type serine protease inhibitors (Supplementary File 4E) is one of only a few ever constructed for leech sequences and this is manifest in the both the lack of leech-derived archetypal variants and the lack of comparative data (only four sequences make up the matrix). Our new sequence forms a clan (LBS = 51%) with a variant derived from Haemadipsa interrupta.</p><p id="34cb9a85-92f3-4b02-8397-a16939a86c15" class="">In the C-type lectin tree (Supplementary File 4F), the sequence for the top hit forms an unsupported clan (LBS &lt; 75%) with variants derived from the African medicinal leech Aliolimnatis fenestrata and Hirudo medicinalis. This clan, in turn, is the adjacent group of the archetypal sequence.</p><p id="2ae86656-1d11-4dda-88ce-6d1189e85b1a" class="">Corroborating the similarity-based orthology determination, our newly derived manillase sequence forms a clan with the archetypal sequence and two other sequences derived from the praobdellid Limnobdella mexicana and Haemadipsa interrupta. Given this placement and the length of the branch leading to our sequence, there is little doubt that it represents an ortholog of manillase (Supplementary File 4G).</p><p id="bbd3cf77-af03-451a-a4f1-4dcbcff72efe" class="">Taken together, the results from the BLAST, alignment and gene tree analyses suggest that each of the following leech-derived protein products are represented in the H. medicinalis genome: eglin C, destabilase I, ghilanten, leech-derived tryptase inhibitor (LDTI), guamerin, cystatin, hirudin, ficolin, Kazal-type serine protease inhibitors (serpins), C-type lectin, manillase, bdellin, piguamerin, antistasin, bdellastasin and an unidentified thrombininhibitor.</p><h2 id="5d75bf8f-cd6c-4762-bf37-3ea5581f467f" class="">Discussion</h2><p id="0fd91d12-dbbb-4074-8c7b-63231ac0dde6" class="">Our draft genome for Hirudo medicinalis and in-depth look at the anticoagulant repertoire in this species provides fodder for comparative studies of anticoagulants in medicinal leech species. High-throughout sequencing employing the 10X Chromium platform allowed us to sequence the genome to an estimated 79–94% completion and evinced 18 separate anticoagulant-related orthologues (or putative orthologues) with BLAST matches against eglin C, destabilase I, ghilanten, leech-derived tryptase inhibitor (LDTI), guamerin, cystatin, hirudin, ficolin, Kazal-type serine protease inhibitors (serpins), C-type lectin, manillase, bdellin, piguamerin, antistasin, bdellastasin and an unidentified thrombininhibitor (Table <a href="https://www.nature.com/articles/s41598-020-66749-5">1</a>). In addition, our searches found robust hits against 23 other anticoagulant-related proteins from non-leech taxa. Several of the sequences possessed signal peptide regions at the 5’-end, indicating their secretion from the unicellular salivary glands. Although BLAST-based orthology statements have been commonplace in leech anticoagulant studies for almost a decade, without complementary phylogenetic analyses of putative orthologues, the robustness of such statements can be questioned<a href="https://www.nature.com/articles/s41598-020-66749-5">20</a>. Following the phylogenetic analyses presented here, it seems likely that 15 of the 16 leech-derived proteins (note that neither lefaxin nor the unidentified thrombin inhibitor was placed in a phylogenetic context due to the uncertainty of their identity and, therefore, comparative data) are indeed orthologous with their top BLAST hits, and the same number for non-leech proteins was 17 out of 23. A large body of literature supports the notion that phylogenetic analyses are needed for accurate orthology and paralogy statements (e.g.<a href="https://www.nature.com/articles/s41598-020-66749-5">43</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">44</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">45</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">46</a>). Given that accurate orthology statements were only made possible via phylogenetic analyses, the present study underscores the importance of incorporating such analyses into any study where orthology is not determined a priori. However, the lack of appropriate outgroups for leech anticoagulants presents its own set of issues and certainly exacerbates rapid and accurate orthology determination. In this study, we assumed that H. medicinalis-derived proteins that formed a clan (equivalent to a monophyletic group in rooted phylogenies) with its archetypal variant (a protein with demonstrated anticoagulant capabilities) was indeed orthologous to that anticoagulant. The levels of potency in preventing blood clotting for each of our newly sequenced proteins remains to be investigated.</p><p id="d7a04999-5a26-4832-b67e-9bc918d8bcab" class="">When comparing the list of anticoagulants found in H. verbana<a href="https://www.nature.com/articles/s41598-020-66749-5">6</a> to that of the closely related H. medicinalis, surprisingly, only bdellin, piguamerin, antistasin, ghilanten, eglin C, manillase and C-type lectin overlap between the two species. The study by Kvist et al.<a href="https://www.nature.com/articles/s41598-020-66749-5">6</a> recovered also hirustasin, therostasin and a leukocyte elastase inhibitor, as well as an unnamed plasmin inhibitor. By contrast, that study did not recover hirudin, destabilase I, LDTI, guamerin, bdellastasin, cystatin, ficolin, Kazal-type serpins or the unnamed thrombin inhibitor that were all recovered from Hirudo medicinalis. Surely, part of this discrepancy should be attributed to differing sequencing platforms, sequencing depth and imperfect search algorithms but, nevertheless, the data generated here allow for the first comparison between anticoagulants of closely related species of medicinal leeches. At the very least, this sets the stage for in-depth sequencing of the Hirudo verbana salivary transcriptome. It should be noted that neither study managed to recover matches against leech antiplatelet proteins (LAPP/saratin), which are commonly encountered in other leech species (see<a href="https://www.nature.com/articles/s41598-020-66749-5">2</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">3</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">6</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">16</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">18</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">20</a>). Moreover, decorsin, which has so far only been recovered from the North American medicinal leech Macrobdella decora was not recovered in either of the studies, perpetuating the notion that this protein may be unique to that species.</p><p id="d444e234-bcb8-44f5-ac6c-3e45e74771ce" class="">The present study adds to the growing body of knowledge of leech anticoagulants and, more generally, the evolution of bloodfeeding in leeches. Relatively few members of Hirudinidae have so far been investigated, with only Hirudinaria manillensis and, now, Hirudo medicinalis having been queried at the genome level<a href="https://www.nature.com/articles/s41598-020-66749-5">47</a>. With the finding of presumably orthologous loci across the leech phylogeny – including antistasin family proteins, manillase and C-type lectin, for example – there is a growing body of evidence suggesting that these are present due to their possession by the common ancestor of all leeches. The present study adds to this and strengthens the case that the ancestral leech was indeed bloodfeeding, or at least possessed proteins that antagonize a normal coagulation cascade.</p><p id="82f3deb1-bdfb-4d1d-99d0-a1cfca09d6a6" class="">Moreover, whereas H. medicinalis is commonly employed in modern hospital settings to relieve venous congestion, the repertoire of secreted proteins that get injected into the patients has not before been fully known. The presence of hirudin is likely the factor that has afforded H. medicinalis its place in both historical and modern medical practices. The present study suggests that the potency of H. medicinalis saliva in counteracting the clotting cascade of blood is highly dependent on a larger cocktail of proteins that have several different targets in the cascade.</p><p id="bb1b1dd6-aca2-49fb-9037-7681e398ae76" class="">Whereas the present study focused on anticoagulant diversity and evolution, the creation of a draft genome holds the potential to ignite and inspire comparative studies of leech genomics. With only two genomes previously sequenced for leeches, our new data unlock the potential for investigations into areas that necessitate comparative data, such as associations between genetic variants and phenotypic expression, comparisons of genomic makeup between bloodfeeding and non-bloodfeeding organisms, differences in signaling pathways in the central nervous system, to name a few.</p><h2 id="66e73b3a-a78c-44ce-8563-d6e663f8c0eb" class="">Material and methods</h2><h3 id="0b415666-84a3-434b-ba15-b0ebf5a95c66" class="">Specimen collection and DNA extraction and sequencing</h3><p id="a2fb862d-4954-41ee-aa73-49d27aeb3220" class="">The single specimen of Hirudo medicinalis used in the present study- was collected in August 2016 from an unnamed pond close to the town of Kočevje in southeastern Slovenia. The leech was collected by wading into the water and netting swimming leeches. The specimen was identified in the field based on external morphology, following specialized literature<a href="https://www.nature.com/articles/s41598-020-66749-5">48</a>,<a href="https://www.nature.com/articles/s41598-020-66749-5">49</a> and later confirmed by dissections. The leech was relaxed by gradual addition of 95% ethanol to pond water and fixed in 95% ethanol; it was kept at 4 °C until sequencing to avoid DNA degradation by freeze-thawing procedures.</p><p id="54a72b41-ac17-4c30-bc12-644a4028bcc9" class="">DNA was extracted from full body tissue avoiding the integument- as this tissue has been shown to exacerbate library creation - and digestive tract. That is, the DNA isolate was derived from gonadal, nervous and muscular tissue in a single extract. The tissues were frozen with liquid nitrogen and macerated using a pestle in a RNase-free Eppendorf tube. DNA was extracted using proteinase K digestion overnight, followed by a standard phenol-chloroform extraction protocol. Genomic DNA fragments &gt;30 Kb were size selected using the HighPass protocol on the BluePippin (Sage Science). Sample indexing and partition barcoded libraries were prepared using the Chromium Genome Library and Gel Bead Kit (10X Genomics) according to manufacturer’s protocols. The Chromium Controller was used to combine a library of 10X Genome Gel Beads with high molecular weight template genomic DNA (0.625 ng), a master mix of enzymes and buffer, and partitioning oil to create droplets containing single gel beads and DNA. During the process, genomic DNA was partitioned across approximately 1 million 10X GEMs. An emulsion containing the GEM partitioned reactions was isothermally incubated (for 3 h at 30 °C; for 10 min at 65 °C; held at 4 °C), and barcoded fragments ranging from a few to several hundred base pairs were generated. After amplification, half of the emulsion was collected, and GEMs were broken. Finally, the recovered barcoded DNA was size selected for library preparation. Illumina-specific sample indexing was added to the barcoded fragments to generate libraries according to the manufacturer’s instructions. The barcode sequencing libraries were then quantified by qPCR (KAPA Biosystems Library Quantification Kit for Illumina platforms). Sequencing was conducted across half a lane on the Illumina HiSeq X platform at the New York Genome Center (New York, NY, USA) with 2 × 150 paired-end reads based on the manufacturer’s protocols.</p><h3 id="bfb0818e-4004-4d59-b6a9-ec9074cf0d14" class="">Genome assembly and annotation</h3><p id="01366134-32ab-4948-bff8-c2b83f9b99bb" class="">The resulting reads were used as input for Supernova ver. 2.0.1<a href="https://www.nature.com/articles/s41598-020-66749-5">50</a> and a pseudo-haploid representation of the assembly was generated using the subcommand mkoutput. The assembly resulted in 21,765 scaffolds, which were then taxonomically assigned using PhymmBL v4.0<a href="https://www.nature.com/articles/s41598-020-66749-5">51</a> in order to remove contaminants. The database for taxonomic assignment consisted of representative reference genomes for Bacteria and Archaea from NCBI, as well as the human reference genome, Saccharomyces cerevisiae S288C, and the annelids Capitella teleta Blake, Grassle &amp; Eckelbarger, 2009, Helobdella robusta, and a preliminary draft genome from the leech Macrobdella decora (Say, 1824) (reference genome in progress); the distribution of taxonomic assignments can be found in the Zenodo repository at https://doi.org/10.5281/zenodo.3555585. Two iterative rounds of taxonomic assignment were run, adding the scaffolds assigned to Annelida to the PhymmBL database after each round. A total of 1,836 scaffolds were removed from the H. medicinalis draft assembly.</p><p id="3c31e3e7-ba98-471f-a24a-e0fed40f799d" class="">The remaining 19,929 scaffolds underwent a two-step annotation in MAKER v2.31.10<a href="https://www.nature.com/articles/s41598-020-66749-5">27</a> using the transcriptomes of H. verbana (GGIQ00000000.1) and H. medicinalis (GBRF00000000.1) as EST sets and the predicted proteomes for Helobdella robusta and Capitella teleta as proteins sets (available from the JGI genome portal [<a href="https://genome.jgi.doe.gov/portal/">https://genome.jgi.doe.gov/portal/</a>, last accessed January 20th 2020]). Next, the resulting proteins were compared to the SWISS-PROT database (including splice variants) using BLASTp ver. 2.6.0 + <a href="https://www.nature.com/articles/s41598-020-66749-5">52</a>. Additionally, a similarity search was run using InterProScan ver. 5.28-67.0<a href="https://www.nature.com/articles/s41598-020-66749-5">53</a>. Gene models with an AED score of 1 were kept only if a significant hit was found. tRNAs were identified using tRNAscan-SE ver. 2.0.0<a href="https://www.nature.com/articles/s41598-020-66749-5">28</a> and ncRNAs and other RNA motifs were searched for using Infernal ver. 1.1.2<a href="https://www.nature.com/articles/s41598-020-66749-5">29</a>. Repeated content in the genome was identified using RepeatModeler ver. 1.0.11<a href="https://www.nature.com/articles/s41598-020-66749-5">30</a>.</p><p id="9e4e18f5-8ba2-4368-92cd-7496a6127879" class="">The full suite of predicted genes in the genome of H. medicinalis were BLASTed (using BLASTp) against a local database containing known leech-derived anticoagulants, as well as proteins derived from other bloodfeeding organisms. The top hit for each leech-derived anticoagulant was then reciprocally BLASTed against three separate databases: GenBank non-redundant protein database, SWISS-PROT and Pfam. This was done to ensure that the best hits against leech-derived anticoagulants did not find a better hit against another known protein in the global databases. The sequence was not considered an orthologue of the best hit in the leech-derived anticoagulant database if a better hit was found against a completely unrelated protein in either of the three databases. As well, hits against non-leech-derived proteins that have been shown to affect hemostasis were recovered and scrutinized through the same pipeline. SignalP ver. 5.0<a href="https://www.nature.com/articles/s41598-020-66749-5">54</a> was used to predict potential signal peptides. If not identified, Phobius ver. 1.01<a href="https://www.nature.com/articles/s41598-020-66749-5">55</a> was used as an alternative identifier of a potential signal peptide.</p><p id="670651b9-f829-404e-a553-693a6469294a" class="">The copy numbers across the genome for each of the hits against well-characterized, leech-derived anticoagulants were estimated by BLASTing the top hit for each protein against the genome. Hits below the cutoff value of 1E−5, with a query length of at least 50% of the target and with at least 70% sequence similarity were considered good hits and, therefore, a copy of the query sequence. Because several of the targeted anticoagulation factors share similar amino acid compositions, each of the hits were compared the remaining pool to ensure that hits were not counted twice as separate anticoagulation factors. As an initial check of the positioning of the anticoagulants across the genome, we also assessed whether or not hits are adjacent to each other on the same scaffold.</p><h3 id="8938f810-4be0-45cb-8a93-0dffc579e74f" class="">Alignments and gene trees</h3><p id="4cc485a6-78d0-4e3c-a85f-f8e692640b3d" class="">Following the initial BLAST-based homology determination, orthology was determined by sequence similarity and phylogenetic analyses. For each of the hits against well-characterized leech-derived anticoagulant proteins, the top hit was aligned with the amino acid sequence of the known (archetypal) anticoagulant. This was done both to visualize the level of conservation between the sequences and identify any insertions and/or deletions. For this purpose, the two sequences were aligned using the online version of MAFFT ver. 7<a href="https://www.nature.com/articles/s41598-020-66749-5">56</a> using default settings under the “Auto” strategy. JalView ver. 2<a href="https://www.nature.com/articles/s41598-020-66749-5">57</a> was used to visualize the alignments. To maximize regions of homology and to augment visualization of the alignments, leading and lagging gaps were truncated from the alignments, but internal gaps in the sequences (i.e., true indels) were maintained. Note that, in some cases, this resulted in the signal peptide region being excised from the sequences (these are still reported in Table <a href="https://www.nature.com/articles/s41598-020-66749-5">1</a>). From the alignments, percent amino acid conservation, with special emphasis on the conservation of disulphide-bond forming cysteines, was calculated by eye.</p><p id="83b6f490-87b4-47b9-b189-000c7383db1e" class="">Phylogenetic analyses were used to test for corroboration of the BLAST-based orthology determinations. Datasets for each of the hits against well-characterized anticoagulant proteins were constructed using the top BLASTp hit for the protein in our newly acquired dataset, amended by comparative data from the datasets used by Iwama et al.<a href="https://www.nature.com/articles/s41598-020-66749-5">20</a> and Khan et al.<a href="https://www.nature.com/articles/s41598-020-66749-5">58</a> for each protein. Sequences were again aligned using MAFFT ver. 7 with the same settings as above. The alignments were then subjected to maximum likelihood analyses using RAxML ver. 8.2.12<a href="https://www.nature.com/articles/s41598-020-66749-5">59</a> on the CIPRES Science Gateway Platform<a href="https://www.nature.com/articles/s41598-020-66749-5">60</a>. The searches were conducted under a general time reversible model (GTR) and rate heterogeneity was modelled by a gamma distribution using 25 rate categories and an estimated proportion of invariable sites. Support values for nodes were estimated through 1,000 standard bootstrap replicates using default settings. Given the paucity of obvious choices for outgroups for leech anticoagulants, the trees were left unrooted. All alignments, tree files and annotation files have been deposited in Zenodo (<a href="https://doi.org/10.5281/zenodo.3555585">https://doi.org/10.5281/zenodo.3555585</a>) (last accessed April 6th 2020).</p><h2 id="0e7abe89-3659-4fc2-a44c-c7615e953ace" class="">References</h2><ol type="1" id="078b60b3-d998-425c-ac8b-0d407ef309b7" class="numbered-list" start="1"><li>Babenko, V. V. et al. Draft genome sequences of Hirudo medicinalis and salivary transcriptome of three closely related medicinal leeches. bioRxiv. <strong>p</strong>.<strong>357681</strong> (2019).</li></ol><ol type="1" id="7c66d3a6-e746-4055-a439-7ec1e769a8dc" class="numbered-list" start="2"><li>Kvist, S., Brugler, M. R., Goh, T. G., Giribet, G. &amp; Siddall, M. E. Pyrosequencing the salivary transcriptome of Haemadipsa interrupta (Annelida: Clitellata: Haemadipsidae): anticoagulant diversity and insight into the evolution of anticoagulation capabilities in leeches. Inv. Biol. <strong>133</strong>(1), 74–98 (2014).</li></ol><ol type="1" id="dec54f74-fe10-4511-852a-19ccd8fa3775" class="numbered-list" start="3"><li>Smit, A. F. A., Hubley, R. &amp; Green, P. RepeatModeler Open-1.0. 2008–2015. Seattle, USA: Institute for Systems Biology. Available from, <a href="http://www.repeatmasker.org/">http://www.repeatmasker.org</a> (2015).</li></ol><ol type="1" id="71660cd9-88f6-4d67-a2d2-2814bcf77e77" class="numbered-list" start="4"><li>Zavalova, L. et al. Genes from the medicinal leech (Hirudo medicinalis) coding for unusual enzymes that specifically cleave endo-ɛ (γ-Glu)-Lys isopeptide bonds and help to dissolve blood clots. Mol. Gen. Genet. <strong>253</strong>(1–2), 20–25 (1996).</li></ol><ol type="1" id="a20fb272-cd73-4c53-b86c-fa2569f645f4" class="numbered-list" start="5"><li>Brankamp, R. G., Manley, G. G., Blankenship, D. T., Bowlin, T. L. &amp; Cardin, A. D. Studies on the anticoagulant, antimetastatic and heparin-binding properties of ghilanten-related inhibitors. Blood Coagul. Fibrin. <strong>2</strong>(1), 161–166 (1991).</li></ol><ol type="1" id="e41f3941-ee96-437e-a945-50a18a07b380" class="numbered-list" start="6"><li>Jung, H. I., Kim, S. I., Ha, K.-S., Joe, C. O. &amp; Kang, K. W. Isolation and characterization of guamerin, a new human leukocyte elastase inhibitor from Hirudo nipponia. J. Biol. Chem. <strong>270</strong>(23), 13879–13884 (1995).</li></ol><ol type="1" id="e7c66e69-58cd-464a-ae34-e12a17f37a05" class="numbered-list" start="7"><li>Moser, M. et al. Bdellastasin, a serine protease inhibitor of the antistasin family from the medical leech (Hirudo medicinalis) primary structure, expression in yeast, and characterisation of native and recombinant inhibitor. Eur. J. Biochem. <strong>253</strong>(1), 212–220 (1998).</li></ol><ol type="1" id="4f282db7-1f6f-4f66-8885-07c06b433a67" class="numbered-list" start="8"><li>Thorp, J. H. &amp; Rogers, C. Thorp and Covich’s freshwater invertebrates: Ecology and general biology, 4th ed., Volume I. Academic Press, London, U.K., 762 p. (2015).</li></ol><ol type="1" id="21cc32dc-02d4-4201-8eb5-17d0730ff1ee" class="numbered-list" start="9"><li>Miller, M. A., Pfeiffer, W. &amp; Schwartz, T. Creating the CIPRES Science Gateway for inference of large phylogenetic trees. In 2010 Gateway Computing Environments Workshop (GCE), 1–8 (2010).</li></ol><h2 id="887837ae-0652-4448-8408-5df2a18d0139" class="">Acknowledgements</h2><p id="07c4bde2-8a47-4851-8857-bab47a156773" class="">We thank Lara Winterkorn (New York Genome Center) for help with sequencing, read assembly and technical issues. We acknowledge the generous support of Judith and William Marth as well as Beth Ann and Rick Willet. This research was, in part, made possible with the generous support of the ROM Governors Invertebrate Zoology Fund, a NSERC Discovery Grant and an Olle Engkvist Byggmästare Foundation stipend to SK. AMM has received funding from the European Union’s Horizon 2020 research and innovation programme under the Marie Skłodowska-Curie grant agreement No. 840270. DdC was funded by an NSERC Postgraduate Scholarship (PGSD2-518435-2018). Insightful comments from the Editor and two anonymous reviewers greatly assisted a previous version of the manuscript.</p><h2 id="db7c6018-826c-4975-89f9-95c00e3ba2a3" class="">Author information</h2><ol type="1" id="0e0281e7-a026-4ba6-9e26-baaddc44f21c" class="numbered-list" start="1"><li>Department of Natural History, Royal Ontario Museum, 100 Queen’s Park, Toronto, ON, M5S 2C6, Canada</li></ol><ol type="1" id="502cf120-fb3e-4e0d-9780-7371df36b2ba" class="numbered-list" start="2"><li>Department of Ecology and Evolutionary Biology, University of Toronto, 25 Willcocks Street, Toronto, ON, M5S 2B4, Canada</li></ol><ol type="1" id="716c73d6-402a-4287-a5fa-43c891188314" class="numbered-list" start="3"><li>Division of invertebrate Zoology, American Museum of Natural History, 79th Street @ Central Park West, New York, NY, 10025, USA</li></ol><h3 id="ffa6bded-39c1-436c-ac82-afc532a37cfc" class="">Contributions</h3><p id="227ee166-c3e1-4c4d-a60d-1c67a0c59bba" class="">S.K., A.M.M., D.d.C., P.T. and M.E.S. designed the research. S.K., A.M.M., D.d.C. and M.E.S. analyzed the data. S.K., A.M.M., D.d.C., P.T. and M.E.S. wrote, reviewed and approved the final version of the manuscript.</p><h3 id="5809b38e-ba49-4751-be6b-e1cd7249048f" class="">Corresponding author</h3><p id="493601e7-c892-457a-85cb-833397118053" class="">Correspondence to <a href="https://www.nature.com/articles/s41598-020-66749-5/email/correspondent/c1/new">Sebastian Kvist</a>.</p><h2 id="11b495c0-4fcb-4a31-814b-547e3db2f1f3" class="">Supplementary information</h2><h2 id="2ea9f9a4-c6f5-4c63-9303-637c8f57afd1" class="">Rights and permissions</h2><p id="dc5a7909-2c10-4491-b613-d3fabf705831" class=""><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</p><h2 id="3d358491-5a80-4a8c-9cc2-f380c6ea3dbb" class="">About this article</h2><p id="d043e0c0-f3d4-4e6f-a402-839d43886b4f" class="">Kvist, S., Manzano-Marín, A., de Carle, D. et al. Draft genome of the European medicinal leech Hirudo medicinalis (Annelida, Clitellata, Hirudiniformes) with emphasis on anticoagulants. Sci Rep <strong>10, </strong>9885 (2020). https://doi.org/10.1038/s41598-020-66749-5</p><h2 id="e4c93f98-6960-453b-b2d5-19f5cf1a2464" class="">Further reading</h2></div></article></body></html>