<html><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><title>The Hirudo Medicinalis Microbiome Is a Source of New Antimicrobial Peptides</title><style>
/* cspell:disable-file */
/* webkit printing magic: print all background colors */
html {
	-webkit-print-color-adjust: exact;
}
* {
	box-sizing: border-box;
	-webkit-print-color-adjust: exact;
}

html,
body {
	margin: 0;
	padding: 0;
}
@media only screen {
	body {
		margin: 2em auto;
		max-width: 900px;
		color: rgb(55, 53, 47);
	}
}

body {
	line-height: 1.5;
	white-space: pre-wrap;
}

a,
a.visited {
	color: inherit;
	text-decoration: underline;
}

.pdf-relative-link-path {
	font-size: 80%;
	color: #444;
}

h1,
h2,
h3 {
	letter-spacing: -0.01em;
	line-height: 1.2;
	font-weight: 600;
	margin-bottom: 0;
}

.page-title {
	font-size: 2.5rem;
	font-weight: 700;
	margin-top: 0;
	margin-bottom: 0.75em;
}

h1 {
	font-size: 1.875rem;
	margin-top: 1.875rem;
}

h2 {
	font-size: 1.5rem;
	margin-top: 1.5rem;
}

h3 {
	font-size: 1.25rem;
	margin-top: 1.25rem;
}

.source {
	border: 1px solid #ddd;
	border-radius: 3px;
	padding: 1.5em;
	word-break: break-all;
}

.callout {
	border-radius: 3px;
	padding: 1rem;
}

figure {
	margin: 1.25em 0;
	page-break-inside: avoid;
}

figcaption {
	opacity: 0.5;
	font-size: 85%;
	margin-top: 0.5em;
}

mark {
	background-color: transparent;
}

.indented {
	padding-left: 1.5em;
}

hr {
	background: transparent;
	display: block;
	width: 100%;
	height: 1px;
	visibility: visible;
	border: none;
	border-bottom: 1px solid rgba(55, 53, 47, 0.09);
}

img {
	max-width: 100%;
}

@media only print {
	img {
		max-height: 100vh;
		object-fit: contain;
	}
}

@page {
	margin: 1in;
}

.collection-content {
	font-size: 0.875rem;
}

.column-list {
	display: flex;
	justify-content: space-between;
}

.column {
	padding: 0 1em;
}

.column:first-child {
	padding-left: 0;
}

.column:last-child {
	padding-right: 0;
}

.table_of_contents-item {
	display: block;
	font-size: 0.875rem;
	line-height: 1.3;
	padding: 0.125rem;
}

.table_of_contents-indent-1 {
	margin-left: 1.5rem;
}

.table_of_contents-indent-2 {
	margin-left: 3rem;
}

.table_of_contents-indent-3 {
	margin-left: 4.5rem;
}

.table_of_contents-link {
	text-decoration: none;
	opacity: 0.7;
	border-bottom: 1px solid rgba(55, 53, 47, 0.18);
}

table,
th,
td {
	border: 1px solid rgba(55, 53, 47, 0.09);
	border-collapse: collapse;
}

table {
	border-left: none;
	border-right: none;
}

th,
td {
	font-weight: normal;
	padding: 0.25em 0.5em;
	line-height: 1.5;
	min-height: 1.5em;
	text-align: left;
}

th {
	color: rgba(55, 53, 47, 0.6);
}

ol,
ul {
	margin: 0;
	margin-block-start: 0.6em;
	margin-block-end: 0.6em;
}

li > ol:first-child,
li > ul:first-child {
	margin-block-start: 0.6em;
}

ul > li {
	list-style: disc;
}

ul.to-do-list {
	text-indent: -1.7em;
}

ul.to-do-list > li {
	list-style: none;
}

.to-do-children-checked {
	text-decoration: line-through;
	opacity: 0.375;
}

ul.toggle > li {
	list-style: none;
}

ul {
	padding-inline-start: 1.7em;
}

ul > li {
	padding-left: 0.1em;
}

ol {
	padding-inline-start: 1.6em;
}

ol > li {
	padding-left: 0.2em;
}

.mono ol {
	padding-inline-start: 2em;
}

.mono ol > li {
	text-indent: -0.4em;
}

.toggle {
	padding-inline-start: 0em;
	list-style-type: none;
}

/* Indent toggle children */
.toggle > li > details {
	padding-left: 1.7em;
}

.toggle > li > details > summary {
	margin-left: -1.1em;
}

.selected-value {
	display: inline-block;
	padding: 0 0.5em;
	background: rgba(206, 205, 202, 0.5);
	border-radius: 3px;
	margin-right: 0.5em;
	margin-top: 0.3em;
	margin-bottom: 0.3em;
	white-space: nowrap;
}

.collection-title {
	display: inline-block;
	margin-right: 1em;
}

.simple-table {
	margin-top: 1em;
	font-size: 0.875rem;
	empty-cells: show;
}
.simple-table td {
	height: 29px;
	min-width: 120px;
}

.simple-table th {
	height: 29px;
	min-width: 120px;
}

.simple-table-header-color {
	background: rgb(247, 246, 243);
	color: black;
}
.simple-table-header {
	font-weight: 500;
}

time {
	opacity: 0.5;
}

.icon {
	display: inline-block;
	max-width: 1.2em;
	max-height: 1.2em;
	text-decoration: none;
	vertical-align: text-bottom;
	margin-right: 0.5em;
}

img.icon {
	border-radius: 3px;
}

.user-icon {
	width: 1.5em;
	height: 1.5em;
	border-radius: 100%;
	margin-right: 0.5rem;
}

.user-icon-inner {
	font-size: 0.8em;
}

.text-icon {
	border: 1px solid #000;
	text-align: center;
}

.page-cover-image {
	display: block;
	object-fit: cover;
	width: 100%;
	max-height: 30vh;
}

.page-header-icon {
	font-size: 3rem;
	margin-bottom: 1rem;
}

.page-header-icon-with-cover {
	margin-top: -0.72em;
	margin-left: 0.07em;
}

.page-header-icon img {
	border-radius: 3px;
}

.link-to-page {
	margin: 1em 0;
	padding: 0;
	border: none;
	font-weight: 500;
}

p > .user {
	opacity: 0.5;
}

td > .user,
td > time {
	white-space: nowrap;
}

input[type="checkbox"] {
	transform: scale(1.5);
	margin-right: 0.6em;
	vertical-align: middle;
}

p {
	margin-top: 0.5em;
	margin-bottom: 0.5em;
}

.image {
	border: none;
	margin: 1.5em 0;
	padding: 0;
	border-radius: 0;
	text-align: center;
}

.code,
code {
	background: rgba(135, 131, 120, 0.15);
	border-radius: 3px;
	padding: 0.2em 0.4em;
	border-radius: 3px;
	font-size: 85%;
	tab-size: 2;
}

code {
	color: #eb5757;
}

.code {
	padding: 1.5em 1em;
}

.code-wrap {
	white-space: pre-wrap;
	word-break: break-all;
}

.code > code {
	background: none;
	padding: 0;
	font-size: 100%;
	color: inherit;
}

blockquote {
	font-size: 1.25em;
	margin: 1em 0;
	padding-left: 1em;
	border-left: 3px solid rgb(55, 53, 47);
}

.bookmark {
	text-decoration: none;
	max-height: 8em;
	padding: 0;
	display: flex;
	width: 100%;
	align-items: stretch;
}

.bookmark-title {
	font-size: 0.85em;
	overflow: hidden;
	text-overflow: ellipsis;
	height: 1.75em;
	white-space: nowrap;
}

.bookmark-text {
	display: flex;
	flex-direction: column;
}

.bookmark-info {
	flex: 4 1 180px;
	padding: 12px 14px 14px;
	display: flex;
	flex-direction: column;
	justify-content: space-between;
}

.bookmark-image {
	width: 33%;
	flex: 1 1 180px;
	display: block;
	position: relative;
	object-fit: cover;
	border-radius: 1px;
}

.bookmark-description {
	color: rgba(55, 53, 47, 0.6);
	font-size: 0.75em;
	overflow: hidden;
	max-height: 4.5em;
	word-break: break-word;
}

.bookmark-href {
	font-size: 0.75em;
	margin-top: 0.25em;
}

.sans { font-family: ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol"; }
.code { font-family: "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace; }
.serif { font-family: Lyon-Text, Georgia, ui-serif, serif; }
.mono { font-family: iawriter-mono, Nitti, Menlo, Courier, monospace; }
.pdf .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK JP'; }
.pdf:lang(zh-CN) .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK SC'; }
.pdf:lang(zh-TW) .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK TC'; }
.pdf:lang(ko-KR) .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK KR'; }
.pdf .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK JP'; }
.pdf:lang(zh-CN) .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK SC'; }
.pdf:lang(zh-TW) .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK TC'; }
.pdf:lang(ko-KR) .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK KR'; }
.pdf .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK JP'; }
.pdf:lang(zh-CN) .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK SC'; }
.pdf:lang(zh-TW) .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK TC'; }
.pdf:lang(ko-KR) .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK KR'; }
.pdf .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK JP'; }
.pdf:lang(zh-CN) .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK SC'; }
.pdf:lang(zh-TW) .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK TC'; }
.pdf:lang(ko-KR) .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK KR'; }
.highlight-default {
	color: rgba(55, 53, 47, 1);
}
.highlight-gray {
	color: rgba(120, 119, 116, 1);
	fill: rgba(120, 119, 116, 1);
}
.highlight-brown {
	color: rgba(159, 107, 83, 1);
	fill: rgba(159, 107, 83, 1);
}
.highlight-orange {
	color: rgba(217, 115, 13, 1);
	fill: rgba(217, 115, 13, 1);
}
.highlight-yellow {
	color: rgba(203, 145, 47, 1);
	fill: rgba(203, 145, 47, 1);
}
.highlight-teal {
	color: rgba(68, 131, 97, 1);
	fill: rgba(68, 131, 97, 1);
}
.highlight-blue {
	color: rgba(51, 126, 169, 1);
	fill: rgba(51, 126, 169, 1);
}
.highlight-purple {
	color: rgba(144, 101, 176, 1);
	fill: rgba(144, 101, 176, 1);
}
.highlight-pink {
	color: rgba(193, 76, 138, 1);
	fill: rgba(193, 76, 138, 1);
}
.highlight-red {
	color: rgba(212, 76, 71, 1);
	fill: rgba(212, 76, 71, 1);
}
.highlight-gray_background {
	background: rgba(241, 241, 239, 1);
}
.highlight-brown_background {
	background: rgba(244, 238, 238, 1);
}
.highlight-orange_background {
	background: rgba(251, 236, 221, 1);
}
.highlight-yellow_background {
	background: rgba(251, 243, 219, 1);
}
.highlight-teal_background {
	background: rgba(237, 243, 236, 1);
}
.highlight-blue_background {
	background: rgba(231, 243, 248, 1);
}
.highlight-purple_background {
	background: rgba(244, 240, 247, 0.8);
}
.highlight-pink_background {
	background: rgba(249, 238, 243, 0.8);
}
.highlight-red_background {
	background: rgba(253, 235, 236, 1);
}
.block-color-default {
	color: inherit;
	fill: inherit;
}
.block-color-gray {
	color: rgba(120, 119, 116, 1);
	fill: rgba(120, 119, 116, 1);
}
.block-color-brown {
	color: rgba(159, 107, 83, 1);
	fill: rgba(159, 107, 83, 1);
}
.block-color-orange {
	color: rgba(217, 115, 13, 1);
	fill: rgba(217, 115, 13, 1);
}
.block-color-yellow {
	color: rgba(203, 145, 47, 1);
	fill: rgba(203, 145, 47, 1);
}
.block-color-teal {
	color: rgba(68, 131, 97, 1);
	fill: rgba(68, 131, 97, 1);
}
.block-color-blue {
	color: rgba(51, 126, 169, 1);
	fill: rgba(51, 126, 169, 1);
}
.block-color-purple {
	color: rgba(144, 101, 176, 1);
	fill: rgba(144, 101, 176, 1);
}
.block-color-pink {
	color: rgba(193, 76, 138, 1);
	fill: rgba(193, 76, 138, 1);
}
.block-color-red {
	color: rgba(212, 76, 71, 1);
	fill: rgba(212, 76, 71, 1);
}
.block-color-gray_background {
	background: rgba(241, 241, 239, 1);
}
.block-color-brown_background {
	background: rgba(244, 238, 238, 1);
}
.block-color-orange_background {
	background: rgba(251, 236, 221, 1);
}
.block-color-yellow_background {
	background: rgba(251, 243, 219, 1);
}
.block-color-teal_background {
	background: rgba(237, 243, 236, 1);
}
.block-color-blue_background {
	background: rgba(231, 243, 248, 1);
}
.block-color-purple_background {
	background: rgba(244, 240, 247, 0.8);
}
.block-color-pink_background {
	background: rgba(249, 238, 243, 0.8);
}
.block-color-red_background {
	background: rgba(253, 235, 236, 1);
}
.select-value-color-pink { background-color: rgba(245, 224, 233, 1); }
.select-value-color-purple { background-color: rgba(232, 222, 238, 1); }
.select-value-color-green { background-color: rgba(219, 237, 219, 1); }
.select-value-color-gray { background-color: rgba(227, 226, 224, 1); }
.select-value-color-opaquegray { background-color: rgba(255, 255, 255, 0.0375); }
.select-value-color-orange { background-color: rgba(250, 222, 201, 1); }
.select-value-color-brown { background-color: rgba(238, 224, 218, 1); }
.select-value-color-red { background-color: rgba(255, 226, 221, 1); }
.select-value-color-yellow { background-color: rgba(253, 236, 200, 1); }
.select-value-color-blue { background-color: rgba(211, 229, 239, 1); }

.checkbox {
	display: inline-flex;
	vertical-align: text-bottom;
	width: 16;
	height: 16;
	background-size: 16px;
	margin-left: 2px;
	margin-right: 5px;
}

.checkbox-on {
	background-image: url("data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2216%22%20height%3D%2216%22%20viewBox%3D%220%200%2016%2016%22%20fill%3D%22none%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%0A%3Crect%20width%3D%2216%22%20height%3D%2216%22%20fill%3D%22%2358A9D7%22%2F%3E%0A%3Cpath%20d%3D%22M6.71429%2012.2852L14%204.9995L12.7143%203.71436L6.71429%209.71378L3.28571%206.2831L2%207.57092L6.71429%2012.2852Z%22%20fill%3D%22white%22%2F%3E%0A%3C%2Fsvg%3E");
}

.checkbox-off {
	background-image: url("data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2216%22%20height%3D%2216%22%20viewBox%3D%220%200%2016%2016%22%20fill%3D%22none%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%0A%3Crect%20x%3D%220.75%22%20y%3D%220.75%22%20width%3D%2214.5%22%20height%3D%2214.5%22%20fill%3D%22white%22%20stroke%3D%22%2336352F%22%20stroke-width%3D%221.5%22%2F%3E%0A%3C%2Fsvg%3E");
}
	
</style></head><body><article id="a48fdf23-7f44-4002-85df-41dbd2a6ae37" class="page sans"><header><h1 class="page-title">The Hirudo Medicinalis Microbiome Is a Source of New Antimicrobial Peptides</h1><table class="properties"><tbody><tr class="property-row property-row-multi_select"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesMultipleSelect"><path d="M1.91602 4.83789C2.44238 4.83789 2.87305 4.40723 2.87305 3.87402C2.87305 3.34766 2.44238 2.91699 1.91602 2.91699C1.38281 2.91699 0.952148 3.34766 0.952148 3.87402C0.952148 4.40723 1.38281 4.83789 1.91602 4.83789ZM5.1084 4.52344H14.3984C14.7607 4.52344 15.0479 4.23633 15.0479 3.87402C15.0479 3.51172 14.7607 3.22461 14.3984 3.22461H5.1084C4.74609 3.22461 4.45898 3.51172 4.45898 3.87402C4.45898 4.23633 4.74609 4.52344 5.1084 4.52344ZM1.91602 9.03516C2.44238 9.03516 2.87305 8.60449 2.87305 8.07129C2.87305 7.54492 2.44238 7.11426 1.91602 7.11426C1.38281 7.11426 0.952148 7.54492 0.952148 8.07129C0.952148 8.60449 1.38281 9.03516 1.91602 9.03516ZM5.1084 8.7207H14.3984C14.7607 8.7207 15.0479 8.43359 15.0479 8.07129C15.0479 7.70898 14.7607 7.42188 14.3984 7.42188H5.1084C4.74609 7.42188 4.45898 7.70898 4.45898 8.07129C4.45898 8.43359 4.74609 8.7207 5.1084 8.7207ZM1.91602 13.2324C2.44238 13.2324 2.87305 12.8018 2.87305 12.2686C2.87305 11.7422 2.44238 11.3115 1.91602 11.3115C1.38281 11.3115 0.952148 11.7422 0.952148 12.2686C0.952148 12.8018 1.38281 13.2324 1.91602 13.2324ZM5.1084 12.918H14.3984C14.7607 12.918 15.0479 12.6309 15.0479 12.2686C15.0479 11.9062 14.7607 11.6191 14.3984 11.6191H5.1084C4.74609 11.6191 4.45898 11.9062 4.45898 12.2686C4.45898 12.6309 4.74609 12.918 5.1084 12.918Z"></path></svg></span>Journal/Publisher</th><td><span class="selected-value select-value-color-default">International Journal of Molecular Sciences</span></td></tr><tr class="property-row property-row-number"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesNumber"><path d="M2.4834 10.9902H4.33594L3.75488 13.8887C3.74121 13.9639 3.72754 14.0664 3.72754 14.1416C3.72754 14.5381 3.99414 14.7637 4.39746 14.7637C4.81445 14.7637 5.09473 14.5449 5.18359 14.1143L5.80566 10.9902H8.79297L8.21191 13.8887C8.19824 13.9639 8.18457 14.0664 8.18457 14.1416C8.18457 14.5381 8.45117 14.7637 8.85449 14.7637C9.27148 14.7637 9.55176 14.5449 9.63379 14.1143L10.2627 10.9902H12.4502C12.9287 10.9902 13.2432 10.6758 13.2432 10.2109C13.2432 9.8418 12.9902 9.56836 12.6006 9.56836H10.5498L11.2129 6.28711H13.3662C13.8379 6.28711 14.1523 5.96582 14.1523 5.50098C14.1523 5.13184 13.9062 4.8584 13.5098 4.8584H11.5L12.0195 2.27441C12.0264 2.21973 12.0469 2.11035 12.0469 2.01465C12.0469 1.625 11.7666 1.39258 11.3633 1.39258C10.9053 1.39258 10.6797 1.63867 10.5977 2.05566L10.0303 4.8584H7.04297L7.5625 2.27441C7.57617 2.21973 7.58984 2.11035 7.58984 2.01465C7.58984 1.625 7.30957 1.39258 6.91309 1.39258C6.44824 1.39258 6.21582 1.63867 6.13379 2.05566L5.57324 4.8584H3.54297C3.06445 4.8584 2.75 5.18652 2.75 5.65137C2.75 6.03418 3.00293 6.28711 3.39258 6.28711H5.28613L4.62305 9.56836H2.63379C2.15527 9.56836 1.84082 9.89648 1.84082 10.3613C1.84082 10.7373 2.09375 10.9902 2.4834 10.9902ZM6.09277 9.56836L6.75586 6.28711H9.74316L9.08008 9.56836H6.09277Z"></path></svg></span>Year</th><td>2020</td></tr><tr class="property-row property-row-text"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesText"><path d="M1.56738 3.25879H14.4258C14.7676 3.25879 15.0479 2.97852 15.0479 2.63672C15.0479 2.29492 14.7744 2.02148 14.4258 2.02148H1.56738C1.21875 2.02148 0.952148 2.29492 0.952148 2.63672C0.952148 2.97852 1.22559 3.25879 1.56738 3.25879ZM1.56738 6.84082H14.4258C14.7676 6.84082 15.0479 6.56055 15.0479 6.21875C15.0479 5.87695 14.7744 5.60352 14.4258 5.60352H1.56738C1.21875 5.60352 0.952148 5.87695 0.952148 6.21875C0.952148 6.56055 1.22559 6.84082 1.56738 6.84082ZM1.56738 10.4229H14.4258C14.7676 10.4229 15.0479 10.1426 15.0479 9.80078C15.0479 9.45898 14.7744 9.18555 14.4258 9.18555H1.56738C1.21875 9.18555 0.952148 9.45898 0.952148 9.80078C0.952148 10.1426 1.22559 10.4229 1.56738 10.4229ZM1.56738 14.0049H8.75879C9.10059 14.0049 9.38086 13.7246 9.38086 13.3828C9.38086 13.041 9.10742 12.7676 8.75879 12.7676H1.56738C1.21875 12.7676 0.952148 13.041 0.952148 13.3828C0.952148 13.7246 1.22559 14.0049 1.56738 14.0049Z"></path></svg></span>Authors</th><td>Grafskaia, E., Pavlova, E., Babenko, V. V., Latsis, I., Malakhova, M., Lavrenova, V. Lazarev, V.</td></tr><tr class="property-row property-row-url"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesUrl"><path d="M7.69922 10.8945L8.73828 9.84863C7.91797 9.77344 7.34375 9.51367 6.91992 9.08984C5.76465 7.93457 5.76465 6.29395 6.91309 5.14551L9.18262 2.87598C10.3379 1.7207 11.9717 1.7207 13.127 2.87598C14.2891 4.04492 14.2822 5.67188 13.1338 6.82031L11.958 7.99609C12.1768 8.49512 12.2451 9.10352 12.1289 9.62988L14.0908 7.6748C15.7725 6 15.7793 3.62109 14.084 1.92578C12.3887 0.223633 10.0098 0.237305 8.33496 1.91211L5.95605 4.29785C4.28125 5.97266 4.26758 8.35156 5.96289 10.0469C6.36621 10.4434 6.90625 10.7441 7.69922 10.8945ZM8.30078 5.13184L7.26855 6.17773C8.08203 6.25293 8.66309 6.51953 9.08008 6.93652C10.2422 8.09863 10.2422 9.73242 9.08691 10.8809L6.81738 13.1504C5.66211 14.3057 4.03516 14.3057 2.87305 13.1504C1.71094 11.9883 1.71777 10.3545 2.87305 9.20605L4.04199 8.03027C3.83008 7.53125 3.75488 6.92969 3.87109 6.39648L1.91602 8.35156C0.234375 10.0264 0.227539 12.4121 1.92285 14.1074C3.61816 15.8027 5.99707 15.7891 7.67188 14.1143L10.0439 11.7354C11.7256 10.0537 11.7324 7.6748 10.0371 5.98633C9.64062 5.58301 9.10059 5.28223 8.30078 5.13184Z"></path></svg></span>Link</th><td><a href="https://sci-hub.se/10.3390/ijms21197141" class="url-value">https://sci-hub.se/10.3390/ijms21197141</a></td></tr><tr class="property-row property-row-url"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesUrl"><path d="M7.69922 10.8945L8.73828 9.84863C7.91797 9.77344 7.34375 9.51367 6.91992 9.08984C5.76465 7.93457 5.76465 6.29395 6.91309 5.14551L9.18262 2.87598C10.3379 1.7207 11.9717 1.7207 13.127 2.87598C14.2891 4.04492 14.2822 5.67188 13.1338 6.82031L11.958 7.99609C12.1768 8.49512 12.2451 9.10352 12.1289 9.62988L14.0908 7.6748C15.7725 6 15.7793 3.62109 14.084 1.92578C12.3887 0.223633 10.0098 0.237305 8.33496 1.91211L5.95605 4.29785C4.28125 5.97266 4.26758 8.35156 5.96289 10.0469C6.36621 10.4434 6.90625 10.7441 7.69922 10.8945ZM8.30078 5.13184L7.26855 6.17773C8.08203 6.25293 8.66309 6.51953 9.08008 6.93652C10.2422 8.09863 10.2422 9.73242 9.08691 10.8809L6.81738 13.1504C5.66211 14.3057 4.03516 14.3057 2.87305 13.1504C1.71094 11.9883 1.71777 10.3545 2.87305 9.20605L4.04199 8.03027C3.83008 7.53125 3.75488 6.92969 3.87109 6.39648L1.91602 8.35156C0.234375 10.0264 0.227539 12.4121 1.92285 14.1074C3.61816 15.8027 5.99707 15.7891 7.67188 14.1143L10.0439 11.7354C11.7256 10.0537 11.7324 7.6748 10.0371 5.98633C9.64062 5.58301 9.10059 5.28223 8.30078 5.13184Z"></path></svg></span>Dataset</th><td><a href="grafskayacath@gmail.com" class="url-value">grafskayacath@gmail.com</a></td></tr><tr class="property-row property-row-created_time"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesCreatedAt"><path d="M8 15.126C11.8623 15.126 15.0615 11.9336 15.0615 8.06445C15.0615 4.20215 11.8623 1.00293 7.99316 1.00293C4.13086 1.00293 0.938477 4.20215 0.938477 8.06445C0.938477 11.9336 4.1377 15.126 8 15.126ZM8 13.7383C4.85547 13.7383 2.33301 11.209 2.33301 8.06445C2.33301 4.91992 4.84863 2.39746 7.99316 2.39746C11.1377 2.39746 13.6738 4.91992 13.6738 8.06445C13.6738 11.209 11.1445 13.7383 8 13.7383ZM4.54102 8.91211H7.99316C8.30078 8.91211 8.54004 8.67285 8.54004 8.37207V3.8877C8.54004 3.58691 8.30078 3.34766 7.99316 3.34766C7.69238 3.34766 7.45312 3.58691 7.45312 3.8877V7.83203H4.54102C4.2334 7.83203 4.00098 8.06445 4.00098 8.37207C4.00098 8.67285 4.2334 8.91211 4.54102 8.91211Z"></path></svg></span>Log Date</th><td><time>@January 24, 2021 7:25 PM</time></td></tr><tr class="property-row property-row-multi_select"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesMultipleSelect"><path d="M1.91602 4.83789C2.44238 4.83789 2.87305 4.40723 2.87305 3.87402C2.87305 3.34766 2.44238 2.91699 1.91602 2.91699C1.38281 2.91699 0.952148 3.34766 0.952148 3.87402C0.952148 4.40723 1.38281 4.83789 1.91602 4.83789ZM5.1084 4.52344H14.3984C14.7607 4.52344 15.0479 4.23633 15.0479 3.87402C15.0479 3.51172 14.7607 3.22461 14.3984 3.22461H5.1084C4.74609 3.22461 4.45898 3.51172 4.45898 3.87402C4.45898 4.23633 4.74609 4.52344 5.1084 4.52344ZM1.91602 9.03516C2.44238 9.03516 2.87305 8.60449 2.87305 8.07129C2.87305 7.54492 2.44238 7.11426 1.91602 7.11426C1.38281 7.11426 0.952148 7.54492 0.952148 8.07129C0.952148 8.60449 1.38281 9.03516 1.91602 9.03516ZM5.1084 8.7207H14.3984C14.7607 8.7207 15.0479 8.43359 15.0479 8.07129C15.0479 7.70898 14.7607 7.42188 14.3984 7.42188H5.1084C4.74609 7.42188 4.45898 7.70898 4.45898 8.07129C4.45898 8.43359 4.74609 8.7207 5.1084 8.7207ZM1.91602 13.2324C2.44238 13.2324 2.87305 12.8018 2.87305 12.2686C2.87305 11.7422 2.44238 11.3115 1.91602 11.3115C1.38281 11.3115 0.952148 11.7422 0.952148 12.2686C0.952148 12.8018 1.38281 13.2324 1.91602 13.2324ZM5.1084 12.918H14.3984C14.7607 12.918 15.0479 12.6309 15.0479 12.2686C15.0479 11.9062 14.7607 11.6191 14.3984 11.6191H5.1084C4.74609 11.6191 4.45898 11.9062 4.45898 12.2686C4.45898 12.6309 4.74609 12.918 5.1084 12.918Z"></path></svg></span>Type</th><td><span class="selected-value select-value-color-default">Peer-Review Record</span></td></tr><tr class="property-row property-row-multi_select"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesMultipleSelect"><path d="M1.91602 4.83789C2.44238 4.83789 2.87305 4.40723 2.87305 3.87402C2.87305 3.34766 2.44238 2.91699 1.91602 2.91699C1.38281 2.91699 0.952148 3.34766 0.952148 3.87402C0.952148 4.40723 1.38281 4.83789 1.91602 4.83789ZM5.1084 4.52344H14.3984C14.7607 4.52344 15.0479 4.23633 15.0479 3.87402C15.0479 3.51172 14.7607 3.22461 14.3984 3.22461H5.1084C4.74609 3.22461 4.45898 3.51172 4.45898 3.87402C4.45898 4.23633 4.74609 4.52344 5.1084 4.52344ZM1.91602 9.03516C2.44238 9.03516 2.87305 8.60449 2.87305 8.07129C2.87305 7.54492 2.44238 7.11426 1.91602 7.11426C1.38281 7.11426 0.952148 7.54492 0.952148 8.07129C0.952148 8.60449 1.38281 9.03516 1.91602 9.03516ZM5.1084 8.7207H14.3984C14.7607 8.7207 15.0479 8.43359 15.0479 8.07129C15.0479 7.70898 14.7607 7.42188 14.3984 7.42188H5.1084C4.74609 7.42188 4.45898 7.70898 4.45898 8.07129C4.45898 8.43359 4.74609 8.7207 5.1084 8.7207ZM1.91602 13.2324C2.44238 13.2324 2.87305 12.8018 2.87305 12.2686C2.87305 11.7422 2.44238 11.3115 1.91602 11.3115C1.38281 11.3115 0.952148 11.7422 0.952148 12.2686C0.952148 12.8018 1.38281 13.2324 1.91602 13.2324ZM5.1084 12.918H14.3984C14.7607 12.918 15.0479 12.6309 15.0479 12.2686C15.0479 11.9062 14.7607 11.6191 14.3984 11.6191H5.1084C4.74609 11.6191 4.45898 11.9062 4.45898 12.2686C4.45898 12.6309 4.74609 12.918 5.1084 12.918Z"></path></svg></span>Technique</th><td></td></tr><tr class="property-row property-row-checkbox"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesCheckbox"><path d="M3.85742 14.4561H12.1357C13.6123 14.4561 14.3779 13.6904 14.3779 12.2344V3.91504C14.3779 2.45215 13.6123 1.69336 12.1357 1.69336H3.85742C2.38086 1.69336 1.61523 2.45215 1.61523 3.91504V12.2344C1.61523 13.6973 2.38086 14.4561 3.85742 14.4561ZM3.93945 13.1162C3.30371 13.1162 2.95508 12.7812 2.95508 12.1182V4.02441C2.95508 3.36133 3.30371 3.0332 3.93945 3.0332H12.0537C12.6826 3.0332 13.0381 3.36133 13.0381 4.02441V12.1182C13.0381 12.7812 12.6826 13.1162 12.0537 13.1162H3.93945ZM7.26855 11.3115C7.51465 11.3115 7.72656 11.1885 7.87012 10.9697L10.9258 6.19141C11.0146 6.04785 11.0967 5.88379 11.0967 5.72656C11.0967 5.3916 10.7959 5.16602 10.4746 5.16602C10.2695 5.16602 10.085 5.27539 9.94141 5.50781L7.24121 9.8418L5.96973 8.22168C5.80566 8.00977 5.6416 7.93457 5.43652 7.93457C5.10156 7.93457 4.8418 8.20117 4.8418 8.54297C4.8418 8.70703 4.90332 8.85742 5.01953 9.00098L6.63281 10.9697C6.81738 11.209 7.01562 11.3115 7.26855 11.3115Z"></path></svg></span>Dataset Raw</th><td><div class="checkbox checkbox-on"></div></td></tr></tbody></table></header><div class="page-body"><p id="d1fd7cb8-7ef5-4742-8d4d-f1c74e7ef19f" class="">Reviewer 1: Anonymous</p><p id="40ff72e0-98bf-4f8d-b381-7fbe3e229ffb" class="">Reviewer 2: Anonymous</p><p id="cff695dd-3161-4184-8d83-c36b2aa97ddf" class=""><strong>Round 1</strong></p><p id="c4f20a96-e9f3-4ea1-81ee-b3aa72f76b7a" class=""><em>Reviewer 1 Report</em></p><p id="02faa352-20b1-4a48-8593-5f3481421580" class="">In the present paper, the authors analyze the antimicrobial characteristics of six different bacterial peptides, selected after a specific <em>in silico</em> screening analysis conducted on <em>Hirudo medicinalis</em> metagenome. The MIC (minimum inhibitory concentration) of every chosen peptide have been calculated observing the effects on different Gram-negative and Gram-positive bacterial strains and the relative antibacterial role was evaluated by different techniques. Moreover, the cytotoxicity of these amino acid sequences has been tested on McCoy cells. Taken together, the results of these experiments highlight as these putative AMPs show a different antimicrobial activity and cytotoxicity.</p><p id="7c58a6f6-d41c-4cb6-a10b-621085b0e225" class="">In my opinion, although the manuscript is well written, several modifications should be provided before publication.</p><p id="84df2c14-e62e-4e48-92e0-6231788a0cd6" class="">Minor revisions:</p><p id="6e91995c-8ed6-4653-9993-aed63c38554a" class=""><strong>Line 161: </strong>in the text, probably authors refer to Figure 2 and not Figure 3.</p><p id="26ddbbe3-4aaa-4e7d-babb-a27f631ee099" class=""><strong>Line 172: </strong>in the text, probably authors refer to Figure 3 and not Figure 4.</p><p id="a761b3b4-a949-4792-903f-3e67100259f3" class="">Major revisions:</p><ul id="5703f1c9-edb8-4745-b815-c2d81b06af4f" class="bulleted-list"><li style="list-style-type:disc"><strong>Section 2.1.</strong> The authors performed a wide and specific work of selection for determining 65 possible AMPs, using different predictive programs and algorithms and 3D models were created with I-TASSER-MR server. However, authors choose only six peptides and it results difficult to understand why they considered only these. Did the authors select only peptides with α-helical structure after I-TASSER analysis?</li></ul><p id="0650be16-207c-4957-aa48-253d145b14c5" class="">As it is well explained in the introduction, although the most widespread AMPs are amphipathic α-helical peptides, these molecules present many forms. Indeed, the well-known human defensins possess β-sheet structures. In this case, maybe other sequences should be considered among these 65. For this reason, in my opinion this step should be better explained, even in the discussion or in the material and methods section.</p><ul id="7e7cb53c-7807-4fd6-8a71-05f1c785de47" class="bulleted-list"><li style="list-style-type:disc"><strong>Section 2.4. </strong>The cytotoxic effect of AMPs was tested on McCoy cells and evaluated with two different techniques: fluorescent imaging and colorimetric assay. Since these two experimental approaches are well explained in the section of Material and Methods, authors could insert a figure showing the effects obtained with the most significant peptides (in each sense) and of the Melittin control.</li></ul><p id="ec101c8c-b9f3-41d4-a864-4f020b66b083" class="">In the figure S2, cell viability has been tested only on three peptides (Pept_1545, Pept_148 and Pept_84), why the authors did not test also the LHD assay on every peptide? The decision to analyze these derives from the fluorescent results? In this case, Pept_1303 showed a percentage higher than Pept_84.</p><p id="6cc8fc55-9f7c-4f74-ac53-eb261d14247d" class="">Moreover, these experiments revealed as some peptides are significantly less cytotoxic compared with Melittin, used as control. Although, in the discussion, this AMP, deriving from bee venom, is described to possess a nonspecific cytotoxicity and haemolytic activity. In this case, why authors choose proper this molecule as control? If the final score is to discover new AMPs that could be used for the development of therapeutic agents, why they did not observe differences with other AMPs, such as the mentioned PAC-113 or dalbavancin, which are already used in this field and that not present cytotoxicity?</p><ul id="3a10f273-6a08-413d-b7b1-59252e3070d3" class="bulleted-list"><li style="list-style-type:disc"><strong>Section 2.5. </strong>As before, a figure or images should be added, in which bacteria treated with propidium iodide are visible. Also in this case, the most significant conditions could be used.</li></ul><p id="555e0e19-ca3c-4166-8530-5e1477f35458" class="">Moreover, in these experiments, the effect of Pept_148 were evaluated during the kinetics of PI penetration (Figure 2), while no data are reported about the fluorescent result (Figure S3). Why the authors decided to not consider this peptide in the S3 graphs?</p><p id="cb0cc568-ddde-445c-af2a-3e995ce26839" class="">Why the same test has not been conducted on <em>B. haemolyticus</em> in order to compare AMPs effects with SEM results?</p><ul id="90d0a7c8-8ae6-423f-a0a8-2081ed3b3299" class="bulleted-list"><li style="list-style-type:disc"><strong>Section 2.6. </strong>Why the authors decided to perform SEM analyses on the Pept_84, which not seems to show a great effect on <em> subtilis</em> in the PI assay and that possess high cytotoxicity, rather not considering the Pept_148?</li></ul><p id="45aa5db3-798a-4a11-8143-3ee91ce4f370" class="">Moreover, the AMP activity tested in the experiments with PI was calculated using the 2x MIC concentration and after a short period of time. Authors suggested a rapid mechanism of action for the antimicrobial peptides. Based on this experimental concentration and timing, why the authors choose 8 hours before observing SEM results and did not used the same concentration for compare the results?</p><p id="cdbb0eb1-233c-4f21-958e-088e71791b7c" class=""><strong>Discussion</strong></p><p id="77f80bf6-9344-44b7-8b41-3eb13727de15" class="">It seems that these six peptides derive from specific bacterial taxa present in the <em>H. medicinalis</em> microbiome (Aminobacter, Sphingobacteriia and Chitinophagia). Is there a correlation between these precise organisms and AMPs? Have these bacteria a precise role in the Hirudo microbiome? This point should be argument in the discussion.</p><p id="a287a36b-e2e0-4fb2-aaf4-c51d795ae1eb" class="">Other important aspects about AMPs are both the positive surface charge and the hydrophobicity, as explained at line 228. Authors should add some information about these points (considering for example the Boman index), analysing the characteristics of the main peptides (or only of Pept_1545).</p><p id="ffb7af96-000d-439d-817d-dbd6e118ed9c" class=""><em>Author Response</em></p><p id="6ec196c0-6369-4f9c-a878-af7a50a4e0fb" class="">We would like to thank the reviewer for their detailed comments and suggestions for the manuscript. We believe that the comments have identified important areas which required improvement. After completion of the suggested edits, the revised manuscript has benefitted from an improvement in the overall presentation and clarity. The detailed answers on issues as well as the modifications introduced into the revised version of the manuscript are given below. Original reviewer comments in boldface, responses in regular typeface.</p><p id="eb3c6435-3ee9-481f-9ab7-fb8fbea92b41" class=""><strong>In the present paper, the authors analyze the antimicrobial characteristics of six different bacterial peptides, selected after a specific </strong><em><strong>in silico</strong></em><strong> screening analysis conducted on </strong><em><strong>Hirudo medicinalis </strong></em><strong>metagenome. The MIC (minimum inhibitory concentration) of every chosen peptide have been calculated observing the effects on different Gram-negative and Gram-positive bacterial strains and the relative antibacterial role was evaluated by different techniques. Moreover, the cytotoxicity of these amino acid sequences has been tested on McCoy cells. Taken together, the results of these experiments highlight as these putative AMPs show a different antimicrobial activity and cytotoxicity.</strong></p><p id="0f150e55-ad51-4c33-a896-f4b60f6d5308" class=""><strong>In my opinion, although the manuscript is well written, several modifications should be provided before publication.</strong></p><p id="0ed1a0ad-3e24-4c82-a0e1-a66fbf2edfca" class=""><strong>Minor revisions:</strong></p><p id="03487519-a919-4824-a951-2996492f840b" class=""><strong>Line 161: in the text, probably authors refer to Figure 2 and not Figure 3.</strong></p><p id="14f06bf4-1c07-4059-8d5d-9cacd40c94e0" class="">Figure reference was corrected.</p><p id="a28b7917-64a6-4d58-ae28-9bc872408720" class=""><strong>Line 172: in the text, probably authors refer to Figure 3 and not Figure 4.</strong></p><p id="74268f55-7abb-44b4-93ab-71f22dc25bed" class="">Figure reference was corrected.</p><p id="e7a89b89-316d-4306-9baa-dd1532a3158d" class=""><strong>Major revisions:</strong></p><ul id="703b8d23-46ec-4750-9f0b-f5ed84de8f54" class="bulleted-list"><li style="list-style-type:disc"><strong>Section 2.1. The authors performed a wide and specific work of selection for determining 65 possible AMPs, using different predictive programs and algorithms and 3D models were created with I-TASSER-MR server. However, authors choose only six peptides and it results difficult to understand why they considered only these. Did the authors select only peptides with α</strong><strong>helical structure after I-TASSER analysis?</strong></li></ul><p id="d9a89284-9d9d-4f2a-85c7-84337c053302" class=""><strong>As it is well explained in the introduction, although the most widespread AMPs are amphipathic α</strong><strong>-helical peptides, these molecules present many forms. Indeed, the well-known human defensins possess β</strong><strong>-sheet structures. In this case, maybe other sequences should be considered among these 65. For this reason, in my opinion this step should be better explained, even in the discussion or in the material and methods section.</strong></p><p id="7351d3d7-407f-441c-abc0-39f6857e287a" class="">Peptides were chosen not by only their potential effects, but also on their secondary structure. From filtered 65 peptides, we chose 5 peptides with highest antimicrobial propensity scores which can exhibit α–helical structure and additionally 1 peptide with unstructured conformation and high propensity score based on the predictions.</p><p id="938bc41b-f2fe-4222-80ed-23718672376f" class="">The sentences “Overall, as potential AMPs, we identified 6 amino acid sequences (Table 1). Five of these peptides can adopt an α–helical structure under prediction. The leftover peptide is unstructured.” in the Result section were replaced by “According to the secondary structure prediction among filtered 65 peptides with the highest antimicrobial propensity scores the predicted secondary structures were only α–helical or unstructured conformations. From 65 peptides we chose 5 peptides with the highest antimicrobial propensity scores which can exhibit α–helical structure and additionally 1 peptide with unstructured conformation and high propensity score according to the predictions (Table 1).”</p><p id="71554d36-fdda-4d41-b59e-6025b67cd93d" class="">The sentence “As a result, we selected six putative α–helical AMPs.” in the Discussion section was corrected to “As a result, we selected six putative AMPs.”</p><ul id="898b01e8-022e-491a-a4ad-ab940b3b6275" class="bulleted-list"><li style="list-style-type:disc"><strong>Section 2.4. The cytotoxic effect of AMPs was tested on McCoy cells and evaluated with two different techniques: fluorescent imaging and colorimetric assay. Since these two experimental approaches are well explained in the section of Material and Methods, authors could insert a figure showing the effects obtained with the most significant peptides (in each sense) and of the Melittin control.</strong></li></ul><p id="e3dab1a1-15d2-46da-8549-d19276646846" class=""><strong>In the figure S2, cell viability has been tested only on three peptides (Pept_1545, Pept_148 and Pept_84), why the authors did not test also the LHD assay on every peptide? The decision to analyze these derives from the fluorescent results? In this case, Pept_1303 showed a percentage higher than Pept_84.</strong></p><p id="886a6e61-e274-429f-b98c-30bf61148916" class=""><strong>Moreover, these experiments revealed as some peptides are significantly less cytotoxic compared with Melittin, used as control. Although, in the discussion, this AMP, deriving from bee venom, is described to possess a nonspecific cytotoxicity and haemolytic activity. In this case, why authors choose proper this molecule as control? If the final score is to discover new AMPs that could be used for the development of therapeutic agents, why they did not observe differences with other AMPs, such as the mentioned PAC-113 or dalbavancin, which are already used in this field and that not present cytotoxicity?</strong></p><p id="ff5ba265-b8bd-468d-82e4-69e9dbb2da9c" class="">Due to the big amount of original data only the table with information about viability AMP–treated McCoy cells calculated from fluorescent images was presented in the manuscript. Figure S2 showing the effects obtained with the peptides and of the Melittin control were added in the Supplementary section.</p><p id="76d7f69f-0f14-4d12-b582-f545002d7383" class=""><strong>Figure S2</strong>. Effect of AMPs on the survival of McCoy cells. Cells were treated for 24 h with peptides at a final concentration equal to 4× MIC. After staining with calcein AM and ethidium homodimer–1, samples were analysed by fluorescence microscopy. The peptide melittin was used as a positive control. The negative control corresponds to the McCoy cells incubated without peptide (DMEM).</p><p id="51176e22-d2ac-4f0a-bd93-f252f65de1a0" class="">Figure S3 was replaced with the figure showing results for all 6 peptides treatment during 24 hours.</p><p id="0b0e1f1b-9409-41d1-932c-f215ca8944d2" class=""><strong>Figure </strong><strong>S</strong><strong>3.</strong> Viability of AMP–treated McCoy cells measured by LHD assay. Cells were treated for 24 h with peptides at a final concentration equal to 4× MIC. Melittin was used as a negative control. The positive control corresponds to untreated cells (Control). The values are indicated as the mean ± SD (n = 3). Statistically significant differences between the control and experimental groups were determined by the non–parametric Kruskal–Wallis test, *<em>p</em> &lt; 0.05, ***<em>p</em> &lt; 0.005.</p><p id="6cd98094-f7ab-45a2-a958-3271ea1d55b2" class="">As positive control, we could use other AMPs with nontoxic effect, but in our case was used untreated cells that are more demonstrative as we suppose.</p><ul id="872454a0-fe7e-4806-81e8-4f4758f5cb9f" class="bulleted-list"><li style="list-style-type:disc"><strong>Section 2.5. As before, a figure or images should be added, in which bacteria treated with propidium iodide are visible. Also in this case, the most significant conditions could be used.</strong></li></ul><p id="41877874-efa3-40b6-9231-2446888743de" class=""><strong>Moreover, in these experiments, the effect of Pept_148 were evaluated during the kinetics of PI penetration (Figure 2), while no data are reported about the fluorescent result (Figure S3). Why the authors decided to not consider this peptide in the S3 graphs?</strong></p><p id="374467f0-3182-40b6-808f-d206695ca5dd" class=""><strong>Why the same test has not been conducted on </strong><em><strong>B. haemolyticus</strong></em><strong> in order to compare AMPs effects with SEM results?</strong></p><p id="1659f4ff-0564-4d4d-90bc-0aa53ea13f41" class="">There are too many data to be shown in supplementary section that is why we decided to show only data in table that is more informative. Figure S4 showing the effects of the peptides on bacteria treated with propidium iodide was added in the Supplementary section.</p><p id="73d12a00-9337-48ff-995e-f92bdf271eb2" class=""><strong>Figure S4</strong>. Effect of AMPs on the survival of bacterial cells after peptide treatment. B. subtilis (top row) and E. coli (bottom row) were incubated with peptides at a final concentration equal to 2× MIC at 37 °C for 1 h. After staining with PI, samples were analyzed by fluorescence microscopy. The peptide melittin was used as a positive control. The negative control corresponds to the bacteria incubated without peptide (MHB). The size of each individual frame is 222 μm× 222 μm.</p><p id="c07452ea-a80d-4347-815c-62f3cee3a52b" class="">Pept_148 demonstrated low intensity of PI penetration that is why it is difficult to unambiguously defy the fluorescent signal from background and peptide. Due to this fact the results are controversial and didn’t show in the publication. Moreover, we tested two bacteria <em>E. coli</em> and <em>B. subtilis</em>, and peptide pept_148 didn’t exhibit antimicrobial activity against <em>E. coli</em>.</p><ol type="1" id="f03218de-a7c7-4658-98a0-1bf39eb633b6" class="numbered-list" start="1"><li><em>haemolyticus</em>is very pathogenic and, unfortunately, our laboratory doesn’t equip properly (with the fluorescent microscopes and spectrofluorometer in the specials boxes) for that kind of experiments. That’s the reason why these experiments were not conducted.</li></ol><ul id="1a0bdd32-dccb-495f-864a-8caed035cf12" class="bulleted-list"><li style="list-style-type:disc"><strong>Section 2.6. Why the authors decided to perform SEM analyses on the Pept_84, which not seems to show a great effect on </strong><em><strong>subtilis</strong></em><strong> in the PI assay and that possess high cytotoxicity, rather not considering the Pept_148?</strong></li></ul><p id="f5ea367f-4021-4378-beee-a0dca6868821" class=""><strong>Moreover, the AMP activity tested in the experiments with PI was calculated using the 2x MIC concentration and after a short period of time. Authors suggested a rapid mechanism of action for the antimicrobial peptides. Based on this experimental concentration and timing, why the authors choose 8 hours before observing SEM results and did not used the same concentration for compare the results?</strong></p><p id="b6ada971-ea74-4b85-84eb-483e13587654" class="">As we examined the peptide effect on three bacteria by SEM we decided to investigate four peptides that affected all of them. Due to the preparation the SEM samples the concentration of peptides and bacteria were different from the experiments with PI to obtain sample with the cell monolayers. Moreover, in experiments with the kinetics of PI penetration, we used 2*MIC peptide concentration in order to perform experiments more quickly and avoid the possible effects of fluorescent dye fading.</p><p id="659c1cfe-3477-4df5-a0bb-5a823d23cf3f" class=""><strong>Discussion</strong></p><p id="b7f5d806-b4ac-43bc-ac99-420b48e9d55d" class=""><strong>It seems that these six peptides derive from specific bacterial taxa present in the </strong><em><strong>H. medicinalis </strong></em><strong>microbiome (Aminobacter, Sphingobacteriia and Chitinophagia). Is there a correlation between these precise organisms and AMPs? Have these bacteria a precise role in the Hirudo microbiome? This point should be argument in the discussion.</strong></p><p id="a70cf4cf-2aaf-4f14-a0a5-114f0998c869" class=""><strong>Other important aspects about AMPs are both the positive surface charge and the hydrophobicity, as explained at line 228. Authors should add some information about these points (considering for example the Boman index), analysing the characteristics of the main peptides (or only of Pept_1545).</strong></p><p id="acb4f30b-1771-4a76-8a83-0b5d762eb10c" class="">It is difficult to judge the role of these bacteria for the leech organism based only on metagenomic data; this requires additional experiments for a deeper understanding of the interaction between the host and its microbiota, unfortunately this is outside the scope of our study.</p><p id="0c0ff555-6a08-415c-9cfb-e6d142718c5a" class="">The corresponding discussion about specific bacterial taxa present in the <em>H. medicinalis</em> microbiome and their role in organism was added to the revised manuscript in the Discussion section:</p><p id="b16abd14-2639-4645-9626-dc756565ded7" class="">“The selected peptides belong to the two phylums of Gram-negative bacteria <em>Bacteroidetes</em> (<em>Sphingobacteriia</em> and <em>Chitinophagia</em>) and <em>Proteobacteria</em> (<em>Aminobacter</em>), bacteria which are typically members of host-associated microbial communities. The literature data concern only investigations of metagenomic data and individual genomes from soil samples. There is little information on AMPs identified in these bacteria. At the moment only non-ribosomal antimicrobial peptides isolated from marine <em>Proteobacteria</em> has been known [35]. According to the existing observations and studies on the metagenomes of leeches of the genus <em>Hirudo</em> these taxa of microorganisms are common symbionts of leeches. The genus <em>Aminobacter</em> belongs to the order <em>Rhizobiales</em>, representatives of this order are components of the microbiota from the mucus and skin of the leech [36,37]. Representatives of bacteria of the class <em>Sphingobacteriia</em> and <em>Chitinophagia</em> are associated with metagenomes of the skin and excretory system of leeches [37,38]. For example, bacteria from the genus <em>Niabella</em> (Class <em>Chitinophagia</em>) can be found in leech cocoons, during the development of the leech, these bacteria colonize the bladder of the leech [39]. In some cases, representatives of these classes of bacteria were observed in insignificant quantities in various parts of the leech&#x27;s digestive system [40, 41]. Thus, it can be assumed that these bacteria are endosymbionts and use antimicrobial peptides to compete successfully in the process of colonization of certain leech organs.”</p><p id="6d175f5c-d91f-423c-b749-b985b6261540" class="">Information about physical-chemical characteristics of AMPs was added in the Materials and Methods and Supplementary sections.</p><p id="e7e0eac5-5bfd-47aa-a919-56742e2b8781" class=""><strong>Table S2.</strong> The physical-chemical characteristics of AMPs from the <em>H. medicinalis</em> microbiome.</p><div id="d8465e99-1272-4f63-a8d6-c5ec60c3e844" class="collection-content"><h4 class="collection-title"></h4><table class="collection-content"><thead><tr><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesTitle"><path d="M0.637695 13.1914C1.0957 13.1914 1.32812 13 1.47852 12.5215L2.24414 10.3887H6.14746L6.90625 12.5215C7.05664 13 7.2959 13.1914 7.74707 13.1914C8.22559 13.1914 8.5332 12.9043 8.5332 12.4531C8.5332 12.2891 8.50586 12.1523 8.44434 11.9678L5.41602 3.79199C5.2041 3.21777 4.82129 2.9375 4.19922 2.9375C3.60449 2.9375 3.21484 3.21777 3.0166 3.78516L-0.0322266 12.002C-0.09375 12.1797 -0.121094 12.3232 -0.121094 12.4668C-0.121094 12.918 0.166016 13.1914 0.637695 13.1914ZM2.63379 9.12402L4.17871 4.68066H4.21973L5.76465 9.12402H2.63379ZM12.2793 13.2324C13.3115 13.2324 14.2891 12.6787 14.7129 11.8037H14.7402V12.5762C14.7471 12.9863 15.0273 13.2393 15.4238 13.2393C15.834 13.2393 16.1143 12.9795 16.1143 12.5215V8.00977C16.1143 6.49902 14.9658 5.52148 13.1543 5.52148C11.7666 5.52148 10.6592 6.08887 10.2695 6.99121C10.1943 7.15527 10.1533 7.3125 10.1533 7.46289C10.1533 7.81152 10.4062 8.04395 10.7686 8.04395C11.0215 8.04395 11.2129 7.94824 11.3496 7.73633C11.7529 6.99121 12.2861 6.65625 13.1064 6.65625C14.0977 6.65625 14.6992 7.20996 14.6992 8.1123V8.67285L12.5664 8.7959C10.7686 8.8916 9.77734 9.69824 9.77734 11.0107C9.77734 12.3369 10.8096 13.2324 12.2793 13.2324ZM12.6621 12.1387C11.8008 12.1387 11.2129 11.667 11.2129 10.9561C11.2129 10.2725 11.7598 9.82129 12.7578 9.75977L14.6992 9.62988V10.3203C14.6992 11.3457 13.7969 12.1387 12.6621 12.1387Z"></path></svg></span>Peptide</th><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesText"><path d="M1.56738 3.25879H14.4258C14.7676 3.25879 15.0479 2.97852 15.0479 2.63672C15.0479 2.29492 14.7744 2.02148 14.4258 2.02148H1.56738C1.21875 2.02148 0.952148 2.29492 0.952148 2.63672C0.952148 2.97852 1.22559 3.25879 1.56738 3.25879ZM1.56738 6.84082H14.4258C14.7676 6.84082 15.0479 6.56055 15.0479 6.21875C15.0479 5.87695 14.7744 5.60352 14.4258 5.60352H1.56738C1.21875 5.60352 0.952148 5.87695 0.952148 6.21875C0.952148 6.56055 1.22559 6.84082 1.56738 6.84082ZM1.56738 10.4229H14.4258C14.7676 10.4229 15.0479 10.1426 15.0479 9.80078C15.0479 9.45898 14.7744 9.18555 14.4258 9.18555H1.56738C1.21875 9.18555 0.952148 9.45898 0.952148 9.80078C0.952148 10.1426 1.22559 10.4229 1.56738 10.4229ZM1.56738 14.0049H8.75879C9.10059 14.0049 9.38086 13.7246 9.38086 13.3828C9.38086 13.041 9.10742 12.7676 8.75879 12.7676H1.56738C1.21875 12.7676 0.952148 13.041 0.952148 13.3828C0.952148 13.7246 1.22559 14.0049 1.56738 14.0049Z"></path></svg></span>Amino acid sequence</th><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesNumber"><path d="M2.4834 10.9902H4.33594L3.75488 13.8887C3.74121 13.9639 3.72754 14.0664 3.72754 14.1416C3.72754 14.5381 3.99414 14.7637 4.39746 14.7637C4.81445 14.7637 5.09473 14.5449 5.18359 14.1143L5.80566 10.9902H8.79297L8.21191 13.8887C8.19824 13.9639 8.18457 14.0664 8.18457 14.1416C8.18457 14.5381 8.45117 14.7637 8.85449 14.7637C9.27148 14.7637 9.55176 14.5449 9.63379 14.1143L10.2627 10.9902H12.4502C12.9287 10.9902 13.2432 10.6758 13.2432 10.2109C13.2432 9.8418 12.9902 9.56836 12.6006 9.56836H10.5498L11.2129 6.28711H13.3662C13.8379 6.28711 14.1523 5.96582 14.1523 5.50098C14.1523 5.13184 13.9062 4.8584 13.5098 4.8584H11.5L12.0195 2.27441C12.0264 2.21973 12.0469 2.11035 12.0469 2.01465C12.0469 1.625 11.7666 1.39258 11.3633 1.39258C10.9053 1.39258 10.6797 1.63867 10.5977 2.05566L10.0303 4.8584H7.04297L7.5625 2.27441C7.57617 2.21973 7.58984 2.11035 7.58984 2.01465C7.58984 1.625 7.30957 1.39258 6.91309 1.39258C6.44824 1.39258 6.21582 1.63867 6.13379 2.05566L5.57324 4.8584H3.54297C3.06445 4.8584 2.75 5.18652 2.75 5.65137C2.75 6.03418 3.00293 6.28711 3.39258 6.28711H5.28613L4.62305 9.56836H2.63379C2.15527 9.56836 1.84082 9.89648 1.84082 10.3613C1.84082 10.7373 2.09375 10.9902 2.4834 10.9902ZM6.09277 9.56836L6.75586 6.28711H9.74316L9.08008 9.56836H6.09277Z"></path></svg></span>Charge</th><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesNumber"><path d="M2.4834 10.9902H4.33594L3.75488 13.8887C3.74121 13.9639 3.72754 14.0664 3.72754 14.1416C3.72754 14.5381 3.99414 14.7637 4.39746 14.7637C4.81445 14.7637 5.09473 14.5449 5.18359 14.1143L5.80566 10.9902H8.79297L8.21191 13.8887C8.19824 13.9639 8.18457 14.0664 8.18457 14.1416C8.18457 14.5381 8.45117 14.7637 8.85449 14.7637C9.27148 14.7637 9.55176 14.5449 9.63379 14.1143L10.2627 10.9902H12.4502C12.9287 10.9902 13.2432 10.6758 13.2432 10.2109C13.2432 9.8418 12.9902 9.56836 12.6006 9.56836H10.5498L11.2129 6.28711H13.3662C13.8379 6.28711 14.1523 5.96582 14.1523 5.50098C14.1523 5.13184 13.9062 4.8584 13.5098 4.8584H11.5L12.0195 2.27441C12.0264 2.21973 12.0469 2.11035 12.0469 2.01465C12.0469 1.625 11.7666 1.39258 11.3633 1.39258C10.9053 1.39258 10.6797 1.63867 10.5977 2.05566L10.0303 4.8584H7.04297L7.5625 2.27441C7.57617 2.21973 7.58984 2.11035 7.58984 2.01465C7.58984 1.625 7.30957 1.39258 6.91309 1.39258C6.44824 1.39258 6.21582 1.63867 6.13379 2.05566L5.57324 4.8584H3.54297C3.06445 4.8584 2.75 5.18652 2.75 5.65137C2.75 6.03418 3.00293 6.28711 3.39258 6.28711H5.28613L4.62305 9.56836H2.63379C2.15527 9.56836 1.84082 9.89648 1.84082 10.3613C1.84082 10.7373 2.09375 10.9902 2.4834 10.9902ZM6.09277 9.56836L6.75586 6.28711H9.74316L9.08008 9.56836H6.09277Z"></path></svg></span>(Boman index), kcal/mol</th><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesNumber"><path d="M2.4834 10.9902H4.33594L3.75488 13.8887C3.74121 13.9639 3.72754 14.0664 3.72754 14.1416C3.72754 14.5381 3.99414 14.7637 4.39746 14.7637C4.81445 14.7637 5.09473 14.5449 5.18359 14.1143L5.80566 10.9902H8.79297L8.21191 13.8887C8.19824 13.9639 8.18457 14.0664 8.18457 14.1416C8.18457 14.5381 8.45117 14.7637 8.85449 14.7637C9.27148 14.7637 9.55176 14.5449 9.63379 14.1143L10.2627 10.9902H12.4502C12.9287 10.9902 13.2432 10.6758 13.2432 10.2109C13.2432 9.8418 12.9902 9.56836 12.6006 9.56836H10.5498L11.2129 6.28711H13.3662C13.8379 6.28711 14.1523 5.96582 14.1523 5.50098C14.1523 5.13184 13.9062 4.8584 13.5098 4.8584H11.5L12.0195 2.27441C12.0264 2.21973 12.0469 2.11035 12.0469 2.01465C12.0469 1.625 11.7666 1.39258 11.3633 1.39258C10.9053 1.39258 10.6797 1.63867 10.5977 2.05566L10.0303 4.8584H7.04297L7.5625 2.27441C7.57617 2.21973 7.58984 2.11035 7.58984 2.01465C7.58984 1.625 7.30957 1.39258 6.91309 1.39258C6.44824 1.39258 6.21582 1.63867 6.13379 2.05566L5.57324 4.8584H3.54297C3.06445 4.8584 2.75 5.18652 2.75 5.65137C2.75 6.03418 3.00293 6.28711 3.39258 6.28711H5.28613L4.62305 9.56836H2.63379C2.15527 9.56836 1.84082 9.89648 1.84082 10.3613C1.84082 10.7373 2.09375 10.9902 2.4834 10.9902ZM6.09277 9.56836L6.75586 6.28711H9.74316L9.08008 9.56836H6.09277Z"></path></svg></span>Hydrophobic ratio, %</th></tr></thead><tbody><tr id="64bba377-ba6a-48b3-ac40-aa00eff93fb3"><td class="cell-title"><a href="pept_1303%2064bba377ba6a48b3ac40aa00eff93fb3.html">pept_1303</a></td><td class="cell-wfwj">IGRHFKRRNSIWGICWF</td><td class="cell-sP;|">4</td><td class="cell-bqLM">2.14</td><td class="cell-oqeK">47</td></tr><tr id="be0b61b0-1798-43a5-9aa5-2b2590a2f32c"><td class="cell-title"><a href="pept_148%20be0b61b0179843a59aa52b2590a2f32c.html">pept_148</a></td><td class="cell-wfwj">VLIRGLIHMLRGG</td><td class="cell-sP;|">2</td><td class="cell-bqLM">0.05</td><td class="cell-oqeK">53</td></tr><tr id="763b3a8e-6ace-409b-b9a4-6c50b621dd70"><td class="cell-title"><a href="pept_970%20763b3a8e6ace409bb9a46c50b621dd70.html">pept_970</a></td><td class="cell-wfwj">FVKILAKLVNYAKN</td><td class="cell-sP;|">3</td><td class="cell-bqLM">0.04</td><td class="cell-oqeK">57</td></tr><tr id="e82c5f45-b85e-460c-be42-c772acc18399"><td class="cell-title"><a href="pept_1545%20e82c5f45b85e460cbe42c772acc18399.html">pept_1545</a></td><td class="cell-wfwj">FLIGKAIKRKFCLRSVWNA</td><td class="cell-sP;|">5</td><td class="cell-bqLM">0.98</td><td class="cell-oqeK">57</td></tr><tr id="0ed0eb04-4f14-4f01-9699-1b00460edd06"><td class="cell-title"><a href="pept_352%200ed0eb044f144f0196991b00460edd06.html">pept_352</a></td><td class="cell-wfwj">KKGKSFKQLHIIVHLVKSWLRTILTHI</td><td class="cell-sP;|">6</td><td class="cell-bqLM">0.75</td><td class="cell-oqeK">44</td></tr><tr id="3a646518-040e-4cb8-a368-ba9f62d3ec76"><td class="cell-title"><a href="pept_84%203a646518040e4cb8a368ba9f62d3ec76.html">pept_84</a></td><td class="cell-wfwj">IVKRFFRISYKLQSLKIIKGKRTFT</td><td class="cell-sP;|">8</td><td class="cell-bqLM">1.86</td><td class="cell-oqeK">40</td></tr></tbody></table></div><p id="dcd4814b-196f-4a1f-b80b-5958306f04fa" class="">Analysis of physical-chemical characteristics of peptides demonstrated that all peptides have a positive net charge, the value of Boman index is varied from 0.04 to 2.14 and hydrophobic ratio is about 40% and higher. There is no reliable correlation between these parameters and antimicrobial action of peptides [Boman HG. Antibacterial peptides: basic facts and emerging concepts. J Intern Med. 2003;254(3):197-215. doi:10.1046/j.1365-2796.2003.01228.x]. We assume, there is no point in adding such information in the discussion.</p><p id="11671d22-53bd-4336-a20c-8199f6cd1328" class="">The sentences “As a result, we chose 5 peptides with the highest antimicrobial propensity scores which can exhibit α–helical structure and additionally 1 peptide with unstructured conformation and high propensity score according to the predictions. The physicochemical properties of peptides were calculated using an APD3 server for the amino acid composition, hydrophobic ratio, Boman index, and molecular weight [4] (Table S2). “ were added in the Materials and Methods section.</p><p id="25198051-518b-4e8c-bd0c-2a7e560b6946" class="">The corresponding references were added to the revised version of the article.</p><p id="ea993325-7af2-4028-ab0f-b06b4b346097" class="">Author Response File: <a href="https://susy.mdpi.com/user/review/displayFile/13943547/YyfnIQX9?file=author-coverletter&amp;report=8667721"> Author Response.docx</a></p><p id="717ca5de-87d2-4f8f-9d19-4fea83f7e17a" class=""><em>Reviewer 2 Report</em></p><p id="713844c2-ea89-40a2-89b5-572744b081c3" class="">The work <em>&quot;The Hirudo Medicinalis Microbiome Is a Source of New Antimicrobial Peptides&quot;</em> by Grafskaia et al. presents in a clear fashion interesting results regarding antimicrobial activity of peptides whose sequences have been inferred from metagenomic study of <em>Hirudo medicinalis</em> microbiome.</p><p id="3a2f3b98-c694-4fe1-ba15-5e17008503a8" class="">I do not have any objections regarding presentation or interpretation of the results. However a few minor remarks have come to my mind.</p><ol type="1" id="9bbb6035-03a0-4026-83b5-40668e525f7d" class="numbered-list" start="1"><li>In materials and methods there is a significant gap regarding how the final 6 peptides were selected. In the result section there is a transition from 65 to 6 peptides without giving any rationale behind it. Please make it clear in the final manuscript.</li></ol><ol type="1" id="848cc328-fad4-4938-bee6-d3e4e06ed385" class="numbered-list" start="2"><li>As the most active AMPs do not form alpha-helices in the examined conditions, what could be their potential mechanism of action? Is there a possibility they form alpha-helices during interaction with bacterial membranes of concrete species only? Wouldn&#x27;t be relatively easy to obtain liposomes from the examined bacterial species and investigate such a possibility? Could authors deliver such an experiment for the investigated peptides?</li></ol><ol type="1" id="ebbf1dfd-a283-4205-9cba-52837559fd6e" class="numbered-list" start="3"><li>What could be a potential reason for the quite promising peptide 1545 to be highly effective towards <em>S. haemolyticus</em> but completely ineffective towards <em>S. aureus</em>? Could authors include in the manuscript any interpretation of this difference based on available research?</li></ol><ol type="1" id="0bb3eacb-eec1-4da5-ad09-ef2017179a62" class="numbered-list" start="4"><li>Is it possible to suggest how should the peptide 1545 be modified in order to make it effective towards <em>S. aureus</em>? Would it be possible to include such a suggestion or a few in the discussion?</li></ol><p id="abf073c9-be71-42c0-b4bf-bfae188bcc0f" class=""><em>Author Response</em></p><p id="3691f1e6-02ce-485c-a27e-ed46ed2b790f" class="">We would like to thank the reviewer for detailed comments regarding the manuscript. We agree that dataset analysis is not totally clear and we implemented corresponding modifications. Responding on general suggestions we want to underline that our paper discusses the identification and functional analysis of the new antimicrobial peptides of <em>Hirudo medicinalis</em>microbiome. Below you can find the detailed responses to the issues. Original reviewer comments in boldface, responses in regular typeface.</p><p id="72732af8-98f5-43e7-81dc-abd29c7be0db" class=""><strong>The work </strong><em><strong>&quot;The Hirudo Medicinalis Microbiome Is a Source of New Antimicrobial Peptides&quot;</strong></em><strong> by Grafskaia et al. presents in a clear fashion interesting results regarding antimicrobial activity of peptides whose sequences have been inferred from metagenomic study of </strong><em><strong>Hirudo medicinalis </strong></em><strong>microbiome.</strong></p><p id="7522a38f-e1fc-45dc-9159-77cbcbebb07a" class=""><strong>I do not have any objections regarding presentation or interpretation of the results. However a few minor remarks have come to my mind.</strong></p><ol type="1" id="d328e2d2-1e3d-441f-83a0-7a2e5ca11d45" class="numbered-list" start="1"><li><strong>In materials and methods there is a significant gap regarding how the final 6 peptides were selected. In the result section there is a transition from 65 to 6 peptides without giving any rationale behind it. Please make it clear in the final manuscript.</strong></li></ol><p id="069ca7ae-52ab-47ee-a72c-5e23436edb80" class="">The structures of AMPs are various but the majority of known AMPs form α–helix. Among filtered 65 peptides with the highest antimicrobial propensity scores the predicted secondary structures were only α–helical or unstructured conformations. According to our previous paper the highest AMP propensity calculated by predictors doesn’t guarantee the best antimicrobial action. That is why peptides were chosen not by only their potential effects but on secondary structure too. From 65 peptides we chose 5 peptides with the highest antimicrobial propensity scores which can exhibit α–helical structure and additionally 1 peptide with unstructured conformation and high propensity score according to the predictions.</p><p id="a11e2228-c396-442d-a04a-0eb3c093616b" class="">The sentences “Overall, as potential AMPs, we identified 6 amino acid sequences (Table 1). Five of these peptides can adopt an α–helical structure under prediction. The leftover peptide is unstructured.” in the Result section were replaced by “According to the secondary structure prediction among filtered 65 peptides with the highest antimicrobial propensity scores the predicted secondary structures were only α–helical or unstructured conformations. From 65 peptides we chose 5 peptides with the highest antimicrobial propensity scores which can exhibit α–helical structure and additionally 1 peptide with unstructured conformation and high propensity score according to the predictions (Table 1).”</p><p id="bade70fa-6eaa-4a62-8403-67aa49155cd0" class="">The sentence “As a result, we selected six putative α–helical AMPs.” in the Discussion section was corrected to “As a result, we selected six putative AMPs.”</p><ol type="1" id="e6d35dcc-9893-4ebe-bee9-7e199662657f" class="numbered-list" start="1"><li><strong>As the most active AMPs do not form alpha-helices in the examined conditions, what could be their potential mechanism of action? Is there a possibility they form alpha-helices during interaction with bacterial membranes of concrete species only? Wouldn&#x27;t be relatively easy to obtain liposomes from the examined bacterial species and investigate such a possibility? Could authors deliver such an experiment for the investigated peptides?</strong></li></ol><p id="c40bc884-6cb2-46b1-87c7-0e12f39d8cc9" class="">Due to the mechanism of action is dependent on many factors as secondary structure, type of pathogens, environmental conditions and peptide concentration it is difficult to define the actual mechanism of AMP. On our findings, we only can assume that the mechanism of the most active AMP is likely non membranolytic but it must be proved by additional experiments. It is a fair point that the experiments with liposomes of different composition mimicking various bacteria could clarify the precise mechanism of action. In our previous study, we demonstrated that AMPs indeed form alpha-helices in the lipid environment [Grafskaia, E. N.; Nadezhdin, K. D.; Talyzina, I. A.; Polina, N. F.; Podgorny, O. V.; Pavlova, E. R.; Bashkirov, P. V.; Kharlampieva, D. D.; Bobrovsky, P. A.; Latsis, I. A.; Manuvera, V. A.; Babenko, V. V.; Trukhan, V. M.; Arseniev, A. S.; Klinov, D. V.; Lazarev, V. N. Medicinal leech antimicrobial peptides lacking toxicity represent a promising alternative strategy to combat antibiotic–resistant pathogens. Eur. J. Med. Chem. 2019, 180, 143–153. https://doi.org/10.1016/j.ejmech.2019.06.080.]. The AMPs interacted only with anionic lipids mimicking the bacteria but did not effect on zwitterionic vesicles which simulated eukaryotic cell membrane. Moreover, the secondary structure predictions of peptides didn’t take into account the environmental conditions, but that factor has a profound effect on the peptide conformation.</p><ol type="1" id="7226d71d-97a0-4e94-91d0-49d048e2bc72" class="numbered-list" start="1"><li><strong>What could be a potential reason for the quite promising peptide 1545 to be highly effective towards </strong><em><strong>S. haemolyticus</strong></em><strong> but completely ineffective towards </strong><em><strong>S. aureus</strong></em><strong>? Could authors include in the manuscript any interpretation of this difference based on available research?</strong></li></ol><p id="25bfb0c7-eddb-4e8e-9cff-87bf3aefca2b" class="">We assume that the membrane composition of <em>S. haemolyticus</em> and <em>S. aureus</em> differ that results in different interaction with peptide and potential relevance to antibiotic response. Moreover, the considerable plasticity of <em>S. aureus</em> membrane lipid composition affects the response and resistance to various antimicrobial agents [Durham, D.R.; Kloos, W.E. Comparative study of the total cellular fatty acids of Staphylococcus species of human origin. Int. J. Syst. Bateriol. 1978, 28, 223–228, Tiwari, Kiran B et al. “Plasticity of Coagulase-Negative Staphylococcal Membrane Fatty Acid Composition and Implications for Responses to Antimicrobial Agents.” Antibiotics (Basel, Switzerland) vol. 9,5 214. 28 Apr. 2020, doi:10.3390/antibiotics9050214].</p><p id="7fe9c1f0-987d-4f38-927a-767cf2b7a4a7" class="">The sentences “We suppose that the difference between the response of <em>S. haemolyticus</em> and <em>S. aureus</em> to peptides dependent of membrane lipid composition of bacteria. The considerable membrane plasticity of <em>S. aureus </em>can provide antimicrobial resistance to peptides [43,44].” were added in the Discussion section.</p><ol type="1" id="624d497f-6e75-4c67-914c-4c616362bd85" class="numbered-list" start="1"><li><strong>Is it possible to suggest how should the peptide 1545 be modified in order to make it effective towards </strong><em><strong>S. aureus</strong></em><strong>? Would it be possible to include such a suggestion or a few in the discussion?</strong></li></ol><p id="34921457-dbfa-42f9-9d55-6fd5029b0350" class="">After the precise investigation of <em>Staphylococcus</em> membrane composition, it can be possible to define proper amino acid substitution which probably improves the antimicrobial activity. On the other hand, this substitution can change the toxicity of the peptides that should be examined as well.</p><p id="095530d7-8c46-4270-be67-77bf6487f0ec" class="">The corresponding references were added to the revised version of the article.</p><p id="2dc63b75-f115-4ab7-9226-1da2acb1b5a5" class="">Author Response File: <a href="https://susy.mdpi.com/user/review/displayFile/14048281/RBZ9QhSF?file=author-coverletter&amp;report=8752032"> Author Response.docx</a></p></div></article></body></html>