<html><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><title>Viruses | Free Full-Text | First Isolation of a Giant Virus from Wild Hirudo medicinalis Leech: Mimiviridae isolation in Hirudo medicinalis | HTML</title><style>
/* cspell:disable-file */
/* webkit printing magic: print all background colors */
html {
	-webkit-print-color-adjust: exact;
}
* {
	box-sizing: border-box;
	-webkit-print-color-adjust: exact;
}

html,
body {
	margin: 0;
	padding: 0;
}
@media only screen {
	body {
		margin: 2em auto;
		max-width: 900px;
		color: rgb(55, 53, 47);
	}
}

body {
	line-height: 1.5;
	white-space: pre-wrap;
}

a,
a.visited {
	color: inherit;
	text-decoration: underline;
}

.pdf-relative-link-path {
	font-size: 80%;
	color: #444;
}

h1,
h2,
h3 {
	letter-spacing: -0.01em;
	line-height: 1.2;
	font-weight: 600;
	margin-bottom: 0;
}

.page-title {
	font-size: 2.5rem;
	font-weight: 700;
	margin-top: 0;
	margin-bottom: 0.75em;
}

h1 {
	font-size: 1.875rem;
	margin-top: 1.875rem;
}

h2 {
	font-size: 1.5rem;
	margin-top: 1.5rem;
}

h3 {
	font-size: 1.25rem;
	margin-top: 1.25rem;
}

.source {
	border: 1px solid #ddd;
	border-radius: 3px;
	padding: 1.5em;
	word-break: break-all;
}

.callout {
	border-radius: 3px;
	padding: 1rem;
}

figure {
	margin: 1.25em 0;
	page-break-inside: avoid;
}

figcaption {
	opacity: 0.5;
	font-size: 85%;
	margin-top: 0.5em;
}

mark {
	background-color: transparent;
}

.indented {
	padding-left: 1.5em;
}

hr {
	background: transparent;
	display: block;
	width: 100%;
	height: 1px;
	visibility: visible;
	border: none;
	border-bottom: 1px solid rgba(55, 53, 47, 0.09);
}

img {
	max-width: 100%;
}

@media only print {
	img {
		max-height: 100vh;
		object-fit: contain;
	}
}

@page {
	margin: 1in;
}

.collection-content {
	font-size: 0.875rem;
}

.column-list {
	display: flex;
	justify-content: space-between;
}

.column {
	padding: 0 1em;
}

.column:first-child {
	padding-left: 0;
}

.column:last-child {
	padding-right: 0;
}

.table_of_contents-item {
	display: block;
	font-size: 0.875rem;
	line-height: 1.3;
	padding: 0.125rem;
}

.table_of_contents-indent-1 {
	margin-left: 1.5rem;
}

.table_of_contents-indent-2 {
	margin-left: 3rem;
}

.table_of_contents-indent-3 {
	margin-left: 4.5rem;
}

.table_of_contents-link {
	text-decoration: none;
	opacity: 0.7;
	border-bottom: 1px solid rgba(55, 53, 47, 0.18);
}

table,
th,
td {
	border: 1px solid rgba(55, 53, 47, 0.09);
	border-collapse: collapse;
}

table {
	border-left: none;
	border-right: none;
}

th,
td {
	font-weight: normal;
	padding: 0.25em 0.5em;
	line-height: 1.5;
	min-height: 1.5em;
	text-align: left;
}

th {
	color: rgba(55, 53, 47, 0.6);
}

ol,
ul {
	margin: 0;
	margin-block-start: 0.6em;
	margin-block-end: 0.6em;
}

li > ol:first-child,
li > ul:first-child {
	margin-block-start: 0.6em;
}

ul > li {
	list-style: disc;
}

ul.to-do-list {
	padding-inline-start: 0;
}

ul.to-do-list > li {
	list-style: none;
}

.to-do-children-checked {
	text-decoration: line-through;
	opacity: 0.375;
}

ul.toggle > li {
	list-style: none;
}

ul {
	padding-inline-start: 1.7em;
}

ul > li {
	padding-left: 0.1em;
}

ol {
	padding-inline-start: 1.6em;
}

ol > li {
	padding-left: 0.2em;
}

.mono ol {
	padding-inline-start: 2em;
}

.mono ol > li {
	text-indent: -0.4em;
}

.toggle {
	padding-inline-start: 0em;
	list-style-type: none;
}

/* Indent toggle children */
.toggle > li > details {
	padding-left: 1.7em;
}

.toggle > li > details > summary {
	margin-left: -1.1em;
}

.selected-value {
	display: inline-block;
	padding: 0 0.5em;
	background: rgba(206, 205, 202, 0.5);
	border-radius: 3px;
	margin-right: 0.5em;
	margin-top: 0.3em;
	margin-bottom: 0.3em;
	white-space: nowrap;
}

.collection-title {
	display: inline-block;
	margin-right: 1em;
}

.page-description {
    margin-bottom: 2em;
}

.simple-table {
	margin-top: 1em;
	font-size: 0.875rem;
	empty-cells: show;
}
.simple-table td {
	height: 29px;
	min-width: 120px;
}

.simple-table th {
	height: 29px;
	min-width: 120px;
}

.simple-table-header-color {
	background: rgb(247, 246, 243);
	color: black;
}
.simple-table-header {
	font-weight: 500;
}

time {
	opacity: 0.5;
}

.icon {
	display: inline-block;
	max-width: 1.2em;
	max-height: 1.2em;
	text-decoration: none;
	vertical-align: text-bottom;
	margin-right: 0.5em;
}

img.icon {
	border-radius: 3px;
}

.user-icon {
	width: 1.5em;
	height: 1.5em;
	border-radius: 100%;
	margin-right: 0.5rem;
}

.user-icon-inner {
	font-size: 0.8em;
}

.text-icon {
	border: 1px solid #000;
	text-align: center;
}

.page-cover-image {
	display: block;
	object-fit: cover;
	width: 100%;
	max-height: 30vh;
}

.page-header-icon {
	font-size: 3rem;
	margin-bottom: 1rem;
}

.page-header-icon-with-cover {
	margin-top: -0.72em;
	margin-left: 0.07em;
}

.page-header-icon img {
	border-radius: 3px;
}

.link-to-page {
	margin: 1em 0;
	padding: 0;
	border: none;
	font-weight: 500;
}

p > .user {
	opacity: 0.5;
}

td > .user,
td > time {
	white-space: nowrap;
}

input[type="checkbox"] {
	transform: scale(1.5);
	margin-right: 0.6em;
	vertical-align: middle;
}

p {
	margin-top: 0.5em;
	margin-bottom: 0.5em;
}

.image {
	border: none;
	margin: 1.5em 0;
	padding: 0;
	border-radius: 0;
	text-align: center;
}

.code,
code {
	background: rgba(135, 131, 120, 0.15);
	border-radius: 3px;
	padding: 0.2em 0.4em;
	border-radius: 3px;
	font-size: 85%;
	tab-size: 2;
}

code {
	color: #eb5757;
}

.code {
	padding: 1.5em 1em;
}

.code-wrap {
	white-space: pre-wrap;
	word-break: break-all;
}

.code > code {
	background: none;
	padding: 0;
	font-size: 100%;
	color: inherit;
}

blockquote {
	font-size: 1.25em;
	margin: 1em 0;
	padding-left: 1em;
	border-left: 3px solid rgb(55, 53, 47);
}

.bookmark {
	text-decoration: none;
	max-height: 8em;
	padding: 0;
	display: flex;
	width: 100%;
	align-items: stretch;
}

.bookmark-title {
	font-size: 0.85em;
	overflow: hidden;
	text-overflow: ellipsis;
	height: 1.75em;
	white-space: nowrap;
}

.bookmark-text {
	display: flex;
	flex-direction: column;
}

.bookmark-info {
	flex: 4 1 180px;
	padding: 12px 14px 14px;
	display: flex;
	flex-direction: column;
	justify-content: space-between;
}

.bookmark-image {
	width: 33%;
	flex: 1 1 180px;
	display: block;
	position: relative;
	object-fit: cover;
	border-radius: 1px;
}

.bookmark-description {
	color: rgba(55, 53, 47, 0.6);
	font-size: 0.75em;
	overflow: hidden;
	max-height: 4.5em;
	word-break: break-word;
}

.bookmark-href {
	font-size: 0.75em;
	margin-top: 0.25em;
}

.sans { font-family: ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol"; }
.code { font-family: "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace; }
.serif { font-family: Lyon-Text, Georgia, ui-serif, serif; }
.mono { font-family: iawriter-mono, Nitti, Menlo, Courier, monospace; }
.pdf .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK JP'; }
.pdf:lang(zh-CN) .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK SC'; }
.pdf:lang(zh-TW) .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK TC'; }
.pdf:lang(ko-KR) .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK KR'; }
.pdf .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK JP'; }
.pdf:lang(zh-CN) .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK SC'; }
.pdf:lang(zh-TW) .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK TC'; }
.pdf:lang(ko-KR) .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK KR'; }
.pdf .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK JP'; }
.pdf:lang(zh-CN) .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK SC'; }
.pdf:lang(zh-TW) .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK TC'; }
.pdf:lang(ko-KR) .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK KR'; }
.pdf .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK JP'; }
.pdf:lang(zh-CN) .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK SC'; }
.pdf:lang(zh-TW) .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK TC'; }
.pdf:lang(ko-KR) .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK KR'; }
.highlight-default {
	color: rgba(55, 53, 47, 1);
}
.highlight-gray {
	color: rgba(120, 119, 116, 1);
	fill: rgba(120, 119, 116, 1);
}
.highlight-brown {
	color: rgba(159, 107, 83, 1);
	fill: rgba(159, 107, 83, 1);
}
.highlight-orange {
	color: rgba(217, 115, 13, 1);
	fill: rgba(217, 115, 13, 1);
}
.highlight-yellow {
	color: rgba(203, 145, 47, 1);
	fill: rgba(203, 145, 47, 1);
}
.highlight-teal {
	color: rgba(68, 131, 97, 1);
	fill: rgba(68, 131, 97, 1);
}
.highlight-blue {
	color: rgba(51, 126, 169, 1);
	fill: rgba(51, 126, 169, 1);
}
.highlight-purple {
	color: rgba(144, 101, 176, 1);
	fill: rgba(144, 101, 176, 1);
}
.highlight-pink {
	color: rgba(193, 76, 138, 1);
	fill: rgba(193, 76, 138, 1);
}
.highlight-red {
	color: rgba(212, 76, 71, 1);
	fill: rgba(212, 76, 71, 1);
}
.highlight-gray_background {
	background: rgba(241, 241, 239, 1);
}
.highlight-brown_background {
	background: rgba(244, 238, 238, 1);
}
.highlight-orange_background {
	background: rgba(251, 236, 221, 1);
}
.highlight-yellow_background {
	background: rgba(251, 243, 219, 1);
}
.highlight-teal_background {
	background: rgba(237, 243, 236, 1);
}
.highlight-blue_background {
	background: rgba(231, 243, 248, 1);
}
.highlight-purple_background {
	background: rgba(244, 240, 247, 0.8);
}
.highlight-pink_background {
	background: rgba(249, 238, 243, 0.8);
}
.highlight-red_background {
	background: rgba(253, 235, 236, 1);
}
.block-color-default {
	color: inherit;
	fill: inherit;
}
.block-color-gray {
	color: rgba(120, 119, 116, 1);
	fill: rgba(120, 119, 116, 1);
}
.block-color-brown {
	color: rgba(159, 107, 83, 1);
	fill: rgba(159, 107, 83, 1);
}
.block-color-orange {
	color: rgba(217, 115, 13, 1);
	fill: rgba(217, 115, 13, 1);
}
.block-color-yellow {
	color: rgba(203, 145, 47, 1);
	fill: rgba(203, 145, 47, 1);
}
.block-color-teal {
	color: rgba(68, 131, 97, 1);
	fill: rgba(68, 131, 97, 1);
}
.block-color-blue {
	color: rgba(51, 126, 169, 1);
	fill: rgba(51, 126, 169, 1);
}
.block-color-purple {
	color: rgba(144, 101, 176, 1);
	fill: rgba(144, 101, 176, 1);
}
.block-color-pink {
	color: rgba(193, 76, 138, 1);
	fill: rgba(193, 76, 138, 1);
}
.block-color-red {
	color: rgba(212, 76, 71, 1);
	fill: rgba(212, 76, 71, 1);
}
.block-color-gray_background {
	background: rgba(241, 241, 239, 1);
}
.block-color-brown_background {
	background: rgba(244, 238, 238, 1);
}
.block-color-orange_background {
	background: rgba(251, 236, 221, 1);
}
.block-color-yellow_background {
	background: rgba(251, 243, 219, 1);
}
.block-color-teal_background {
	background: rgba(237, 243, 236, 1);
}
.block-color-blue_background {
	background: rgba(231, 243, 248, 1);
}
.block-color-purple_background {
	background: rgba(244, 240, 247, 0.8);
}
.block-color-pink_background {
	background: rgba(249, 238, 243, 0.8);
}
.block-color-red_background {
	background: rgba(253, 235, 236, 1);
}
.select-value-color-interactiveBlue { background-color: rgba(35, 131, 226, .07); }
.select-value-color-pink { background-color: rgba(245, 224, 233, 1); }
.select-value-color-purple { background-color: rgba(232, 222, 238, 1); }
.select-value-color-green { background-color: rgba(219, 237, 219, 1); }
.select-value-color-gray { background-color: rgba(227, 226, 224, 1); }
.select-value-color-translucentGray { background-color: rgba(255, 255, 255, 0.0375); }
.select-value-color-orange { background-color: rgba(250, 222, 201, 1); }
.select-value-color-brown { background-color: rgba(238, 224, 218, 1); }
.select-value-color-red { background-color: rgba(255, 226, 221, 1); }
.select-value-color-yellow { background-color: rgba(253, 236, 200, 1); }
.select-value-color-blue { background-color: rgba(211, 229, 239, 1); }

.checkbox {
	display: inline-flex;
	vertical-align: text-bottom;
	width: 16;
	height: 16;
	background-size: 16px;
	margin-left: 2px;
	margin-right: 5px;
}

.checkbox-on {
	background-image: url("data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2216%22%20height%3D%2216%22%20viewBox%3D%220%200%2016%2016%22%20fill%3D%22none%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%0A%3Crect%20width%3D%2216%22%20height%3D%2216%22%20fill%3D%22%2358A9D7%22%2F%3E%0A%3Cpath%20d%3D%22M6.71429%2012.2852L14%204.9995L12.7143%203.71436L6.71429%209.71378L3.28571%206.2831L2%207.57092L6.71429%2012.2852Z%22%20fill%3D%22white%22%2F%3E%0A%3C%2Fsvg%3E");
}

.checkbox-off {
	background-image: url("data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2216%22%20height%3D%2216%22%20viewBox%3D%220%200%2016%2016%22%20fill%3D%22none%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%0A%3Crect%20x%3D%220.75%22%20y%3D%220.75%22%20width%3D%2214.5%22%20height%3D%2214.5%22%20fill%3D%22white%22%20stroke%3D%22%2336352F%22%20stroke-width%3D%221.5%22%2F%3E%0A%3C%2Fsvg%3E");
}
	
</style></head><body><article id="8c8ca38e-3f7f-4be8-8fc1-c78ecebe06e5" class="page sans"><header><h1 class="page-title">Viruses | Free Full-Text | First Isolation of a Giant Virus from Wild Hirudo medicinalis Leech: Mimiviridae isolation in Hirudo medicinalis | HTML</h1><p class="page-description"></p><table class="properties"><tbody><tr class="property-row property-row-multi_select"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesMultipleSelect"><path d="M1.91602 4.83789C2.44238 4.83789 2.87305 4.40723 2.87305 3.87402C2.87305 3.34766 2.44238 2.91699 1.91602 2.91699C1.38281 2.91699 0.952148 3.34766 0.952148 3.87402C0.952148 4.40723 1.38281 4.83789 1.91602 4.83789ZM5.1084 4.52344H14.3984C14.7607 4.52344 15.0479 4.23633 15.0479 3.87402C15.0479 3.51172 14.7607 3.22461 14.3984 3.22461H5.1084C4.74609 3.22461 4.45898 3.51172 4.45898 3.87402C4.45898 4.23633 4.74609 4.52344 5.1084 4.52344ZM1.91602 9.03516C2.44238 9.03516 2.87305 8.60449 2.87305 8.07129C2.87305 7.54492 2.44238 7.11426 1.91602 7.11426C1.38281 7.11426 0.952148 7.54492 0.952148 8.07129C0.952148 8.60449 1.38281 9.03516 1.91602 9.03516ZM5.1084 8.7207H14.3984C14.7607 8.7207 15.0479 8.43359 15.0479 8.07129C15.0479 7.70898 14.7607 7.42188 14.3984 7.42188H5.1084C4.74609 7.42188 4.45898 7.70898 4.45898 8.07129C4.45898 8.43359 4.74609 8.7207 5.1084 8.7207ZM1.91602 13.2324C2.44238 13.2324 2.87305 12.8018 2.87305 12.2686C2.87305 11.7422 2.44238 11.3115 1.91602 11.3115C1.38281 11.3115 0.952148 11.7422 0.952148 12.2686C0.952148 12.8018 1.38281 13.2324 1.91602 13.2324ZM5.1084 12.918H14.3984C14.7607 12.918 15.0479 12.6309 15.0479 12.2686C15.0479 11.9062 14.7607 11.6191 14.3984 11.6191H5.1084C4.74609 11.6191 4.45898 11.9062 4.45898 12.2686C4.45898 12.6309 4.74609 12.918 5.1084 12.918Z"></path></svg></span>Journal/Publisher</th><td></td></tr><tr class="property-row property-row-number"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesNumber"><path d="M2.4834 10.9902H4.33594L3.75488 13.8887C3.74121 13.9639 3.72754 14.0664 3.72754 14.1416C3.72754 14.5381 3.99414 14.7637 4.39746 14.7637C4.81445 14.7637 5.09473 14.5449 5.18359 14.1143L5.80566 10.9902H8.79297L8.21191 13.8887C8.19824 13.9639 8.18457 14.0664 8.18457 14.1416C8.18457 14.5381 8.45117 14.7637 8.85449 14.7637C9.27148 14.7637 9.55176 14.5449 9.63379 14.1143L10.2627 10.9902H12.4502C12.9287 10.9902 13.2432 10.6758 13.2432 10.2109C13.2432 9.8418 12.9902 9.56836 12.6006 9.56836H10.5498L11.2129 6.28711H13.3662C13.8379 6.28711 14.1523 5.96582 14.1523 5.50098C14.1523 5.13184 13.9062 4.8584 13.5098 4.8584H11.5L12.0195 2.27441C12.0264 2.21973 12.0469 2.11035 12.0469 2.01465C12.0469 1.625 11.7666 1.39258 11.3633 1.39258C10.9053 1.39258 10.6797 1.63867 10.5977 2.05566L10.0303 4.8584H7.04297L7.5625 2.27441C7.57617 2.21973 7.58984 2.11035 7.58984 2.01465C7.58984 1.625 7.30957 1.39258 6.91309 1.39258C6.44824 1.39258 6.21582 1.63867 6.13379 2.05566L5.57324 4.8584H3.54297C3.06445 4.8584 2.75 5.18652 2.75 5.65137C2.75 6.03418 3.00293 6.28711 3.39258 6.28711H5.28613L4.62305 9.56836H2.63379C2.15527 9.56836 1.84082 9.89648 1.84082 10.3613C1.84082 10.7373 2.09375 10.9902 2.4834 10.9902ZM6.09277 9.56836L6.75586 6.28711H9.74316L9.08008 9.56836H6.09277Z"></path></svg></span>Year</th><td></td></tr><tr class="property-row property-row-text"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesText"><path d="M1.56738 3.25879H14.4258C14.7676 3.25879 15.0479 2.97852 15.0479 2.63672C15.0479 2.29492 14.7744 2.02148 14.4258 2.02148H1.56738C1.21875 2.02148 0.952148 2.29492 0.952148 2.63672C0.952148 2.97852 1.22559 3.25879 1.56738 3.25879ZM1.56738 6.84082H14.4258C14.7676 6.84082 15.0479 6.56055 15.0479 6.21875C15.0479 5.87695 14.7744 5.60352 14.4258 5.60352H1.56738C1.21875 5.60352 0.952148 5.87695 0.952148 6.21875C0.952148 6.56055 1.22559 6.84082 1.56738 6.84082ZM1.56738 10.4229H14.4258C14.7676 10.4229 15.0479 10.1426 15.0479 9.80078C15.0479 9.45898 14.7744 9.18555 14.4258 9.18555H1.56738C1.21875 9.18555 0.952148 9.45898 0.952148 9.80078C0.952148 10.1426 1.22559 10.4229 1.56738 10.4229ZM1.56738 14.0049H8.75879C9.10059 14.0049 9.38086 13.7246 9.38086 13.3828C9.38086 13.041 9.10742 12.7676 8.75879 12.7676H1.56738C1.21875 12.7676 0.952148 13.041 0.952148 13.3828C0.952148 13.7246 1.22559 14.0049 1.56738 14.0049Z"></path></svg></span>Authors</th><td></td></tr><tr class="property-row property-row-url"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesUrl"><path d="M7.69922 10.8945L8.73828 9.84863C7.91797 9.77344 7.34375 9.51367 6.91992 9.08984C5.76465 7.93457 5.76465 6.29395 6.91309 5.14551L9.18262 2.87598C10.3379 1.7207 11.9717 1.7207 13.127 2.87598C14.2891 4.04492 14.2822 5.67188 13.1338 6.82031L11.958 7.99609C12.1768 8.49512 12.2451 9.10352 12.1289 9.62988L14.0908 7.6748C15.7725 6 15.7793 3.62109 14.084 1.92578C12.3887 0.223633 10.0098 0.237305 8.33496 1.91211L5.95605 4.29785C4.28125 5.97266 4.26758 8.35156 5.96289 10.0469C6.36621 10.4434 6.90625 10.7441 7.69922 10.8945ZM8.30078 5.13184L7.26855 6.17773C8.08203 6.25293 8.66309 6.51953 9.08008 6.93652C10.2422 8.09863 10.2422 9.73242 9.08691 10.8809L6.81738 13.1504C5.66211 14.3057 4.03516 14.3057 2.87305 13.1504C1.71094 11.9883 1.71777 10.3545 2.87305 9.20605L4.04199 8.03027C3.83008 7.53125 3.75488 6.92969 3.87109 6.39648L1.91602 8.35156C0.234375 10.0264 0.227539 12.4121 1.92285 14.1074C3.61816 15.8027 5.99707 15.7891 7.67188 14.1143L10.0439 11.7354C11.7256 10.0537 11.7324 7.6748 10.0371 5.98633C9.64062 5.58301 9.10059 5.28223 8.30078 5.13184Z"></path></svg></span>Link</th><td><a href="https://www.mdpi.com/1999-4915/5/12/2920/htm" class="url-value">https://www.mdpi.com/1999-4915/5/12/2920/htm</a></td></tr><tr class="property-row property-row-url"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesUrl"><path d="M7.69922 10.8945L8.73828 9.84863C7.91797 9.77344 7.34375 9.51367 6.91992 9.08984C5.76465 7.93457 5.76465 6.29395 6.91309 5.14551L9.18262 2.87598C10.3379 1.7207 11.9717 1.7207 13.127 2.87598C14.2891 4.04492 14.2822 5.67188 13.1338 6.82031L11.958 7.99609C12.1768 8.49512 12.2451 9.10352 12.1289 9.62988L14.0908 7.6748C15.7725 6 15.7793 3.62109 14.084 1.92578C12.3887 0.223633 10.0098 0.237305 8.33496 1.91211L5.95605 4.29785C4.28125 5.97266 4.26758 8.35156 5.96289 10.0469C6.36621 10.4434 6.90625 10.7441 7.69922 10.8945ZM8.30078 5.13184L7.26855 6.17773C8.08203 6.25293 8.66309 6.51953 9.08008 6.93652C10.2422 8.09863 10.2422 9.73242 9.08691 10.8809L6.81738 13.1504C5.66211 14.3057 4.03516 14.3057 2.87305 13.1504C1.71094 11.9883 1.71777 10.3545 2.87305 9.20605L4.04199 8.03027C3.83008 7.53125 3.75488 6.92969 3.87109 6.39648L1.91602 8.35156C0.234375 10.0264 0.227539 12.4121 1.92285 14.1074C3.61816 15.8027 5.99707 15.7891 7.67188 14.1143L10.0439 11.7354C11.7256 10.0537 11.7324 7.6748 10.0371 5.98633C9.64062 5.58301 9.10059 5.28223 8.30078 5.13184Z"></path></svg></span>Dataset</th><td></td></tr><tr class="property-row property-row-created_time"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesCreatedAt"><path d="M8 15.126C11.8623 15.126 15.0615 11.9336 15.0615 8.06445C15.0615 4.20215 11.8623 1.00293 7.99316 1.00293C4.13086 1.00293 0.938477 4.20215 0.938477 8.06445C0.938477 11.9336 4.1377 15.126 8 15.126ZM8 13.7383C4.85547 13.7383 2.33301 11.209 2.33301 8.06445C2.33301 4.91992 4.84863 2.39746 7.99316 2.39746C11.1377 2.39746 13.6738 4.91992 13.6738 8.06445C13.6738 11.209 11.1445 13.7383 8 13.7383ZM4.54102 8.91211H7.99316C8.30078 8.91211 8.54004 8.67285 8.54004 8.37207V3.8877C8.54004 3.58691 8.30078 3.34766 7.99316 3.34766C7.69238 3.34766 7.45312 3.58691 7.45312 3.8877V7.83203H4.54102C4.2334 7.83203 4.00098 8.06445 4.00098 8.37207C4.00098 8.67285 4.2334 8.91211 4.54102 8.91211Z"></path></svg></span>Log Date</th><td><time>@January 24, 2021 6:20 PM</time></td></tr><tr class="property-row property-row-multi_select"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesMultipleSelect"><path d="M1.91602 4.83789C2.44238 4.83789 2.87305 4.40723 2.87305 3.87402C2.87305 3.34766 2.44238 2.91699 1.91602 2.91699C1.38281 2.91699 0.952148 3.34766 0.952148 3.87402C0.952148 4.40723 1.38281 4.83789 1.91602 4.83789ZM5.1084 4.52344H14.3984C14.7607 4.52344 15.0479 4.23633 15.0479 3.87402C15.0479 3.51172 14.7607 3.22461 14.3984 3.22461H5.1084C4.74609 3.22461 4.45898 3.51172 4.45898 3.87402C4.45898 4.23633 4.74609 4.52344 5.1084 4.52344ZM1.91602 9.03516C2.44238 9.03516 2.87305 8.60449 2.87305 8.07129C2.87305 7.54492 2.44238 7.11426 1.91602 7.11426C1.38281 7.11426 0.952148 7.54492 0.952148 8.07129C0.952148 8.60449 1.38281 9.03516 1.91602 9.03516ZM5.1084 8.7207H14.3984C14.7607 8.7207 15.0479 8.43359 15.0479 8.07129C15.0479 7.70898 14.7607 7.42188 14.3984 7.42188H5.1084C4.74609 7.42188 4.45898 7.70898 4.45898 8.07129C4.45898 8.43359 4.74609 8.7207 5.1084 8.7207ZM1.91602 13.2324C2.44238 13.2324 2.87305 12.8018 2.87305 12.2686C2.87305 11.7422 2.44238 11.3115 1.91602 11.3115C1.38281 11.3115 0.952148 11.7422 0.952148 12.2686C0.952148 12.8018 1.38281 13.2324 1.91602 13.2324ZM5.1084 12.918H14.3984C14.7607 12.918 15.0479 12.6309 15.0479 12.2686C15.0479 11.9062 14.7607 11.6191 14.3984 11.6191H5.1084C4.74609 11.6191 4.45898 11.9062 4.45898 12.2686C4.45898 12.6309 4.74609 12.918 5.1084 12.918Z"></path></svg></span>Type</th><td></td></tr><tr class="property-row property-row-multi_select"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesMultipleSelect"><path d="M1.91602 4.83789C2.44238 4.83789 2.87305 4.40723 2.87305 3.87402C2.87305 3.34766 2.44238 2.91699 1.91602 2.91699C1.38281 2.91699 0.952148 3.34766 0.952148 3.87402C0.952148 4.40723 1.38281 4.83789 1.91602 4.83789ZM5.1084 4.52344H14.3984C14.7607 4.52344 15.0479 4.23633 15.0479 3.87402C15.0479 3.51172 14.7607 3.22461 14.3984 3.22461H5.1084C4.74609 3.22461 4.45898 3.51172 4.45898 3.87402C4.45898 4.23633 4.74609 4.52344 5.1084 4.52344ZM1.91602 9.03516C2.44238 9.03516 2.87305 8.60449 2.87305 8.07129C2.87305 7.54492 2.44238 7.11426 1.91602 7.11426C1.38281 7.11426 0.952148 7.54492 0.952148 8.07129C0.952148 8.60449 1.38281 9.03516 1.91602 9.03516ZM5.1084 8.7207H14.3984C14.7607 8.7207 15.0479 8.43359 15.0479 8.07129C15.0479 7.70898 14.7607 7.42188 14.3984 7.42188H5.1084C4.74609 7.42188 4.45898 7.70898 4.45898 8.07129C4.45898 8.43359 4.74609 8.7207 5.1084 8.7207ZM1.91602 13.2324C2.44238 13.2324 2.87305 12.8018 2.87305 12.2686C2.87305 11.7422 2.44238 11.3115 1.91602 11.3115C1.38281 11.3115 0.952148 11.7422 0.952148 12.2686C0.952148 12.8018 1.38281 13.2324 1.91602 13.2324ZM5.1084 12.918H14.3984C14.7607 12.918 15.0479 12.6309 15.0479 12.2686C15.0479 11.9062 14.7607 11.6191 14.3984 11.6191H5.1084C4.74609 11.6191 4.45898 11.9062 4.45898 12.2686C4.45898 12.6309 4.74609 12.918 5.1084 12.918Z"></path></svg></span>Technique</th><td></td></tr><tr class="property-row property-row-checkbox"><th><span class="icon property-icon"><svg viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesCheckbox"><path d="M3.85742 14.4561H12.1357C13.6123 14.4561 14.3779 13.6904 14.3779 12.2344V3.91504C14.3779 2.45215 13.6123 1.69336 12.1357 1.69336H3.85742C2.38086 1.69336 1.61523 2.45215 1.61523 3.91504V12.2344C1.61523 13.6973 2.38086 14.4561 3.85742 14.4561ZM3.93945 13.1162C3.30371 13.1162 2.95508 12.7812 2.95508 12.1182V4.02441C2.95508 3.36133 3.30371 3.0332 3.93945 3.0332H12.0537C12.6826 3.0332 13.0381 3.36133 13.0381 4.02441V12.1182C13.0381 12.7812 12.6826 13.1162 12.0537 13.1162H3.93945ZM7.26855 11.3115C7.51465 11.3115 7.72656 11.1885 7.87012 10.9697L10.9258 6.19141C11.0146 6.04785 11.0967 5.88379 11.0967 5.72656C11.0967 5.3916 10.7959 5.16602 10.4746 5.16602C10.2695 5.16602 10.085 5.27539 9.94141 5.50781L7.24121 9.8418L5.96973 8.22168C5.80566 8.00977 5.6416 7.93457 5.43652 7.93457C5.10156 7.93457 4.8418 8.20117 4.8418 8.54297C4.8418 8.70703 4.90332 8.85742 5.01953 9.00098L6.63281 10.9697C6.81738 11.209 7.01562 11.3115 7.26855 11.3115Z"></path></svg></span>Dataset Raw</th><td><div class="checkbox checkbox-off"></div></td></tr></tbody></table></header><div class="page-body"><h2 id="649fb7e8-b149-43d1-b8fe-6b3c999d489a" class="">1. Introduction</h2><p id="f4b38254-2c51-4137-9361-0262582701ae" class="">Hirudo medicinalis is a leech used medically to extract blood, and it is also valued for its capacity to avoid coagulation by secreting hirudin, an anticoagulant substance discovered in leech saliva by Haycraft in 1884. It took another century for the substance to be successfully isolated and characterized by Markwardt, in 1986 [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">1</a>].</p><p id="e6b3c4a6-c20a-4aed-8ae1-f02eb081dfe2" class="">The natural habitat of leeches is mostly fresh water from puddles, and the progressive disappearance of this habitat, as well as the intensive collection of wild leeches for medical purpose in the eighteenth century in Central Europe, led to a dramatic decrease in the population and the concomitant classification of the species on the International Union for Conservation of Nature (IUCN) Red List of endangered species [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">2</a>]. Collecting wild leeches from natural environments for medical use can be problematic because of they represent a risk for micro-organism transmission. Indeed, in the natural habitat, leeches are in constant interaction with environmental bacteria, protozoa and viruses, which can colonize their digestive tract. The symbiosis of the leeches from the genus Hirudo with Aeromonas sp. has been well known for years [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">3</a>], and several cases of human transmission of A. hydrophila have been reported when used in microsurgery. In the natural environment, the interactions between leeches and micro-organisms have not been well characterized, even when other associated bacteria have been identified in the animal. Indeed, some Ochrobactrum sp., Bdellovibrio sp. and Sphingobacterium sp. were detected in the bladders of medical leeches [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">7</a>], and a Rikenella-like bacterium was isolated from the gut of a H. verbena species [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">8</a>].</p><p id="34099bcb-4956-4a81-84b2-5d01a468310a" class="">The role of leeches as a potential reservoir for viruses has also been suspected. Even if they are not known to be natural carriers of viruses, the persistence of the classical swine fever virus and the Myxoma virus in their digestive tract was demonstrated after leeches were fed with swine and rabbit blood contaminated with those viruses [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">9</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">10</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">11</a>]. In recent years, several other viruses have been demonstrated or suspected to be harbored by leeches [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">10</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">11</a>]. In the natural environment, leeches may come into contact with other species of viruses, including the giant viruses that infect amoebae and are now well known to be widely present in aquatic environments and soil [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">12</a>]. We describe here the first isolation of a giant virus of the family Mimiviridae from a wild specimen of a medical leech, Hirudo medicinalis.</p><h2 id="97c5c533-e908-419a-b8e5-2078f70afb2f" class="">2. Results</h2><h3 id="b74bfa99-fc58-4183-9bfd-4e9f79d1a0c4" class="">2.1. Virus isolation and preliminary characterization</h3><p id="994be674-dae7-48b7-aac5-3670ca5cafca" class="">The inoculations on amoebae in agar plate showing lysis plaque (<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">Figure 1</a>) resulted in the isolation of one single virus from one Tunisian Hirudo medicinalis (<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">Figure 2</a>) collected at “Oued Sarrath” (35°46&#x27;0&quot; N, 8°37&#x27;0&quot; E). No virus was isolated from PAS buffer used to rinse leeches in order to remove ethanol. The virus was isolated from the internal organs and digestive tract Electron microscopic examination of the negative-stained culture supernatant showed the typical icosahedral structure of giant viruses, with a capsid (410 nm±10) surrounded by long fibrils (110 nm±10) and an overall viral particle diameter of 520 nm±10. Transmission electronic microscopy showed a typical aspect of virus factory within A. polyphaga 12h post infection (<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">Figure 3</a>). Based on this observation, we pre-classified the virus in the family Mimiviridae. This new strain of Mimivirus was named Hirudovirus.</p><figure id="390945a9-d628-4aa3-9e73-b5f4ab9fba22" class="image"><a href="viruses-05-02920-g001-1024.png"><img style="width:700px" src="viruses-05-02920-g001-1024.png"/></a></figure><p id="6c33d664-8c4e-4d72-9f03-37da2d2dbcbf" class="">Figure 1. Aspect of agar plate on which samples were inoculated. (a) negative control, (b) Hirudovirus starin growth leading to lysis of amoebas.</p><figure id="ab581f4e-e9f5-4e8b-874c-bd60d36595b5" class="image"><a href="viruses-05-02920-g002-1024.png"><img style="width:700px" src="viruses-05-02920-g002-1024.png"/></a></figure><p id="e6b0b01b-a990-4605-8000-24c366d536ac" class="">Figure 2. Hirudo medicinalis in which Hirudovirus strain was isolated.</p><figure id="010c86ce-f46f-41ac-a9d4-fd0423a80229" class="image"><a href="viruses-05-02920-g003-1024.png"><img style="width:700px" src="viruses-05-02920-g003-1024.png"/></a></figure><p id="10694075-7d9b-4cba-b8a6-d66afbd01e5d" class="">Typical aspect if Hirudovirus virus factory (VF) growing in</p><p id="e5f00ef7-1c4a-4e49-9201-23390879e0fa" class=""><strong>Figure 3.</strong>A. polyphaga.</p><p id="e711d0f5-583e-47ac-accc-5da878286704" class="">The sequenced genome assembly resulted in two scaffolds of 1,155,382 and 25,660 base pairs, with a low (28%) mean GC content, which is similar to those of other mimiviruses of amoebae [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">13</a>]. A total of 992 predicted genes, ranging in size from 34 to 2,959 amino acids, were detected in the draft genome of the strain Hirudovirus. This corresponds to a protein coding density of 0.9 genes/kilobase pair similar to values determined for Mimivirus (0.83) and Mamavirus (0.86), which belong to lineage A of amoebal mimiviruses [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">13</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">14</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">15</a>]. A functional annotation was identified for 47% of these genes (corresponding to 466 proteins). The comparative genomic analyses showed that the Hirudovirus strain genome shared the highest number of bona fide orthologs with Mimivirus (n= 806; 82% of the Hirudovirus gene content) and Mamavirus (805), whereas 435, 402, 361 and 338 bona fide orthologs were found from other mimiviruses of amoeba including Megavirus chiliensis [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">16</a>], LBA111 [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">17</a>], Moumouvirus [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">18</a>] and Monve [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">12</a>], respectively. Besides, 38% of the Hirudovirus predicted proteins had significant BLASTp hits from organisms outside the family Mimiviridae. Of the Hirudovirus predicted proteins, 774 (78%) and 190 (19%) found a protein from Mimivirus or Mamavirus, respectively, as top hit in the GenBank non-redundant database. Proteins from other mimiviruses were identified as top hits for other Hirudovirus proteins, and among them, 18 were detected in the genome of Lentillevirus, which is classified within lineage C of amoebal mimiviruses [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">19</a>]. Four Hirudovirus predicted proteins were not found by BLASTp in Mimivirus and Mamavirus but were found in other mimiviruses of amoeba; especially, all four were found in Courdo7 virus, a mimivirus of amoeba from lineage C [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">13</a>]. Finally, two Hirudovirus proteins (HIRU_S223 and HIRU_S244) had no significant BLASTp hit in the NCBI Genbank non-redundant and environmental protein sequence databases but BLAST search using nucleotide sequences detected hits in the Mimivirus and Mamavirus genomes. The Hirudovirus and Mimivirus genomes are highly similar and collinear (<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">Figure 4</a>). Bona fide orthologs from these viruses showed a mean amino acid identity of 99.1±2.6% (range, 72.1-100%), respectively. In addition, mean nucleotide identities for these bona fide orthologs was 99.3±1.8% (range, 80.7-100%), respectively. These values are in the same order of magnitude than those described for bona fide orthologs of Mimivirus and Mamavirus that is another strain of Mimivirus [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">15</a>].</p><figure id="0ddfa7f1-d8b7-4d51-b696-4d4c9b7f8aa2" class="image"><a href="viruses-05-02920-g004-1024.png"><img style="width:700px" src="viruses-05-02920-g004-1024.png"/></a></figure><p id="d1f7504f-8ae6-4ec2-bea7-5255d3e6298c" class=""><strong>Figure 4.</strong> Genomic dotplot between Mimivirus and Hirudovirus pairs of orthologs by BLASTp analysis. The diameter of the circles is proportional with the BLASTp bit score. The arrow indicates the position between the two contigs.</p><p id="3878d9cb-1c33-42e8-963a-5b3934fc82fc" class="">Phylogenic reconstructions based on a concatenated alignment of three conserved genes shows that Hirudovirus strain clusters with Mimivirus and Mamavirus in the lineage A, previously defined for mimiviruses of amoebae [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">13</a>], which agrees with the results of the comparative genomic analyses (<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">Figure 5</a>).</p><figure id="26e9a209-a7ee-4c5f-9211-27c385a35c75" class="image"><a href="viruses-05-02920-g005-1024.png"><img style="width:700px" src="viruses-05-02920-g005-1024.png"/></a></figure><p id="07d60e01-0471-4bfc-a016-042ccf1a76f6" class=""><strong>Figure 5.</strong> Maximum likelihood phylogeny reconstruction based on a concatenated alignment of the VVA18 helicase, the DNA polymerase B family and the VV-A32 packaging ATPase. Phylogeny reconstruction was performed using the MEGA5 software on a total of 1835 positions in the final dataset. Bootstrap values (as a result of 1,000 replicates) are shown as percentages next to the branches. A discrete Gamma distribution was used to model evolutionary rate differences among sites. The tree is drawn to scale, with branch lengths measured in the number of substitutions per site.</p><h2 id="8ec863cf-b65e-4efc-88f6-153139aaec8d" class="">3. Discussion</h2><p id="378b3745-43df-4f97-ba10-903bf8e5fdb8" class="">One single specimen, among the 5 Hirudo medicinalis analyzed, led to the isolation of a giant virus from the internal part of the animal. The virus was absent from the flesh, and the presence of the virus in the internal organs and the gastrointestinal tract suggests that the giant virus may have been ingested with food. Indeed, giant viruses can naturally be found in several ecological niches, such as fresh water, seawater (including hypersaline environments), and soil [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">12</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">20</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">21</a>]. Hence, the presence of giant viruses in the same ecological niches as wild Hirudo species can explain the presence of a mimivirus in the digestive tract of the leech. Several studies have shown that viruses persist in the digestive tract of leeches fed with contaminated blood [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">3</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">5</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">6</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">9</a>]. Moreover, leeches are known to exhibit symbiosis with several bacteria, mainly those species from the genus Aeromonas, and with several proteobacterial genera [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">8</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">9</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">20</a>]. This symbiosis becomes problematic when leeches are used in a medical context and infected animals eject symbiotic bacteria from their digestive tracts into the bite. Indeed, feeding leeches blood from animals treated with antibiotics can lead to the acquisition of resistance in the symbiotic bacteria, and this complicates the treatment of patients in case of accidental infection through hirudotherapy [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">7</a>]. Alternatively, we cannot exclude the possibility that mimiviruses can be contaminant of water in hospital aquaria used to grow leeches. Further studies should examine this potential niche because giant viruses have already been isolated from aquatic environments (including some that are close to humans, such as hospital water). Another concern stemming from the presence of amoebal mimiviruses in leeches is that a growing body of evidence suggest that these viruses could be a rare cause of pneumonia [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">17</a>]. Notably, mimiviruses of lineage C were recently isolated from the bronchoalveolar fluid and the stools of Tunisian patients with pneumonia. Additionally, a mimivirus of lineage A, a close relative to the strain Hirudovirus, was isolated from the contact lens fluid of a young woman exhibiting keratitis [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">19</a>]. Our work does not find evidence of pathogenic effects of the Mimivirus on the leech. However, further investigations should be performed to evaluate the impact of infection on the viability of these medically important animals. The analysis of the 1.18 megabase genome of the strain shows gene content closely related to Mimivirus and Mamavirus, which belong to lineage A of amoebal mimiviruses [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">13</a>]. Notably, 97% of the Hirudovirus genes were most similar (via BLASTp) to proteins from either Mimivirus or Mamavirus, which is another Mimivirus strain and Hirudovirus and Mimivirus genomes showed high levels of similarity and collinearity. Moreover, these findings are consistent with those of the phylogenetic analysis. Hirudovirus strain is the first giant virus to be isolated from a member of the family Hirudinae. However, other mimiviruses have been isolated from humans, whereas marseilleviruses, other giant viruses that infect amoebae, have been isolated from humans and Eristalis tenax, an insect member of the order Diptera [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">22</a>]. These data suggest that the diversity of the ecological niches for these giant viruses is not yet fully understood.</p><h2 id="86c16352-30cf-42cd-9cdd-0884be46feb6" class="">4. Materials and methods</h2><p id="4ad84402-2b12-4bf3-ad50-972caaec2b73" class=""><strong>Samples and inoculation of amoeba.</strong> Animals tested for the presence of giant viruses came from two small streams at two locations in Tunisia and France. A total of five leeches of the species Hirudo medicinalis were isolated, three in Tunisia and two in France. Wet bottles were used for transporting live samples to the laboratory. The method used to prepare the animals and the procedure for isolating the virus have been recently detailed elsewhere [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">20</a>]. Briefly, after morphological identification, Hirudo medicinalis were rinsed with 96% ethanol for 20 min and then washed with sterile PAS (Page’s amoebal saline). The washing solution was saved. The animals were dissected to obtain samples of flesh (F) and internal organs in digestive tract (IO+DT). These samples were crushed mechanically before inoculation using the high throughput isolation system previously described [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">21</a>].</p><p id="df29d29c-7ad1-4a18-9a6c-28f9c6c354a4" class=""><strong>Preliminary characterization of viruses.</strong> After the appearance of an amoebal lysis plaque, indicates the presence of a giant virus, the agar plate was cut into small pieces around the plaque, resuspended in 500 µl sterile PAS, centrifuged at 15000 rpm for 30 minutes, and re-inoculated onto a fresh amoebal monolayer. The presence of the giant virus was then assessed by Hemacolor staining after cytocentrifugation of a 100 µl sample of culture supernatant. The preliminary classification of the virus was performed by observation of the culture supernatant, which was negative stained with a 3% ammonium molybdate solution and evaluated with electron microscopy.</p><p id="a988deb3-7c44-421b-b315-375a5f08da5f" class=""><strong>Whole genome sequencing.</strong> For whole genome sequencing, 25 culture flasks (150 cm2) were prepared with 5 ml of virus, 10 ml of amoebae and 30 ml of nutritive medium PYG (Proteose peptone, Yeast extract, Glucose). After amoebal lysis, viruses were purified and concentrated; genomic DNA extraction was performed with phenol-chloroform, and the whole genome was pyrosequenced on the 454-Roche GSFLX as previously described [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">23</a>].</p><p id="d69b7a2a-9a52-4b50-8657-93136666a9be" class=""><strong>Sequence assembly, gene prediction and annotation.</strong> The genome assembly was performed with the Newbler software [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">24</a>]. Genes were predicted using the GeneMarkS program [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">25</a>]. All ORFs smaller than 100 codons were considered only if they had a significant hit (BLASTp e-value &lt;1e-4, both query and target coverages &gt;25%) in the NCBI GenBank non-redundant protein sequence database (nr), including genes from Acanthamoeba polyphaga Mimivirus [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">26</a>], Acanthamoeba castellanii mamavirus [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">15</a>], Acanthamoeba polyphaga moumouvirus [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">18</a>], LBA111 [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">17</a>], Megavirus chiliensis [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">16</a>], Moumouvirus monve and the recently described Pandoravirus dulcis and Pandoravirus salinus [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">27</a>]. The Hirudovirus predicted proteins were additionally searched by BLASTp against the NCBI environmental protein sequence (env_nr) and non-redundant nucleotide sequence (nt) databases. Next, genes were annotated using comparative genomics, including BLASTp searches against the GenBank nr database and the published gene repertoires of mimiviruses, and P. dulcis and P. salinus. Bona fide orthologs, which are orthologs identified through the widely used approach consisting in the identification of reciprocal best hits between two gene repertoires, were determined here by the Proteinortho program with an e-value cut-off of 1e-3 and minimum amino acid identity and coverage of 30% and 70%, respectively [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">28</a>]. In addition, a search was performed for all of the orthologous genes shared with the previously cited viruses using BLASTp with an e-value threshold of 1e-4 and minimal coverages of query and target sequences of 25%. Predictions of transfer RNA (tRNA) were performed using the ARAGORN software [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">29</a>].</p><p id="40edbe4e-dd27-4926-931d-a3c4f3e3abb4" class=""><strong>Phylogenetic analysis.</strong> The phylogenetic analysis was based on three concatenated core genes (the VV-A18 helicase, the VV-A32 packaging ATPase and the B family DNA polymerase) of the nucleocytoplasmic large DNA viruses [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">14</a>]. Multiple sequences alignments were performed using the MUSCLE program [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">30</a>], informational positions were selected by the Gblocks tool [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">31</a>] and a phylogenetic tree was constructed using MEGA5 [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">32</a>] and the maximum likelihood method. Genes from Marseillevirus and pandoraviruses were used as outgroups.</p><h2 id="55465203-45a1-44c0-9415-76e2ee5decca" class="">References</h2><ol type="1" id="d8d455e7-025f-44a0-9ba5-22dd580d00ce" class="numbered-list" start="1"><li>Whitaker, I.S.; Oboumarzouk, O.; Rozen, W.M.; Naderi, N.; Balasubramanian, S.P.; Azzopardi, E.A.; Kon, M. The efficacy of medicinal leeches in plastic and reconstructive surgery: a systematic review of 277 reported clinical cases. Microsurgery <strong>2012</strong>, 32, 240–249. [<a href="https://scholar.google.com/scholar_lookup?title=The+efficacy+of+medicinal+leeches+in+plastic+and+reconstructive+surgery:+a+systematic+review+of+277+reported+clinical+cases&amp;author=Whitaker,+I.S.&amp;author=Oboumarzouk,+O.&amp;author=Rozen,+W.M.&amp;author=Naderi,+N.&amp;author=Balasubramanian,+S.P.&amp;author=Azzopardi,+E.A.&amp;author=Kon,+M.&amp;publication_year=2012&amp;journal=Microsurgery&amp;volume=32&amp;pages=240%E2%80%93249&amp;doi=10.1002/micr.20971">Google Scholar</a>] [<a href="https://dx.doi.org/10.1002/micr.20971">CrossRef</a>]</li></ol><ol type="1" id="e28b0ef6-5731-475f-827c-ab553dec27b6" class="numbered-list" start="2"><li>Pagnier, I.; Ikanga Reteno, D.G.; Saadi, H.; Boughalmi, M.; Gaia, M.; Slimani, M.; Nto, T.; Bekliz, M.; Colson, P.; Raoult, D.; La Scola, B. A decade of improvements in Mimiviridae and Marseilleviridae isolation from amoeba. Intervirology <strong>2013</strong>, (in press). [<a href="https://scholar.google.com/scholar_lookup?title=A+decade+of+improvements+in+Mimiviridae+and+Marseilleviridae+isolation+from+amoeba&amp;author=Pagnier,+I.&amp;author=Ikanga+Reteno,+D.G.&amp;author=Saadi,+H.&amp;author=Boughalmi,+M.&amp;author=Gaia,+M.&amp;author=Slimani,+M.&amp;author=Nto,+T.&amp;author=Bekliz,+M.&amp;author=Colson,+P.&amp;author=Raoult,+D.&amp;publication_year=2013&amp;journal=Intervirology">Google Scholar</a>]</li></ol><ol type="1" id="54f12f81-416c-4893-afc7-0af782406d9b" class="numbered-list" start="3"><li>Colson, P.; Yutin, N.; Shabalina, S.A.; Robert, C.; Fournous, G.; La Scola, B.; Raoult, D.; Koonin, E.V. Viruses with more than 1,000 genes: Mamavirus, a new Acanthamoeba polyphaga mimivirus strain, and reannotation of Mimivirus genes. Genome Biol. Evol. <strong>2011</strong>, 3, 737–742. [<a href="https://scholar.google.com/scholar_lookup?title=Viruses+with+more+than+1,000+genes:+Mamavirus,+a+new+Acanthamoeba+polyphaga+mimivirus+strain,+and+reannotation+of+Mimivirus+genes&amp;author=Colson,+P.&amp;author=Yutin,+N.&amp;author=Shabalina,+S.A.&amp;author=Robert,+C.&amp;author=Fournous,+G.&amp;author=La+Scola,+B.&amp;author=Raoult,+D.&amp;author=Koonin,+E.V.&amp;publication_year=2011&amp;journal=Genome+Biol.+Evol.&amp;volume=3&amp;pages=737%E2%80%93742&amp;doi=10.1093/gbe/evr048">Google Scholar</a>] [<a href="https://dx.doi.org/10.1093/gbe/evr048">CrossRef</a>]</li></ol><ol type="1" id="fec522b5-976d-4250-be10-2db29c4abbfc" class="numbered-list" start="4"><li>Yoosuf, N.; Yutin, N.; Colson, P.; Shabalina, S.A.; Pagnier, I.; Robert, C.; Azza, S.; Klose, T.; Wong, J.; Rossmann, M.G.; La Scola, B.; Raoult, D.; Koonin, E.V. Related giant viruses in distant locations and different habitats: Acanthamoeba polyphaga moumouvirus represents a third lineage of the Mimiviridae that is close to the megavirus lineage. Genome. Biol. Evol. 2012 4, 1324–1330.</li></ol><ol type="1" id="79db884a-9b03-4576-ab85-a49d037ad7c6" class="numbered-list" start="5"><li>Boyer, M.; Yutin, N.; Pagnier, I.; Barrassi, L.; Fournous, G.; Espinosa, L.; Robert, C.; Azza, S.; Sun, S.; Rossmann, M.G.; Suzan-Monti, M.; La Scola, B.; Koonin, E.V.; Raoult, D. Giant Marseillevirus highlights the role of amoebae as a melting pot in emergence of chimeric microorganisms. Proc. Natl. Acad. Sci. U.S.A. <strong>2009</strong>, 106, 21848–21853. [<a href="https://scholar.google.com/scholar_lookup?title=Giant+Marseillevirus+highlights+the+role+of+amoebae+as+a+melting+pot+in+emergence+of+chimeric+microorganisms&amp;author=Boyer,+M.&amp;author=Yutin,+N.&amp;author=Pagnier,+I.&amp;author=Barrassi,+L.&amp;author=Fournous,+G.&amp;author=Espinosa,+L.&amp;author=Robert,+C.&amp;author=Azza,+S.&amp;author=Sun,+S.&amp;author=Rossmann,+M.G.&amp;publication_year=2009&amp;journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;volume=106&amp;pages=21848%E2%80%9321853">Google Scholar</a>]</li></ol><ol type="1" id="d914f619-4e66-46e4-92a5-decb8c26d8f6" class="numbered-list" start="6"><li>Margulies, M.; Egholm, M.; Altman, W.E.; Attiya, S.; Bader, J.S.; Bemben, L.A.; Berka, J.; Braverman, M.S.; Chen, Y.J.; Chen, Z.; Dewell, S.B.; Du, L.; Fierro, J.M.; Gomes, X.V.; Godwin, B.C.; He, W.; Helgesen, S.; Ho, C.H.; Irzyk, G.P.; Jando, S.C.; Alenquer, M.L.; Jarvie, T.P.; Jirage, K.B.; Kim, J.B.; Knight, J.R.; Lanza, J.R.; Leamon, J.H.; Lefkowitz, S.M.; Lei, M.; Li, J.; Lohman, K.L.; Lu, H.; Makhijani, V.B.; McDade, K.E.; McKenna, M.P.; Myers, E.W.; Nickerson, E.; Nobile, J.R.; Plant, R.; Puc, B.P.; Ronan, M.T.; Roth, G.T.; Sarkis, G.J.; Simons, J.F.; Simpson, J.W.; Srinivasan, M.; Tartaro, K.R.; Tomasz, A.; Vogt, K.A.; Volkmer, G.A.; Wang, S.H.; Wang, Y.; Weiner, M.P.; Yu, P.; Begley, R.F.; Rothberg, J.M. Genome sequencing in microfabricated high-density picolitre reactors. Nature <strong>2005</strong>, 437, 376–380. [<a href="https://scholar.google.com/scholar_lookup?title=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors&amp;author=Margulies,+M.&amp;author=Egholm,+M.&amp;author=Altman,+W.E.&amp;author=Attiya,+S.&amp;author=Bader,+J.S.&amp;author=Bemben,+L.A.&amp;author=Berka,+J.&amp;author=Braverman,+M.S.&amp;author=Chen,+Y.J.&amp;author=Chen,+Z.&amp;publication_year=2005&amp;journal=Nature&amp;volume=437&amp;pages=376%E2%80%93380">Google Scholar</a>]</li></ol><ol type="1" id="962847aa-3836-41ef-82cc-387e40734172" class="numbered-list" start="7"><li>Philippe, N.; Legendre, M.; Doutre, G.; Couté, Y.; Poirot, O.; Lescot, M.; Arslan, D.; Seltzer, V.; Bertaux, L.; Bruley, C.; Garin, J.; Claverie, J.M.; Abergel, C. Pandoraviruses: amoeba viruses with genomes up to 2.5 Mb reaching that of parasitic eukaryotes. Science <strong>2013</strong>, 341, 281–286. [<a href="https://scholar.google.com/scholar_lookup?title=Pandoraviruses:+amoeba+viruses+with+genomes+up+to+2.5+Mb+reaching+that+of+parasitic+eukaryotes&amp;author=Philippe,+N.&amp;author=Legendre,+M.&amp;author=Doutre,+G.&amp;author=Cout%C3%A9,+Y.&amp;author=Poirot,+O.&amp;author=Lescot,+M.&amp;author=Arslan,+D.&amp;author=Seltzer,+V.&amp;author=Bertaux,+L.&amp;author=Bruley,+C.&amp;publication_year=2013&amp;journal=Science&amp;volume=341&amp;pages=281%E2%80%93286&amp;doi=10.1126/science.1239181">Google Scholar</a>] [<a href="https://dx.doi.org/10.1126/science.1239181">CrossRef</a>]</li></ol><p id="8cb75707-1d0b-4297-af38-7bb9f5b5c18c" class="">© 2013 by the authors; licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution license (<a href="http://creativecommons.org/licenses/by/3.0/">http://creativecommons.org/licenses/by/3.0/</a>).</p></div></article></body></html>