<html><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><title>Viruses | Free Full-Text | First Isolation of a Giant Virus from Wild Hirudo medicinalis Leech: Mimiviridae isolation in Hirudo medicinalis | HTML</title><style>
/* cspell:disable-file */
/* webkit printing magic: print all background colors */
html {
	-webkit-print-color-adjust: exact;
}
* {
	box-sizing: border-box;
	-webkit-print-color-adjust: exact;
}

html,
body {
	margin: 0;
	padding: 0;
}
@media only screen {
	body {
		margin: 2em auto;
		max-width: 900px;
		color: rgb(55, 53, 47);
	}
}

body {
	line-height: 1.5;
	white-space: pre-wrap;
}

a,
a.visited {
	color: inherit;
	text-decoration: underline;
}

.pdf-relative-link-path {
	font-size: 80%;
	color: #444;
}

h1,
h2,
h3 {
	letter-spacing: -0.01em;
	line-height: 1.2;
	font-weight: 600;
	margin-bottom: 0;
}

.page-title {
	font-size: 2.5rem;
	font-weight: 700;
	margin-top: 0;
	margin-bottom: 0.75em;
}

h1 {
	font-size: 1.875rem;
	margin-top: 1.875rem;
}

h2 {
	font-size: 1.5rem;
	margin-top: 1.5rem;
}

h3 {
	font-size: 1.25rem;
	margin-top: 1.25rem;
}

.source {
	border: 1px solid #ddd;
	border-radius: 3px;
	padding: 1.5em;
	word-break: break-all;
}

.callout {
	border-radius: 3px;
	padding: 1rem;
}

figure {
	margin: 1.25em 0;
	page-break-inside: avoid;
}

figcaption {
	opacity: 0.5;
	font-size: 85%;
	margin-top: 0.5em;
}

mark {
	background-color: transparent;
}

.indented {
	padding-left: 1.5em;
}

hr {
	background: transparent;
	display: block;
	width: 100%;
	height: 1px;
	visibility: visible;
	border: none;
	border-bottom: 1px solid rgba(55, 53, 47, 0.09);
}

img {
	max-width: 100%;
}

@media only print {
	img {
		max-height: 100vh;
		object-fit: contain;
	}
}

@page {
	margin: 1in;
}

.collection-content {
	font-size: 0.875rem;
}

.column-list {
	display: flex;
	justify-content: space-between;
}

.column {
	padding: 0 1em;
}

.column:first-child {
	padding-left: 0;
}

.column:last-child {
	padding-right: 0;
}

.table_of_contents-item {
	display: block;
	font-size: 0.875rem;
	line-height: 1.3;
	padding: 0.125rem;
}

.table_of_contents-indent-1 {
	margin-left: 1.5rem;
}

.table_of_contents-indent-2 {
	margin-left: 3rem;
}

.table_of_contents-indent-3 {
	margin-left: 4.5rem;
}

.table_of_contents-link {
	text-decoration: none;
	opacity: 0.7;
	border-bottom: 1px solid rgba(55, 53, 47, 0.18);
}

table,
th,
td {
	border: 1px solid rgba(55, 53, 47, 0.09);
	border-collapse: collapse;
}

table {
	border-left: none;
	border-right: none;
}

th,
td {
	font-weight: normal;
	padding: 0.25em 0.5em;
	line-height: 1.5;
	min-height: 1.5em;
	text-align: left;
}

th {
	color: rgba(55, 53, 47, 0.6);
}

ol,
ul {
	margin: 0;
	margin-block-start: 0.6em;
	margin-block-end: 0.6em;
}

li > ol:first-child,
li > ul:first-child {
	margin-block-start: 0.6em;
}

ul > li {
	list-style: disc;
}

ul.to-do-list {
	text-indent: -1.7em;
}

ul.to-do-list > li {
	list-style: none;
}

.to-do-children-checked {
	text-decoration: line-through;
	opacity: 0.375;
}

ul.toggle > li {
	list-style: none;
}

ul {
	padding-inline-start: 1.7em;
}

ul > li {
	padding-left: 0.1em;
}

ol {
	padding-inline-start: 1.6em;
}

ol > li {
	padding-left: 0.2em;
}

.mono ol {
	padding-inline-start: 2em;
}

.mono ol > li {
	text-indent: -0.4em;
}

.toggle {
	padding-inline-start: 0em;
	list-style-type: none;
}

/* Indent toggle children */
.toggle > li > details {
	padding-left: 1.7em;
}

.toggle > li > details > summary {
	margin-left: -1.1em;
}

.selected-value {
	display: inline-block;
	padding: 0 0.5em;
	background: rgba(206, 205, 202, 0.5);
	border-radius: 3px;
	margin-right: 0.5em;
	margin-top: 0.3em;
	margin-bottom: 0.3em;
	white-space: nowrap;
}

.collection-title {
	display: inline-block;
	margin-right: 1em;
}

.simple-table {
	margin-top: 1em;
	font-size: 0.875rem;
	empty-cells: show;
}
.simple-table td {
	height: 29px;
	min-width: 120px;
}

.simple-table th {
	height: 29px;
	min-width: 120px;
}

.simple-table-header-color {
	background: rgb(247, 246, 243);
	color: black;
}
.simple-table-header {
	font-weight: 500;
}

time {
	opacity: 0.5;
}

.icon {
	display: inline-block;
	max-width: 1.2em;
	max-height: 1.2em;
	text-decoration: none;
	vertical-align: text-bottom;
	margin-right: 0.5em;
}

img.icon {
	border-radius: 3px;
}

.user-icon {
	width: 1.5em;
	height: 1.5em;
	border-radius: 100%;
	margin-right: 0.5rem;
}

.user-icon-inner {
	font-size: 0.8em;
}

.text-icon {
	border: 1px solid #000;
	text-align: center;
}

.page-cover-image {
	display: block;
	object-fit: cover;
	width: 100%;
	max-height: 30vh;
}

.page-header-icon {
	font-size: 3rem;
	margin-bottom: 1rem;
}

.page-header-icon-with-cover {
	margin-top: -0.72em;
	margin-left: 0.07em;
}

.page-header-icon img {
	border-radius: 3px;
}

.link-to-page {
	margin: 1em 0;
	padding: 0;
	border: none;
	font-weight: 500;
}

p > .user {
	opacity: 0.5;
}

td > .user,
td > time {
	white-space: nowrap;
}

input[type="checkbox"] {
	transform: scale(1.5);
	margin-right: 0.6em;
	vertical-align: middle;
}

p {
	margin-top: 0.5em;
	margin-bottom: 0.5em;
}

.image {
	border: none;
	margin: 1.5em 0;
	padding: 0;
	border-radius: 0;
	text-align: center;
}

.code,
code {
	background: rgba(135, 131, 120, 0.15);
	border-radius: 3px;
	padding: 0.2em 0.4em;
	border-radius: 3px;
	font-size: 85%;
	tab-size: 2;
}

code {
	color: #eb5757;
}

.code {
	padding: 1.5em 1em;
}

.code-wrap {
	white-space: pre-wrap;
	word-break: break-all;
}

.code > code {
	background: none;
	padding: 0;
	font-size: 100%;
	color: inherit;
}

blockquote {
	font-size: 1.25em;
	margin: 1em 0;
	padding-left: 1em;
	border-left: 3px solid rgb(55, 53, 47);
}

.bookmark {
	text-decoration: none;
	max-height: 8em;
	padding: 0;
	display: flex;
	width: 100%;
	align-items: stretch;
}

.bookmark-title {
	font-size: 0.85em;
	overflow: hidden;
	text-overflow: ellipsis;
	height: 1.75em;
	white-space: nowrap;
}

.bookmark-text {
	display: flex;
	flex-direction: column;
}

.bookmark-info {
	flex: 4 1 180px;
	padding: 12px 14px 14px;
	display: flex;
	flex-direction: column;
	justify-content: space-between;
}

.bookmark-image {
	width: 33%;
	flex: 1 1 180px;
	display: block;
	position: relative;
	object-fit: cover;
	border-radius: 1px;
}

.bookmark-description {
	color: rgba(55, 53, 47, 0.6);
	font-size: 0.75em;
	overflow: hidden;
	max-height: 4.5em;
	word-break: break-word;
}

.bookmark-href {
	font-size: 0.75em;
	margin-top: 0.25em;
}

.sans { font-family: ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol"; }
.code { font-family: "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace; }
.serif { font-family: Lyon-Text, Georgia, ui-serif, serif; }
.mono { font-family: iawriter-mono, Nitti, Menlo, Courier, monospace; }
.pdf .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK JP'; }
.pdf:lang(zh-CN) .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK SC'; }
.pdf:lang(zh-TW) .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK TC'; }
.pdf:lang(ko-KR) .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK KR'; }
.pdf .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK JP'; }
.pdf:lang(zh-CN) .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK SC'; }
.pdf:lang(zh-TW) .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK TC'; }
.pdf:lang(ko-KR) .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK KR'; }
.pdf .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK JP'; }
.pdf:lang(zh-CN) .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK SC'; }
.pdf:lang(zh-TW) .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK TC'; }
.pdf:lang(ko-KR) .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK KR'; }
.pdf .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK JP'; }
.pdf:lang(zh-CN) .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK SC'; }
.pdf:lang(zh-TW) .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK TC'; }
.pdf:lang(ko-KR) .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK KR'; }
.highlight-default {
	color: rgba(55, 53, 47, 1);
}
.highlight-gray {
	color: rgba(120, 119, 116, 1);
	fill: rgba(120, 119, 116, 1);
}
.highlight-brown {
	color: rgba(159, 107, 83, 1);
	fill: rgba(159, 107, 83, 1);
}
.highlight-orange {
	color: rgba(217, 115, 13, 1);
	fill: rgba(217, 115, 13, 1);
}
.highlight-yellow {
	color: rgba(203, 145, 47, 1);
	fill: rgba(203, 145, 47, 1);
}
.highlight-teal {
	color: rgba(68, 131, 97, 1);
	fill: rgba(68, 131, 97, 1);
}
.highlight-blue {
	color: rgba(51, 126, 169, 1);
	fill: rgba(51, 126, 169, 1);
}
.highlight-purple {
	color: rgba(144, 101, 176, 1);
	fill: rgba(144, 101, 176, 1);
}
.highlight-pink {
	color: rgba(193, 76, 138, 1);
	fill: rgba(193, 76, 138, 1);
}
.highlight-red {
	color: rgba(212, 76, 71, 1);
	fill: rgba(212, 76, 71, 1);
}
.highlight-gray_background {
	background: rgba(241, 241, 239, 1);
}
.highlight-brown_background {
	background: rgba(244, 238, 238, 1);
}
.highlight-orange_background {
	background: rgba(251, 236, 221, 1);
}
.highlight-yellow_background {
	background: rgba(251, 243, 219, 1);
}
.highlight-teal_background {
	background: rgba(237, 243, 236, 1);
}
.highlight-blue_background {
	background: rgba(231, 243, 248, 1);
}
.highlight-purple_background {
	background: rgba(244, 240, 247, 0.8);
}
.highlight-pink_background {
	background: rgba(249, 238, 243, 0.8);
}
.highlight-red_background {
	background: rgba(253, 235, 236, 1);
}
.block-color-default {
	color: inherit;
	fill: inherit;
}
.block-color-gray {
	color: rgba(120, 119, 116, 1);
	fill: rgba(120, 119, 116, 1);
}
.block-color-brown {
	color: rgba(159, 107, 83, 1);
	fill: rgba(159, 107, 83, 1);
}
.block-color-orange {
	color: rgba(217, 115, 13, 1);
	fill: rgba(217, 115, 13, 1);
}
.block-color-yellow {
	color: rgba(203, 145, 47, 1);
	fill: rgba(203, 145, 47, 1);
}
.block-color-teal {
	color: rgba(68, 131, 97, 1);
	fill: rgba(68, 131, 97, 1);
}
.block-color-blue {
	color: rgba(51, 126, 169, 1);
	fill: rgba(51, 126, 169, 1);
}
.block-color-purple {
	color: rgba(144, 101, 176, 1);
	fill: rgba(144, 101, 176, 1);
}
.block-color-pink {
	color: rgba(193, 76, 138, 1);
	fill: rgba(193, 76, 138, 1);
}
.block-color-red {
	color: rgba(212, 76, 71, 1);
	fill: rgba(212, 76, 71, 1);
}
.block-color-gray_background {
	background: rgba(241, 241, 239, 1);
}
.block-color-brown_background {
	background: rgba(244, 238, 238, 1);
}
.block-color-orange_background {
	background: rgba(251, 236, 221, 1);
}
.block-color-yellow_background {
	background: rgba(251, 243, 219, 1);
}
.block-color-teal_background {
	background: rgba(237, 243, 236, 1);
}
.block-color-blue_background {
	background: rgba(231, 243, 248, 1);
}
.block-color-purple_background {
	background: rgba(244, 240, 247, 0.8);
}
.block-color-pink_background {
	background: rgba(249, 238, 243, 0.8);
}
.block-color-red_background {
	background: rgba(253, 235, 236, 1);
}
.select-value-color-pink { background-color: rgba(245, 224, 233, 1); }
.select-value-color-purple { background-color: rgba(232, 222, 238, 1); }
.select-value-color-green { background-color: rgba(219, 237, 219, 1); }
.select-value-color-gray { background-color: rgba(227, 226, 224, 1); }
.select-value-color-opaquegray { background-color: rgba(255, 255, 255, 0.0375); }
.select-value-color-orange { background-color: rgba(250, 222, 201, 1); }
.select-value-color-brown { background-color: rgba(238, 224, 218, 1); }
.select-value-color-red { background-color: rgba(255, 226, 221, 1); }
.select-value-color-yellow { background-color: rgba(253, 236, 200, 1); }
.select-value-color-blue { background-color: rgba(211, 229, 239, 1); }

.checkbox {
	display: inline-flex;
	vertical-align: text-bottom;
	width: 16;
	height: 16;
	background-size: 16px;
	margin-left: 2px;
	margin-right: 5px;
}

.checkbox-on {
	background-image: url("data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2216%22%20height%3D%2216%22%20viewBox%3D%220%200%2016%2016%22%20fill%3D%22none%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%0A%3Crect%20width%3D%2216%22%20height%3D%2216%22%20fill%3D%22%2358A9D7%22%2F%3E%0A%3Cpath%20d%3D%22M6.71429%2012.2852L14%204.9995L12.7143%203.71436L6.71429%209.71378L3.28571%206.2831L2%207.57092L6.71429%2012.2852Z%22%20fill%3D%22white%22%2F%3E%0A%3C%2Fsvg%3E");
}

.checkbox-off {
	background-image: url("data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2216%22%20height%3D%2216%22%20viewBox%3D%220%200%2016%2016%22%20fill%3D%22none%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%0A%3Crect%20x%3D%220.75%22%20y%3D%220.75%22%20width%3D%2214.5%22%20height%3D%2214.5%22%20fill%3D%22white%22%20stroke%3D%22%2336352F%22%20stroke-width%3D%221.5%22%2F%3E%0A%3C%2Fsvg%3E");
}
	
</style></head><body><article id="8c8ca38e-3f7f-4be8-8fc1-c78ecebe06e5" class="page sans"><header><h1 class="page-title">Viruses | Free Full-Text | First Isolation of a Giant Virus from Wild Hirudo medicinalis Leech: Mimiviridae isolation in Hirudo medicinalis | HTML</h1><table class="properties"><tbody><tr class="property-row property-row-multi_select"><th><span class="icon property-icon"><svg viewBox="0 0 14 14" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesMultipleSelect"><path d="M4,3 C4,2.447715 4.447715,2 5,2 L12,2 C12.5523,2 13,2.447716 13,3 C13,3.55228 12.5523,4 12,4 L5,4 C4.447715,4 4,3.55228 4,3 Z M4,7 C4,6.447715 4.447715,6 5,6 L12,6 C12.5523,6 13,6.447716 13,7 C13,7.55228 12.5523,8 12,8 L5,8 C4.447715,8 4,7.55228 4,7 Z M4,11 C4,10.447715 4.447715,10 5,10 L12,10 C12.5523,10 13,10.447716 13,11 C13,11.55228 12.5523,12 12,12 L5,12 C4.447715,12 4,11.55228 4,11 Z M2,4 C1.44771525,4 1,3.55228475 1,3 C1,2.44771525 1.44771525,2 2,2 C2.55228475,2 3,2.44771525 3,3 C3,3.55228475 2.55228475,4 2,4 Z M2,8 C1.44771525,8 1,7.55228475 1,7 C1,6.44771525 1.44771525,6 2,6 C2.55228475,6 3,6.44771525 3,7 C3,7.55228475 2.55228475,8 2,8 Z M2,12 C1.44771525,12 1,11.5522847 1,11 C1,10.4477153 1.44771525,10 2,10 C2.55228475,10 3,10.4477153 3,11 C3,11.5522847 2.55228475,12 2,12 Z"></path></svg></span>Journal/Publisher</th><td></td></tr><tr class="property-row property-row-number"><th><span class="icon property-icon"><svg viewBox="0 0 14 14" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesNumber"><path d="M4.46191,0 C3.8667,0 3.38428,0.482422 3.38428,1.07751 L3.38428,3.38425 L1.07764,3.38425 C0.482422,3.38425 0,3.86667 0,4.46179 C0,5.05688 0.482422,5.53931 1.07764,5.53931 L3.38428,5.53931 L3.38428,8.46063 L1.07764,8.46063 C0.482422,8.46063 0,8.94308 0,9.53818 C0,10.1333 0.482422,10.6157 1.07764,10.6157 L3.38428,10.6157 L3.38428,12.9224 C3.38428,13.5175 3.8667,13.9999 4.46191,13.9999 C5.05664,13.9999 5.53906,13.5175 5.53906,12.9224 L5.53906,10.6157 L8.46045,10.6157 L8.46045,12.9224 C8.46045,13.5175 8.94287,13.9999 9.53809,13.9999 C10.1333,13.9999 10.6157,13.5175 10.6157,12.9224 L10.6157,10.6157 L12.9224,10.6157 C13.5176,10.6157 14,10.1333 14,9.53818 C14,8.94308 13.5176,8.46063 12.9224,8.46063 L10.6157,8.46063 L10.6157,5.53931 L12.9224,5.53931 C13.5176,5.53931 14,5.05688 14,4.46179 C14,3.86667 13.5176,3.38425 12.9224,3.38425 L10.6157,3.38425 L10.6157,1.07751 C10.6157,0.482422 10.1333,0 9.53809,0 C8.94287,0 8.46045,0.482422 8.46045,1.07751 L8.46045,3.38425 L5.53906,3.38425 L5.53906,1.07751 C5.53906,0.482422 5.05664,0 4.46191,0 Z M5.53906,8.46063 L5.53906,5.53931 L8.46045,5.53931 L8.46045,8.46063 L5.53906,8.46063 Z"></path></svg></span>Year</th><td></td></tr><tr class="property-row property-row-text"><th><span class="icon property-icon"><svg viewBox="0 0 14 14" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesText"><path d="M7,4.56818 C7,4.29204 6.77614,4.06818 6.5,4.06818 L0.5,4.06818 C0.223858,4.06818 0,4.29204 0,4.56818 L0,5.61364 C0,5.88978 0.223858,6.11364 0.5,6.11364 L6.5,6.11364 C6.77614,6.11364 7,5.88978 7,5.61364 L7,4.56818 Z M0.5,1 C0.223858,1 0,1.223858 0,1.5 L0,2.54545 C0,2.8216 0.223858,3.04545 0.5,3.04545 L12.5,3.04545 C12.7761,3.04545 13,2.8216 13,2.54545 L13,1.5 C13,1.223858 12.7761,1 12.5,1 L0.5,1 Z M0,8.68182 C0,8.95796 0.223858,9.18182 0.5,9.18182 L11.5,9.18182 C11.7761,9.18182 12,8.95796 12,8.68182 L12,7.63636 C12,7.36022 11.7761,7.13636 11.5,7.13636 L0.5,7.13636 C0.223858,7.13636 0,7.36022 0,7.63636 L0,8.68182 Z M0,11.75 C0,12.0261 0.223858,12.25 0.5,12.25 L9.5,12.25 C9.77614,12.25 10,12.0261 10,11.75 L10,10.70455 C10,10.4284 9.77614,10.20455 9.5,10.20455 L0.5,10.20455 C0.223858,10.20455 0,10.4284 0,10.70455 L0,11.75 Z"></path></svg></span>Authors</th><td></td></tr><tr class="property-row property-row-url"><th><span class="icon property-icon"><svg viewBox="0 0 14 14" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesUrl"><path d="M3.73333,3.86667 L7.46667,3.86667 C8.49613,3.86667 9.33333,4.70387 9.33333,5.73333 C9.33333,6.7628 8.49613,7.6 7.46667,7.6 L6.53333,7.6 C6.01813,7.6 5.6,8.0186 5.6,8.53333 C5.6,9.04807 6.01813,9.46667 6.53333,9.46667 L7.46667,9.46667 C9.5284,9.46667 11.2,7.79507 11.2,5.73333 C11.2,3.6716 9.5284,2 7.46667,2 L3.73333,2 C1.6716,2 0,3.6716 0,5.73333 C0,7.124 0.762067,8.33453 1.88953,8.97713 C1.87553,8.83107 1.86667,8.6836 1.86667,8.53333 C1.86667,7.92013 1.98753,7.33447 2.2036,6.7978 C1.99267,6.4954 1.86667,6.12953 1.86667,5.73333 C1.86667,4.70387 2.70387,3.86667 3.73333,3.86667 Z M12.1095,5.28907 C12.1231,5.4356 12.1333,5.58307 12.1333,5.73333 C12.1333,6.34607 12.0101,6.9294 11.7931,7.46513 C12.0059,7.768 12.1333,8.13573 12.1333,8.53333 C12.1333,9.5628 11.2961,10.4 10.2667,10.4 L6.53333,10.4 C5.50387,10.4 4.66667,9.5628 4.66667,8.53333 C4.66667,7.50387 5.50387,6.66667 6.53333,6.66667 L7.46667,6.66667 C7.98187,6.66667 8.4,6.24807 8.4,5.73333 C8.4,5.2186 7.98187,4.8 7.46667,4.8 L6.53333,4.8 C4.4716,4.8 2.8,6.4716 2.8,8.53333 C2.8,10.59507 4.4716,12.2667 6.53333,12.2667 L10.2667,12.2667 C12.3284,12.2667 14,10.59507 14,8.53333 C14,7.14267 13.2375,5.93167 12.1095,5.28907 Z"></path></svg></span>Link</th><td><a href="https://www.mdpi.com/1999-4915/5/12/2920/htm" class="url-value">https://www.mdpi.com/1999-4915/5/12/2920/htm</a></td></tr><tr class="property-row property-row-url"><th><span class="icon property-icon"><svg viewBox="0 0 14 14" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesUrl"><path d="M3.73333,3.86667 L7.46667,3.86667 C8.49613,3.86667 9.33333,4.70387 9.33333,5.73333 C9.33333,6.7628 8.49613,7.6 7.46667,7.6 L6.53333,7.6 C6.01813,7.6 5.6,8.0186 5.6,8.53333 C5.6,9.04807 6.01813,9.46667 6.53333,9.46667 L7.46667,9.46667 C9.5284,9.46667 11.2,7.79507 11.2,5.73333 C11.2,3.6716 9.5284,2 7.46667,2 L3.73333,2 C1.6716,2 0,3.6716 0,5.73333 C0,7.124 0.762067,8.33453 1.88953,8.97713 C1.87553,8.83107 1.86667,8.6836 1.86667,8.53333 C1.86667,7.92013 1.98753,7.33447 2.2036,6.7978 C1.99267,6.4954 1.86667,6.12953 1.86667,5.73333 C1.86667,4.70387 2.70387,3.86667 3.73333,3.86667 Z M12.1095,5.28907 C12.1231,5.4356 12.1333,5.58307 12.1333,5.73333 C12.1333,6.34607 12.0101,6.9294 11.7931,7.46513 C12.0059,7.768 12.1333,8.13573 12.1333,8.53333 C12.1333,9.5628 11.2961,10.4 10.2667,10.4 L6.53333,10.4 C5.50387,10.4 4.66667,9.5628 4.66667,8.53333 C4.66667,7.50387 5.50387,6.66667 6.53333,6.66667 L7.46667,6.66667 C7.98187,6.66667 8.4,6.24807 8.4,5.73333 C8.4,5.2186 7.98187,4.8 7.46667,4.8 L6.53333,4.8 C4.4716,4.8 2.8,6.4716 2.8,8.53333 C2.8,10.59507 4.4716,12.2667 6.53333,12.2667 L10.2667,12.2667 C12.3284,12.2667 14,10.59507 14,8.53333 C14,7.14267 13.2375,5.93167 12.1095,5.28907 Z"></path></svg></span>Dataset</th><td></td></tr><tr class="property-row property-row-created_time"><th><span class="icon property-icon"><svg viewBox="0 0 14 14" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesCreatedAt"><path d="M7.01356 14.0001C8.8042 14.0001 10.5958 13.3107 11.9575 11.9324C14.681 9.21201 14.6808 4.7603 11.9571 2.04013C9.23336 -0.680043 4.77573 -0.680043 2.05199 2.04013C0.727519 3.36277 0 5.13301 0 6.99553C0 8.8764 0.727811 10.6285 2.05199 11.9509C3.43207 13.3106 5.22243 14.0001 7.01356 14.0001ZM3.72947 7.00914V8.461V8.65543H3.92382H5.34563H8.2794H8.4738V8.461V5.52541V3.37947V3.18502H8.2794H6.82747H6.63307V3.37947V6.81467H3.92382H3.72947V7.00914ZM1.83985 6.99553C1.83985 5.61698 2.38099 4.32597 3.36061 3.3477C5.36746 1.34337 8.64803 1.34062 10.6585 3.33944C10.6613 3.34219 10.6639 3.34494 10.6668 3.3477C12.676 5.3546 12.6763 8.63642 10.6668 10.6434C8.65705 12.6504 5.37031 12.6504 3.36061 10.6434C2.38099 9.66506 1.83985 8.37408 1.83985 6.99553Z"></path></svg></span>Log Date</th><td><time>@January 24, 2021 8:20 PM</time></td></tr><tr class="property-row property-row-multi_select"><th><span class="icon property-icon"><svg viewBox="0 0 14 14" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesMultipleSelect"><path d="M4,3 C4,2.447715 4.447715,2 5,2 L12,2 C12.5523,2 13,2.447716 13,3 C13,3.55228 12.5523,4 12,4 L5,4 C4.447715,4 4,3.55228 4,3 Z M4,7 C4,6.447715 4.447715,6 5,6 L12,6 C12.5523,6 13,6.447716 13,7 C13,7.55228 12.5523,8 12,8 L5,8 C4.447715,8 4,7.55228 4,7 Z M4,11 C4,10.447715 4.447715,10 5,10 L12,10 C12.5523,10 13,10.447716 13,11 C13,11.55228 12.5523,12 12,12 L5,12 C4.447715,12 4,11.55228 4,11 Z M2,4 C1.44771525,4 1,3.55228475 1,3 C1,2.44771525 1.44771525,2 2,2 C2.55228475,2 3,2.44771525 3,3 C3,3.55228475 2.55228475,4 2,4 Z M2,8 C1.44771525,8 1,7.55228475 1,7 C1,6.44771525 1.44771525,6 2,6 C2.55228475,6 3,6.44771525 3,7 C3,7.55228475 2.55228475,8 2,8 Z M2,12 C1.44771525,12 1,11.5522847 1,11 C1,10.4477153 1.44771525,10 2,10 C2.55228475,10 3,10.4477153 3,11 C3,11.5522847 2.55228475,12 2,12 Z"></path></svg></span>Type</th><td></td></tr><tr class="property-row property-row-multi_select"><th><span class="icon property-icon"><svg viewBox="0 0 14 14" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesMultipleSelect"><path d="M4,3 C4,2.447715 4.447715,2 5,2 L12,2 C12.5523,2 13,2.447716 13,3 C13,3.55228 12.5523,4 12,4 L5,4 C4.447715,4 4,3.55228 4,3 Z M4,7 C4,6.447715 4.447715,6 5,6 L12,6 C12.5523,6 13,6.447716 13,7 C13,7.55228 12.5523,8 12,8 L5,8 C4.447715,8 4,7.55228 4,7 Z M4,11 C4,10.447715 4.447715,10 5,10 L12,10 C12.5523,10 13,10.447716 13,11 C13,11.55228 12.5523,12 12,12 L5,12 C4.447715,12 4,11.55228 4,11 Z M2,4 C1.44771525,4 1,3.55228475 1,3 C1,2.44771525 1.44771525,2 2,2 C2.55228475,2 3,2.44771525 3,3 C3,3.55228475 2.55228475,4 2,4 Z M2,8 C1.44771525,8 1,7.55228475 1,7 C1,6.44771525 1.44771525,6 2,6 C2.55228475,6 3,6.44771525 3,7 C3,7.55228475 2.55228475,8 2,8 Z M2,12 C1.44771525,12 1,11.5522847 1,11 C1,10.4477153 1.44771525,10 2,10 C2.55228475,10 3,10.4477153 3,11 C3,11.5522847 2.55228475,12 2,12 Z"></path></svg></span>Technique</th><td></td></tr><tr class="property-row property-row-checkbox"><th><span class="icon property-icon"><svg viewBox="0 0 14 14" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0;-webkit-backface-visibility:hidden" class="typesCheckbox"><path d="M0,3 C0,1.34314 1.34326,0 3,0 L11,0 C12.6567,0 14,1.34314 14,3 L14,11 C14,12.6569 12.6567,14 11,14 L3,14 C1.34326,14 0,12.6569 0,11 L0,3 Z M3,1.5 C2.17139,1.5 1.5,2.17157 1.5,3 L1.5,11 C1.5,11.8284 2.17139,12.5 3,12.5 L11,12.5 C11.8286,12.5 12.5,11.8284 12.5,11 L12.5,3 C12.5,2.17157 11.8286,1.5 11,1.5 L3,1.5 Z M2.83252,6.8161 L3.39893,6.27399 L3.57617,6.10425 L3.92334,5.77216 L4.26904,6.10559 L4.44531,6.27582 L5.58398,7.37402 L9.28271,3.81073 L9.45996,3.64008 L9.80664,3.3056 L10.1538,3.63989 L10.3311,3.81067 L10.8936,4.35303 L11.0708,4.52399 L11.4434,4.88379 L11.0708,5.24353 L10.8936,5.41437 L6.1084,10.0291 L5.93115,10.2 L5.58398,10.5344 L5.23682,10.2 L5.05957,10.0292 L2.83057,7.87946 L2.65283,7.70801 L2.27832,7.34674 L2.6543,6.98694 L2.83252,6.8161 Z"></path></svg></span>Dataset Raw</th><td><div class="checkbox checkbox-off"></div></td></tr></tbody></table></header><div class="page-body"><h2 id="649fb7e8-b149-43d1-b8fe-6b3c999d489a" class="">1. Introduction</h2><p id="f4b38254-2c51-4137-9361-0262582701ae" class="">Hirudo medicinalis is a leech used medically to extract blood, and it is also valued for its capacity to avoid coagulation by secreting hirudin, an anticoagulant substance discovered in leech saliva by Haycraft in 1884. It took another century for the substance to be successfully isolated and characterized by Markwardt, in 1986 [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">1</a>].</p><p id="e6b3c4a6-c20a-4aed-8ae1-f02eb081dfe2" class="">The natural habitat of leeches is mostly fresh water from puddles, and the progressive disappearance of this habitat, as well as the intensive collection of wild leeches for medical purpose in the eighteenth century in Central Europe, led to a dramatic decrease in the population and the concomitant classification of the species on the International Union for Conservation of Nature (IUCN) Red List of endangered species [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">2</a>]. Collecting wild leeches from natural environments for medical use can be problematic because of they represent a risk for micro-organism transmission. Indeed, in the natural habitat, leeches are in constant interaction with environmental bacteria, protozoa and viruses, which can colonize their digestive tract. The symbiosis of the leeches from the genus Hirudo with Aeromonas sp. has been well known for years [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">3</a>], and several cases of human transmission of A. hydrophila have been reported when used in microsurgery. In the natural environment, the interactions between leeches and micro-organisms have not been well characterized, even when other associated bacteria have been identified in the animal. Indeed, some Ochrobactrum sp., Bdellovibrio sp. and Sphingobacterium sp. were detected in the bladders of medical leeches [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">7</a>], and a Rikenella-like bacterium was isolated from the gut of a H. verbena species [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">8</a>].</p><p id="34099bcb-4956-4a81-84b2-5d01a468310a" class="">The role of leeches as a potential reservoir for viruses has also been suspected. Even if they are not known to be natural carriers of viruses, the persistence of the classical swine fever virus and the Myxoma virus in their digestive tract was demonstrated after leeches were fed with swine and rabbit blood contaminated with those viruses [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">9</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">10</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">11</a>]. In recent years, several other viruses have been demonstrated or suspected to be harbored by leeches [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">10</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">11</a>]. In the natural environment, leeches may come into contact with other species of viruses, including the giant viruses that infect amoebae and are now well known to be widely present in aquatic environments and soil [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">12</a>]. We describe here the first isolation of a giant virus of the family Mimiviridae from a wild specimen of a medical leech, Hirudo medicinalis.</p><h2 id="97c5c533-e908-419a-b8e5-2078f70afb2f" class="">2. Results</h2><h3 id="b74bfa99-fc58-4183-9bfd-4e9f79d1a0c4" class="">2.1. Virus isolation and preliminary characterization</h3><p id="994be674-dae7-48b7-aac5-3670ca5cafca" class="">The inoculations on amoebae in agar plate showing lysis plaque (<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">Figure 1</a>) resulted in the isolation of one single virus from one Tunisian Hirudo medicinalis (<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">Figure 2</a>) collected at “Oued Sarrath” (35°46&#x27;0&quot; N, 8°37&#x27;0&quot; E). No virus was isolated from PAS buffer used to rinse leeches in order to remove ethanol. The virus was isolated from the internal organs and digestive tract Electron microscopic examination of the negative-stained culture supernatant showed the typical icosahedral structure of giant viruses, with a capsid (410 nm±10) surrounded by long fibrils (110 nm±10) and an overall viral particle diameter of 520 nm±10. Transmission electronic microscopy showed a typical aspect of virus factory within A. polyphaga 12h post infection (<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">Figure 3</a>). Based on this observation, we pre-classified the virus in the family Mimiviridae. This new strain of Mimivirus was named Hirudovirus.</p><figure id="390945a9-d628-4aa3-9e73-b5f4ab9fba22" class="image"><a href="viruses-05-02920-g001-1024.png"><img style="width:700px" src="viruses-05-02920-g001-1024.png"/></a></figure><p id="6c33d664-8c4e-4d72-9f03-37da2d2dbcbf" class="">Figure 1. Aspect of agar plate on which samples were inoculated. (a) negative control, (b) Hirudovirus starin growth leading to lysis of amoebas.</p><figure id="ab581f4e-e9f5-4e8b-874c-bd60d36595b5" class="image"><a href="viruses-05-02920-g002-1024.png"><img style="width:700px" src="viruses-05-02920-g002-1024.png"/></a></figure><p id="e6b0b01b-a990-4605-8000-24c366d536ac" class="">Figure 2. Hirudo medicinalis in which Hirudovirus strain was isolated.</p><figure id="010c86ce-f46f-41ac-a9d4-fd0423a80229" class="image"><a href="viruses-05-02920-g003-1024.png"><img style="width:700px" src="viruses-05-02920-g003-1024.png"/></a></figure><p id="10694075-7d9b-4cba-b8a6-d66afbd01e5d" class="">Typical aspect if Hirudovirus virus factory (VF) growing in</p><p id="e5f00ef7-1c4a-4e49-9201-23390879e0fa" class=""><strong>Figure 3.</strong>A. polyphaga.</p><p id="e711d0f5-583e-47ac-accc-5da878286704" class="">The sequenced genome assembly resulted in two scaffolds of 1,155,382 and 25,660 base pairs, with a low (28%) mean GC content, which is similar to those of other mimiviruses of amoebae [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">13</a>]. A total of 992 predicted genes, ranging in size from 34 to 2,959 amino acids, were detected in the draft genome of the strain Hirudovirus. This corresponds to a protein coding density of 0.9 genes/kilobase pair similar to values determined for Mimivirus (0.83) and Mamavirus (0.86), which belong to lineage A of amoebal mimiviruses [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">13</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">14</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">15</a>]. A functional annotation was identified for 47% of these genes (corresponding to 466 proteins). The comparative genomic analyses showed that the Hirudovirus strain genome shared the highest number of bona fide orthologs with Mimivirus (n= 806; 82% of the Hirudovirus gene content) and Mamavirus (805), whereas 435, 402, 361 and 338 bona fide orthologs were found from other mimiviruses of amoeba including Megavirus chiliensis [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">16</a>], LBA111 [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">17</a>], Moumouvirus [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">18</a>] and Monve [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">12</a>], respectively. Besides, 38% of the Hirudovirus predicted proteins had significant BLASTp hits from organisms outside the family Mimiviridae. Of the Hirudovirus predicted proteins, 774 (78%) and 190 (19%) found a protein from Mimivirus or Mamavirus, respectively, as top hit in the GenBank non-redundant database. Proteins from other mimiviruses were identified as top hits for other Hirudovirus proteins, and among them, 18 were detected in the genome of Lentillevirus, which is classified within lineage C of amoebal mimiviruses [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">19</a>]. Four Hirudovirus predicted proteins were not found by BLASTp in Mimivirus and Mamavirus but were found in other mimiviruses of amoeba; especially, all four were found in Courdo7 virus, a mimivirus of amoeba from lineage C [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">13</a>]. Finally, two Hirudovirus proteins (HIRU_S223 and HIRU_S244) had no significant BLASTp hit in the NCBI Genbank non-redundant and environmental protein sequence databases but BLAST search using nucleotide sequences detected hits in the Mimivirus and Mamavirus genomes. The Hirudovirus and Mimivirus genomes are highly similar and collinear (<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">Figure 4</a>). Bona fide orthologs from these viruses showed a mean amino acid identity of 99.1±2.6% (range, 72.1-100%), respectively. In addition, mean nucleotide identities for these bona fide orthologs was 99.3±1.8% (range, 80.7-100%), respectively. These values are in the same order of magnitude than those described for bona fide orthologs of Mimivirus and Mamavirus that is another strain of Mimivirus [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">15</a>].</p><figure id="0ddfa7f1-d8b7-4d51-b696-4d4c9b7f8aa2" class="image"><a href="viruses-05-02920-g004-1024.png"><img style="width:700px" src="viruses-05-02920-g004-1024.png"/></a></figure><p id="d1f7504f-8ae6-4ec2-bea7-5255d3e6298c" class=""><strong>Figure 4.</strong> Genomic dotplot between Mimivirus and Hirudovirus pairs of orthologs by BLASTp analysis. The diameter of the circles is proportional with the BLASTp bit score. The arrow indicates the position between the two contigs.</p><p id="3878d9cb-1c33-42e8-963a-5b3934fc82fc" class="">Phylogenic reconstructions based on a concatenated alignment of three conserved genes shows that Hirudovirus strain clusters with Mimivirus and Mamavirus in the lineage A, previously defined for mimiviruses of amoebae [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">13</a>], which agrees with the results of the comparative genomic analyses (<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">Figure 5</a>).</p><figure id="26e9a209-a7ee-4c5f-9211-27c385a35c75" class="image"><a href="viruses-05-02920-g005-1024.png"><img style="width:700px" src="viruses-05-02920-g005-1024.png"/></a></figure><p id="07d60e01-0471-4bfc-a016-042ccf1a76f6" class=""><strong>Figure 5.</strong> Maximum likelihood phylogeny reconstruction based on a concatenated alignment of the VVA18 helicase, the DNA polymerase B family and the VV-A32 packaging ATPase. Phylogeny reconstruction was performed using the MEGA5 software on a total of 1835 positions in the final dataset. Bootstrap values (as a result of 1,000 replicates) are shown as percentages next to the branches. A discrete Gamma distribution was used to model evolutionary rate differences among sites. The tree is drawn to scale, with branch lengths measured in the number of substitutions per site.</p><h2 id="8ec863cf-b65e-4efc-88f6-153139aaec8d" class="">3. Discussion</h2><p id="378b3745-43df-4f97-ba10-903bf8e5fdb8" class="">One single specimen, among the 5 Hirudo medicinalis analyzed, led to the isolation of a giant virus from the internal part of the animal. The virus was absent from the flesh, and the presence of the virus in the internal organs and the gastrointestinal tract suggests that the giant virus may have been ingested with food. Indeed, giant viruses can naturally be found in several ecological niches, such as fresh water, seawater (including hypersaline environments), and soil [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">12</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">20</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">21</a>]. Hence, the presence of giant viruses in the same ecological niches as wild Hirudo species can explain the presence of a mimivirus in the digestive tract of the leech. Several studies have shown that viruses persist in the digestive tract of leeches fed with contaminated blood [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">3</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">5</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">6</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">9</a>]. Moreover, leeches are known to exhibit symbiosis with several bacteria, mainly those species from the genus Aeromonas, and with several proteobacterial genera [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">8</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">9</a>,<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">20</a>]. This symbiosis becomes problematic when leeches are used in a medical context and infected animals eject symbiotic bacteria from their digestive tracts into the bite. Indeed, feeding leeches blood from animals treated with antibiotics can lead to the acquisition of resistance in the symbiotic bacteria, and this complicates the treatment of patients in case of accidental infection through hirudotherapy [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">7</a>]. Alternatively, we cannot exclude the possibility that mimiviruses can be contaminant of water in hospital aquaria used to grow leeches. Further studies should examine this potential niche because giant viruses have already been isolated from aquatic environments (including some that are close to humans, such as hospital water). Another concern stemming from the presence of amoebal mimiviruses in leeches is that a growing body of evidence suggest that these viruses could be a rare cause of pneumonia [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">17</a>]. Notably, mimiviruses of lineage C were recently isolated from the bronchoalveolar fluid and the stools of Tunisian patients with pneumonia. Additionally, a mimivirus of lineage A, a close relative to the strain Hirudovirus, was isolated from the contact lens fluid of a young woman exhibiting keratitis [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">19</a>]. Our work does not find evidence of pathogenic effects of the Mimivirus on the leech. However, further investigations should be performed to evaluate the impact of infection on the viability of these medically important animals. The analysis of the 1.18 megabase genome of the strain shows gene content closely related to Mimivirus and Mamavirus, which belong to lineage A of amoebal mimiviruses [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">13</a>]. Notably, 97% of the Hirudovirus genes were most similar (via BLASTp) to proteins from either Mimivirus or Mamavirus, which is another Mimivirus strain and Hirudovirus and Mimivirus genomes showed high levels of similarity and collinearity. Moreover, these findings are consistent with those of the phylogenetic analysis. Hirudovirus strain is the first giant virus to be isolated from a member of the family Hirudinae. However, other mimiviruses have been isolated from humans, whereas marseilleviruses, other giant viruses that infect amoebae, have been isolated from humans and Eristalis tenax, an insect member of the order Diptera [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">22</a>]. These data suggest that the diversity of the ecological niches for these giant viruses is not yet fully understood.</p><h2 id="86c16352-30cf-42cd-9cdd-0884be46feb6" class="">4. Materials and methods</h2><p id="4ad84402-2b12-4bf3-ad50-972caaec2b73" class=""><strong>Samples and inoculation of amoeba.</strong> Animals tested for the presence of giant viruses came from two small streams at two locations in Tunisia and France. A total of five leeches of the species Hirudo medicinalis were isolated, three in Tunisia and two in France. Wet bottles were used for transporting live samples to the laboratory. The method used to prepare the animals and the procedure for isolating the virus have been recently detailed elsewhere [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">20</a>]. Briefly, after morphological identification, Hirudo medicinalis were rinsed with 96% ethanol for 20 min and then washed with sterile PAS (Page’s amoebal saline). The washing solution was saved. The animals were dissected to obtain samples of flesh (F) and internal organs in digestive tract (IO+DT). These samples were crushed mechanically before inoculation using the high throughput isolation system previously described [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">21</a>].</p><p id="df29d29c-7ad1-4a18-9a6c-28f9c6c354a4" class=""><strong>Preliminary characterization of viruses.</strong> After the appearance of an amoebal lysis plaque, indicates the presence of a giant virus, the agar plate was cut into small pieces around the plaque, resuspended in 500 µl sterile PAS, centrifuged at 15000 rpm for 30 minutes, and re-inoculated onto a fresh amoebal monolayer. The presence of the giant virus was then assessed by Hemacolor staining after cytocentrifugation of a 100 µl sample of culture supernatant. The preliminary classification of the virus was performed by observation of the culture supernatant, which was negative stained with a 3% ammonium molybdate solution and evaluated with electron microscopy.</p><p id="a988deb3-7c44-421b-b315-375a5f08da5f" class=""><strong>Whole genome sequencing.</strong> For whole genome sequencing, 25 culture flasks (150 cm2) were prepared with 5 ml of virus, 10 ml of amoebae and 30 ml of nutritive medium PYG (Proteose peptone, Yeast extract, Glucose). After amoebal lysis, viruses were purified and concentrated; genomic DNA extraction was performed with phenol-chloroform, and the whole genome was pyrosequenced on the 454-Roche GSFLX as previously described [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">23</a>].</p><p id="d69b7a2a-9a52-4b50-8657-93136666a9be" class=""><strong>Sequence assembly, gene prediction and annotation.</strong> The genome assembly was performed with the Newbler software [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">24</a>]. Genes were predicted using the GeneMarkS program [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">25</a>]. All ORFs smaller than 100 codons were considered only if they had a significant hit (BLASTp e-value &lt;1e-4, both query and target coverages &gt;25%) in the NCBI GenBank non-redundant protein sequence database (nr), including genes from Acanthamoeba polyphaga Mimivirus [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">26</a>], Acanthamoeba castellanii mamavirus [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">15</a>], Acanthamoeba polyphaga moumouvirus [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">18</a>], LBA111 [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">17</a>], Megavirus chiliensis [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">16</a>], Moumouvirus monve and the recently described Pandoravirus dulcis and Pandoravirus salinus [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">27</a>]. The Hirudovirus predicted proteins were additionally searched by BLASTp against the NCBI environmental protein sequence (env_nr) and non-redundant nucleotide sequence (nt) databases. Next, genes were annotated using comparative genomics, including BLASTp searches against the GenBank nr database and the published gene repertoires of mimiviruses, and P. dulcis and P. salinus. Bona fide orthologs, which are orthologs identified through the widely used approach consisting in the identification of reciprocal best hits between two gene repertoires, were determined here by the Proteinortho program with an e-value cut-off of 1e-3 and minimum amino acid identity and coverage of 30% and 70%, respectively [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">28</a>]. In addition, a search was performed for all of the orthologous genes shared with the previously cited viruses using BLASTp with an e-value threshold of 1e-4 and minimal coverages of query and target sequences of 25%. Predictions of transfer RNA (tRNA) were performed using the ARAGORN software [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">29</a>].</p><p id="40edbe4e-dd27-4926-931d-a3c4f3e3abb4" class=""><strong>Phylogenetic analysis.</strong> The phylogenetic analysis was based on three concatenated core genes (the VV-A18 helicase, the VV-A32 packaging ATPase and the B family DNA polymerase) of the nucleocytoplasmic large DNA viruses [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">14</a>]. Multiple sequences alignments were performed using the MUSCLE program [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">30</a>], informational positions were selected by the Gblocks tool [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">31</a>] and a phylogenetic tree was constructed using MEGA5 [<a href="https://www.mdpi.com/1999-4915/5/12/2920/htm">32</a>] and the maximum likelihood method. Genes from Marseillevirus and pandoraviruses were used as outgroups.</p><h2 id="55465203-45a1-44c0-9415-76e2ee5decca" class="">References</h2><ol type="1" id="d8d455e7-025f-44a0-9ba5-22dd580d00ce" class="numbered-list" start="1"><li>Whitaker, I.S.; Oboumarzouk, O.; Rozen, W.M.; Naderi, N.; Balasubramanian, S.P.; Azzopardi, E.A.; Kon, M. The efficacy of medicinal leeches in plastic and reconstructive surgery: a systematic review of 277 reported clinical cases. Microsurgery <strong>2012</strong>, 32, 240–249. [<a href="https://scholar.google.com/scholar_lookup?title=The+efficacy+of+medicinal+leeches+in+plastic+and+reconstructive+surgery:+a+systematic+review+of+277+reported+clinical+cases&amp;author=Whitaker,+I.S.&amp;author=Oboumarzouk,+O.&amp;author=Rozen,+W.M.&amp;author=Naderi,+N.&amp;author=Balasubramanian,+S.P.&amp;author=Azzopardi,+E.A.&amp;author=Kon,+M.&amp;publication_year=2012&amp;journal=Microsurgery&amp;volume=32&amp;pages=240%E2%80%93249&amp;doi=10.1002/micr.20971">Google Scholar</a>] [<a href="https://dx.doi.org/10.1002/micr.20971">CrossRef</a>]</li></ol><ol type="1" id="e28b0ef6-5731-475f-827c-ab553dec27b6" class="numbered-list" start="2"><li>Pagnier, I.; Ikanga Reteno, D.G.; Saadi, H.; Boughalmi, M.; Gaia, M.; Slimani, M.; Nto, T.; Bekliz, M.; Colson, P.; Raoult, D.; La Scola, B. A decade of improvements in Mimiviridae and Marseilleviridae isolation from amoeba. Intervirology <strong>2013</strong>, (in press). [<a href="https://scholar.google.com/scholar_lookup?title=A+decade+of+improvements+in+Mimiviridae+and+Marseilleviridae+isolation+from+amoeba&amp;author=Pagnier,+I.&amp;author=Ikanga+Reteno,+D.G.&amp;author=Saadi,+H.&amp;author=Boughalmi,+M.&amp;author=Gaia,+M.&amp;author=Slimani,+M.&amp;author=Nto,+T.&amp;author=Bekliz,+M.&amp;author=Colson,+P.&amp;author=Raoult,+D.&amp;publication_year=2013&amp;journal=Intervirology">Google Scholar</a>]</li></ol><ol type="1" id="54f12f81-416c-4893-afc7-0af782406d9b" class="numbered-list" start="3"><li>Colson, P.; Yutin, N.; Shabalina, S.A.; Robert, C.; Fournous, G.; La Scola, B.; Raoult, D.; Koonin, E.V. Viruses with more than 1,000 genes: Mamavirus, a new Acanthamoeba polyphaga mimivirus strain, and reannotation of Mimivirus genes. Genome Biol. Evol. <strong>2011</strong>, 3, 737–742. [<a href="https://scholar.google.com/scholar_lookup?title=Viruses+with+more+than+1,000+genes:+Mamavirus,+a+new+Acanthamoeba+polyphaga+mimivirus+strain,+and+reannotation+of+Mimivirus+genes&amp;author=Colson,+P.&amp;author=Yutin,+N.&amp;author=Shabalina,+S.A.&amp;author=Robert,+C.&amp;author=Fournous,+G.&amp;author=La+Scola,+B.&amp;author=Raoult,+D.&amp;author=Koonin,+E.V.&amp;publication_year=2011&amp;journal=Genome+Biol.+Evol.&amp;volume=3&amp;pages=737%E2%80%93742&amp;doi=10.1093/gbe/evr048">Google Scholar</a>] [<a href="https://dx.doi.org/10.1093/gbe/evr048">CrossRef</a>]</li></ol><ol type="1" id="fec522b5-976d-4250-be10-2db29c4abbfc" class="numbered-list" start="4"><li>Yoosuf, N.; Yutin, N.; Colson, P.; Shabalina, S.A.; Pagnier, I.; Robert, C.; Azza, S.; Klose, T.; Wong, J.; Rossmann, M.G.; La Scola, B.; Raoult, D.; Koonin, E.V. Related giant viruses in distant locations and different habitats: Acanthamoeba polyphaga moumouvirus represents a third lineage of the Mimiviridae that is close to the megavirus lineage. Genome. Biol. Evol. 2012 4, 1324–1330.</li></ol><ol type="1" id="79db884a-9b03-4576-ab85-a49d037ad7c6" class="numbered-list" start="5"><li>Boyer, M.; Yutin, N.; Pagnier, I.; Barrassi, L.; Fournous, G.; Espinosa, L.; Robert, C.; Azza, S.; Sun, S.; Rossmann, M.G.; Suzan-Monti, M.; La Scola, B.; Koonin, E.V.; Raoult, D. Giant Marseillevirus highlights the role of amoebae as a melting pot in emergence of chimeric microorganisms. Proc. Natl. Acad. Sci. U.S.A. <strong>2009</strong>, 106, 21848–21853. [<a href="https://scholar.google.com/scholar_lookup?title=Giant+Marseillevirus+highlights+the+role+of+amoebae+as+a+melting+pot+in+emergence+of+chimeric+microorganisms&amp;author=Boyer,+M.&amp;author=Yutin,+N.&amp;author=Pagnier,+I.&amp;author=Barrassi,+L.&amp;author=Fournous,+G.&amp;author=Espinosa,+L.&amp;author=Robert,+C.&amp;author=Azza,+S.&amp;author=Sun,+S.&amp;author=Rossmann,+M.G.&amp;publication_year=2009&amp;journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;volume=106&amp;pages=21848%E2%80%9321853">Google Scholar</a>]</li></ol><ol type="1" id="d914f619-4e66-46e4-92a5-decb8c26d8f6" class="numbered-list" start="6"><li>Margulies, M.; Egholm, M.; Altman, W.E.; Attiya, S.; Bader, J.S.; Bemben, L.A.; Berka, J.; Braverman, M.S.; Chen, Y.J.; Chen, Z.; Dewell, S.B.; Du, L.; Fierro, J.M.; Gomes, X.V.; Godwin, B.C.; He, W.; Helgesen, S.; Ho, C.H.; Irzyk, G.P.; Jando, S.C.; Alenquer, M.L.; Jarvie, T.P.; Jirage, K.B.; Kim, J.B.; Knight, J.R.; Lanza, J.R.; Leamon, J.H.; Lefkowitz, S.M.; Lei, M.; Li, J.; Lohman, K.L.; Lu, H.; Makhijani, V.B.; McDade, K.E.; McKenna, M.P.; Myers, E.W.; Nickerson, E.; Nobile, J.R.; Plant, R.; Puc, B.P.; Ronan, M.T.; Roth, G.T.; Sarkis, G.J.; Simons, J.F.; Simpson, J.W.; Srinivasan, M.; Tartaro, K.R.; Tomasz, A.; Vogt, K.A.; Volkmer, G.A.; Wang, S.H.; Wang, Y.; Weiner, M.P.; Yu, P.; Begley, R.F.; Rothberg, J.M. Genome sequencing in microfabricated high-density picolitre reactors. Nature <strong>2005</strong>, 437, 376–380. [<a href="https://scholar.google.com/scholar_lookup?title=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors&amp;author=Margulies,+M.&amp;author=Egholm,+M.&amp;author=Altman,+W.E.&amp;author=Attiya,+S.&amp;author=Bader,+J.S.&amp;author=Bemben,+L.A.&amp;author=Berka,+J.&amp;author=Braverman,+M.S.&amp;author=Chen,+Y.J.&amp;author=Chen,+Z.&amp;publication_year=2005&amp;journal=Nature&amp;volume=437&amp;pages=376%E2%80%93380">Google Scholar</a>]</li></ol><ol type="1" id="962847aa-3836-41ef-82cc-387e40734172" class="numbered-list" start="7"><li>Philippe, N.; Legendre, M.; Doutre, G.; Couté, Y.; Poirot, O.; Lescot, M.; Arslan, D.; Seltzer, V.; Bertaux, L.; Bruley, C.; Garin, J.; Claverie, J.M.; Abergel, C. Pandoraviruses: amoeba viruses with genomes up to 2.5 Mb reaching that of parasitic eukaryotes. Science <strong>2013</strong>, 341, 281–286. [<a href="https://scholar.google.com/scholar_lookup?title=Pandoraviruses:+amoeba+viruses+with+genomes+up+to+2.5+Mb+reaching+that+of+parasitic+eukaryotes&amp;author=Philippe,+N.&amp;author=Legendre,+M.&amp;author=Doutre,+G.&amp;author=Cout%C3%A9,+Y.&amp;author=Poirot,+O.&amp;author=Lescot,+M.&amp;author=Arslan,+D.&amp;author=Seltzer,+V.&amp;author=Bertaux,+L.&amp;author=Bruley,+C.&amp;publication_year=2013&amp;journal=Science&amp;volume=341&amp;pages=281%E2%80%93286&amp;doi=10.1126/science.1239181">Google Scholar</a>] [<a href="https://dx.doi.org/10.1126/science.1239181">CrossRef</a>]</li></ol><p id="8cb75707-1d0b-4297-af38-7bb9f5b5c18c" class="">© 2013 by the authors; licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution license (<a href="http://creativecommons.org/licenses/by/3.0/">http://creativecommons.org/licenses/by/3.0/</a>).</p></div></article></body></html>